// lucide-solid/dist/source/aliases/prefixed.jsx
/**
* @license lucide-solid v1.16.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/prefixed.ts
import { default as default2 } from "../icons/a-arrow-down";
import { default as default3 } from "../icons/a-arrow-up";
import { default as default4 } from "../icons/a-large-small";
import { default as default5 } from "../icons/activity";
import { default as default6 } from "../icons/accessibility";
import { default as default7 } from "../icons/air-vent";
import { default as default8 } from "../icons/airplay";
import { default as default9 } from "../icons/alarm-clock-off";
import { default as default10 } from "../icons/alarm-clock";
import { default as default11 } from "../icons/alarm-smoke";
import { default as default12 } from "../icons/album";
import { default as default13 } from "../icons/align-center-horizontal";
import { default as default14 } from "../icons/align-center-vertical";
import { default as default15 } from "../icons/align-end-horizontal";
import { default as default16 } from "../icons/align-end-vertical";
import { default as default17 } from "../icons/align-horizontal-distribute-center";
import { default as default18 } from "../icons/align-horizontal-distribute-end";
import { default as default19 } from "../icons/align-horizontal-distribute-start";
import { default as default20 } from "../icons/align-horizontal-justify-center";
import { default as default21 } from "../icons/align-horizontal-justify-end";
import { default as default22 } from "../icons/align-horizontal-justify-start";
import { default as default23 } from "../icons/align-horizontal-space-around";
import { default as default24 } from "../icons/align-horizontal-space-between";
import { default as default25 } from "../icons/align-start-horizontal";
import { default as default26 } from "../icons/align-vertical-distribute-center";
import { default as default27 } from "../icons/align-vertical-distribute-end";
import { default as default28 } from "../icons/align-start-vertical";
import { default as default29 } from "../icons/align-vertical-distribute-start";
import { default as default30 } from "../icons/align-vertical-justify-center";
import { default as default31 } from "../icons/align-vertical-justify-start";
import { default as default32 } from "../icons/align-vertical-justify-end";
import { default as default33 } from "../icons/align-vertical-space-around";
import { default as default34 } from "../icons/align-vertical-space-between";
import { default as default35 } from "../icons/ambulance";
import { default as default36 } from "../icons/ampersand";
import { default as default37 } from "../icons/ampersands";
import { default as default38 } from "../icons/amphora";
import { default as default39 } from "../icons/anchor";
import { default as default40 } from "../icons/angry";
import { default as default41 } from "../icons/annoyed";
import { default as default42 } from "../icons/antenna";
import { default as default43 } from "../icons/anvil";
import { default as default44 } from "../icons/aperture";
import { default as default45 } from "../icons/app-window-mac";
import { default as default46 } from "../icons/app-window";
import { default as default47 } from "../icons/apple";
import { default as default48 } from "../icons/archive-x";
import { default as default49 } from "../icons/archive-restore";
import { default as default50 } from "../icons/archive";
import { default as default51 } from "../icons/armchair";
import { default as default52 } from "../icons/arrow-big-down-dash";
import { default as default53 } from "../icons/arrow-big-down";
import { default as default54 } from "../icons/arrow-big-left-dash";
import { default as default55 } from "../icons/arrow-big-left";
import { default as default56 } from "../icons/arrow-big-right-dash";
import { default as default57 } from "../icons/arrow-big-right";
import { default as default58 } from "../icons/arrow-big-up-dash";
import { default as default59 } from "../icons/arrow-big-up";
import { default as default60 } from "../icons/arrow-down-from-line";
import { default as default61 } from "../icons/arrow-down-left";
import { default as default62 } from "../icons/arrow-down-narrow-wide";
import { default as default63 } from "../icons/arrow-down-right";
import { default as default64 } from "../icons/arrow-down-to-dot";
import { default as default65 } from "../icons/arrow-down-up";
import { default as default66 } from "../icons/arrow-down-to-line";
import { default as default67 } from "../icons/arrow-down";
import { default as default68 } from "../icons/arrow-left-from-line";
import { default as default69 } from "../icons/arrow-left-to-line";
import { default as default70 } from "../icons/arrow-left-right";
import { default as default71 } from "../icons/arrow-left";
import { default as default72 } from "../icons/arrow-right-from-line";
import { default as default73 } from "../icons/arrow-right-left";
import { default as default74 } from "../icons/arrow-right-to-line";
import { default as default75 } from "../icons/arrow-right";
import { default as default76 } from "../icons/arrow-up-down";
import { default as default77 } from "../icons/arrow-up-from-dot";
import { default as default78 } from "../icons/arrow-up-left";
import { default as default79 } from "../icons/arrow-up-from-line";
import { default as default80 } from "../icons/arrow-up-right";
import { default as default81 } from "../icons/arrow-up-to-line";
import { default as default82 } from "../icons/arrow-up-wide-narrow";
import { default as default83 } from "../icons/arrow-up";
import { default as default84 } from "../icons/arrows-up-from-line";
import { default as default85 } from "../icons/asterisk";
import { default as default86 } from "../icons/astroid";
import { default as default87 } from "../icons/at-sign";
import { default as default88 } from "../icons/atom";
import { default as default89 } from "../icons/audio-lines";
import { default as default90 } from "../icons/audio-waveform";
import { default as default91 } from "../icons/award";
import { default as default92 } from "../icons/axe";
import { default as default93 } from "../icons/baby";
import { default as default94 } from "../icons/backpack";
import { default as default95 } from "../icons/badge-cent";
import { default as default96 } from "../icons/badge-alert";
import { default as default97 } from "../icons/badge-dollar-sign";
import { default as default98 } from "../icons/badge-euro";
import { default as default99 } from "../icons/badge-indian-rupee";
import { default as default100 } from "../icons/badge-info";
import { default as default101 } from "../icons/badge-minus";
import { default as default102 } from "../icons/badge-japanese-yen";
import { default as default103 } from "../icons/badge-percent";
import { default as default104 } from "../icons/badge-plus";
import { default as default105 } from "../icons/badge-pound-sterling";
import { default as default106 } from "../icons/badge-russian-ruble";
import { default as default107 } from "../icons/badge-swiss-franc";
import { default as default108 } from "../icons/badge-x";
import { default as default109 } from "../icons/badge-turkish-lira";
import { default as default110 } from "../icons/badge";
import { default as default111 } from "../icons/baggage-claim";
import { default as default112 } from "../icons/balloon";
import { default as default113 } from "../icons/ban";
import { default as default114 } from "../icons/banana";
import { default as default115 } from "../icons/bandage";
import { default as default116 } from "../icons/banknote-arrow-down";
import { default as default117 } from "../icons/banknote-arrow-up";
import { default as default118 } from "../icons/banknote-x";
import { default as default119 } from "../icons/banknote";
import { default as default120 } from "../icons/barcode";
import { default as default121 } from "../icons/barrel";
import { default as default122 } from "../icons/baseline";
import { default as default123 } from "../icons/bath";
import { default as default124 } from "../icons/battery-charging";
import { default as default125 } from "../icons/battery-low";
import { default as default126 } from "../icons/battery-full";
import { default as default127 } from "../icons/battery-medium";
import { default as default128 } from "../icons/battery-plus";
import { default as default129 } from "../icons/battery-warning";
import { default as default130 } from "../icons/battery";
import { default as default131 } from "../icons/beaker";
import { default as default132 } from "../icons/bean-off";
import { default as default133 } from "../icons/bean";
import { default as default134 } from "../icons/bed-double";
import { default as default135 } from "../icons/bed-single";
import { default as default136 } from "../icons/bed";
import { default as default137 } from "../icons/beef-off";
import { default as default138 } from "../icons/beef";
import { default as default139 } from "../icons/beer-off";
import { default as default140 } from "../icons/beer";
import { default as default141 } from "../icons/bell-check";
import { default as default142 } from "../icons/bell-dot";
import { default as default143 } from "../icons/bell-electric";
import { default as default144 } from "../icons/bell-minus";
import { default as default145 } from "../icons/bell-off";
import { default as default146 } from "../icons/bell-plus";
import { default as default147 } from "../icons/bell-ring";
import { default as default148 } from "../icons/bell";
import { default as default149 } from "../icons/between-vertical-end";
import { default as default150 } from "../icons/between-vertical-start";
import { default as default151 } from "../icons/biceps-flexed";
import { default as default152 } from "../icons/bike";
import { default as default153 } from "../icons/binary";
import { default as default154 } from "../icons/binoculars";
import { default as default155 } from "../icons/biohazard";
import { default as default156 } from "../icons/bird";
import { default as default157 } from "../icons/birdhouse";
import { default as default158 } from "../icons/bitcoin";
import { default as default159 } from "../icons/blend";
import { default as default160 } from "../icons/blender";
import { default as default161 } from "../icons/blinds";
import { default as default162 } from "../icons/blocks";
import { default as default163 } from "../icons/bluetooth-connected";
import { default as default164 } from "../icons/bluetooth-searching";
import { default as default165 } from "../icons/bluetooth-off";
import { default as default166 } from "../icons/bluetooth";
import { default as default167 } from "../icons/bold";
import { default as default168 } from "../icons/bolt";
import { default as default169 } from "../icons/bomb";
import { default as default170 } from "../icons/bone";
import { default as default171 } from "../icons/book-a";
import { default as default172 } from "../icons/book-alert";
import { default as default173 } from "../icons/book-audio";
import { default as default174 } from "../icons/book-check";
import { default as default175 } from "../icons/book-copy";
import { default as default176 } from "../icons/book-down";
import { default as default177 } from "../icons/book-headphones";
import { default as default178 } from "../icons/book-image";
import { default as default179 } from "../icons/book-heart";
import { default as default180 } from "../icons/book-key";
import { default as default181 } from "../icons/book-marked";
import { default as default182 } from "../icons/book-lock";
import { default as default183 } from "../icons/book-minus";
import { default as default184 } from "../icons/book-open-text";
import { default as default185 } from "../icons/book-open-check";
import { default as default186 } from "../icons/book-plus";
import { default as default187 } from "../icons/book-open";
import { default as default188 } from "../icons/book-text";
import { default as default189 } from "../icons/book-search";
import { default as default190 } from "../icons/book-type";
import { default as default191 } from "../icons/book-up-2";
import { default as default192 } from "../icons/book-up";
import { default as default193 } from "../icons/book-user";
import { default as default194 } from "../icons/book-x";
import { default as default195 } from "../icons/book";
import { default as default196 } from "../icons/bookmark-check";
import { default as default197 } from "../icons/bookmark-minus";
import { default as default198 } from "../icons/bookmark-plus";
import { default as default199 } from "../icons/bookmark-x";
import { default as default200 } from "../icons/bookmark-off";
import { default as default201 } from "../icons/bookmark";
import { default as default202 } from "../icons/bot-message-square";
import { default as default203 } from "../icons/boom-box";
import { default as default204 } from "../icons/bot-off";
import { default as default205 } from "../icons/bot";
import { default as default206 } from "../icons/bottle-wine";
import { default as default207 } from "../icons/bow-arrow";
import { default as default208 } from "../icons/box";
import { default as default209 } from "../icons/boxes";
import { default as default210 } from "../icons/brackets";
import { default as default211 } from "../icons/brain-circuit";
import { default as default212 } from "../icons/brain";
import { default as default213 } from "../icons/brick-wall-fire";
import { default as default214 } from "../icons/brain-cog";
import { default as default215 } from "../icons/brick-wall-shield";
import { default as default216 } from "../icons/brick-wall";
import { default as default217 } from "../icons/briefcase-business";
import { default as default218 } from "../icons/briefcase-conveyor-belt";
import { default as default219 } from "../icons/briefcase-medical";
import { default as default220 } from "../icons/briefcase";
import { default as default221 } from "../icons/bring-to-front";
import { default as default222 } from "../icons/broccoli";
import { default as default223 } from "../icons/brush-cleaning";
import { default as default224 } from "../icons/brush";
import { default as default225 } from "../icons/bubbles";
import { default as default226 } from "../icons/bug-off";
import { default as default227 } from "../icons/bug-play";
import { default as default228 } from "../icons/bug";
import { default as default229 } from "../icons/building-2";
import { default as default230 } from "../icons/building";
import { default as default231 } from "../icons/bus-front";
import { default as default232 } from "../icons/cable-car";
import { default as default233 } from "../icons/bus";
import { default as default234 } from "../icons/cable";
import { default as default235 } from "../icons/cake-slice";
import { default as default236 } from "../icons/cake";
import { default as default237 } from "../icons/calculator";
import { default as default238 } from "../icons/calendar-1";
import { default as default239 } from "../icons/calendar-arrow-down";
import { default as default240 } from "../icons/calendar-arrow-up";
import { default as default241 } from "../icons/calendar-check-2";
import { default as default242 } from "../icons/calendar-clock";
import { default as default243 } from "../icons/calendar-cog";
import { default as default244 } from "../icons/calendar-check";
import { default as default245 } from "../icons/calendar-days";
import { default as default246 } from "../icons/calendar-fold";
import { default as default247 } from "../icons/calendar-heart";
import { default as default248 } from "../icons/calendar-minus-2";
import { default as default249 } from "../icons/calendar-minus";
import { default as default250 } from "../icons/calendar-off";
import { default as default251 } from "../icons/calendar-plus-2";
import { default as default252 } from "../icons/calendar-plus";
import { default as default253 } from "../icons/calendar-range";
import { default as default254 } from "../icons/calendar-search";
import { default as default255 } from "../icons/calendar-sync";
import { default as default256 } from "../icons/calendar-x-2";
import { default as default257 } from "../icons/calendar-x";
import { default as default258 } from "../icons/calendar";
import { default as default259 } from "../icons/calendars";
import { default as default260 } from "../icons/camera-off";
import { default as default261 } from "../icons/camera";
import { default as default262 } from "../icons/candy-cane";
import { default as default263 } from "../icons/candy-off";
import { default as default264 } from "../icons/cannabis-off";
import { default as default265 } from "../icons/candy";
import { default as default266 } from "../icons/cannabis";
import { default as default267 } from "../icons/car-front";
import { default as default268 } from "../icons/car-taxi-front";
import { default as default269 } from "../icons/captions-off";
import { default as default270 } from "../icons/car";
import { default as default271 } from "../icons/caravan";
import { default as default272 } from "../icons/card-sim";
import { default as default273 } from "../icons/case-lower";
import { default as default274 } from "../icons/carrot";
import { default as default275 } from "../icons/case-sensitive";
import { default as default276 } from "../icons/case-upper";
import { default as default277 } from "../icons/cassette-tape";
import { default as default278 } from "../icons/cast";
import { default as default279 } from "../icons/castle";
import { default as default280 } from "../icons/cat";
import { default as default281 } from "../icons/cctv-off";
import { default as default282 } from "../icons/cctv";
import { default as default283 } from "../icons/chart-bar-decreasing";
import { default as default284 } from "../icons/chart-bar-increasing";
import { default as default285 } from "../icons/chart-bar-stacked";
import { default as default286 } from "../icons/chart-column-decreasing";
import { default as default287 } from "../icons/chart-column-stacked";
import { default as default288 } from "../icons/chart-gantt";
import { default as default289 } from "../icons/chart-network";
import { default as default290 } from "../icons/chart-no-axes-column-decreasing";
import { default as default291 } from "../icons/chart-no-axes-combined";
import { default as default292 } from "../icons/chart-spline";
import { default as default293 } from "../icons/check-check";
import { default as default294 } from "../icons/check-line";
import { default as default295 } from "../icons/check";
import { default as default296 } from "../icons/chef-hat";
import { default as default297 } from "../icons/cherry";
import { default as default298 } from "../icons/chess-bishop";
import { default as default299 } from "../icons/chess-king";
import { default as default300 } from "../icons/chess-knight";
import { default as default301 } from "../icons/chess-pawn";
import { default as default302 } from "../icons/chess-queen";
import { default as default303 } from "../icons/chevron-down";
import { default as default304 } from "../icons/chess-rook";
import { default as default305 } from "../icons/chevron-first";
import { default as default306 } from "../icons/chevron-last";
import { default as default307 } from "../icons/chevron-left";
import { default as default308 } from "../icons/chevron-right";
import { default as default309 } from "../icons/chevron-up";
import { default as default310 } from "../icons/chevrons-down-up";
import { default as default311 } from "../icons/chevrons-down";
import { default as default312 } from "../icons/chevrons-left-right-ellipsis";
import { default as default313 } from "../icons/chevrons-left-right";
import { default as default314 } from "../icons/chevrons-left";
import { default as default315 } from "../icons/chevrons-right-left";
import { default as default316 } from "../icons/chevrons-right";
import { default as default317 } from "../icons/chevrons-up-down";
import { default as default318 } from "../icons/chevrons-up";
import { default as default319 } from "../icons/church";
import { default as default320 } from "../icons/cigarette-off";
import { default as default321 } from "../icons/cigarette";
import { default as default322 } from "../icons/circle-dashed";
import { default as default323 } from "../icons/circle-dollar-sign";
import { default as default324 } from "../icons/circle-dot-dashed";
import { default as default325 } from "../icons/circle-dot";
import { default as default326 } from "../icons/circle-ellipsis";
import { default as default327 } from "../icons/circle-equal";
import { default as default328 } from "../icons/circle-fading-arrow-up";
import { default as default329 } from "../icons/circle-fading-plus";
import { default as default330 } from "../icons/circle-off";
import { default as default331 } from "../icons/circle-pile";
import { default as default332 } from "../icons/circle-pound-sterling";
import { default as default333 } from "../icons/circle-slash";
import { default as default334 } from "../icons/circle-star";
import { default as default335 } from "../icons/circle-small";
import { default as default336 } from "../icons/circle";
import { default as default337 } from "../icons/circuit-board";
import { default as default338 } from "../icons/citrus";
import { default as default339 } from "../icons/clipboard-check";
import { default as default340 } from "../icons/clipboard-clock";
import { default as default341 } from "../icons/clapperboard";
import { default as default342 } from "../icons/clipboard-copy";
import { default as default343 } from "../icons/clipboard-list";
import { default as default344 } from "../icons/clipboard-minus";
import { default as default345 } from "../icons/clipboard-paste";
import { default as default346 } from "../icons/clipboard-plus";
import { default as default347 } from "../icons/clipboard-type";
import { default as default348 } from "../icons/clipboard-x";
import { default as default349 } from "../icons/clipboard";
import { default as default350 } from "../icons/clock-1";
import { default as default351 } from "../icons/clock-10";
import { default as default352 } from "../icons/clock-11";
import { default as default353 } from "../icons/clock-12";
import { default as default354 } from "../icons/clock-2";
import { default as default355 } from "../icons/clock-3";
import { default as default356 } from "../icons/clock-4";
import { default as default357 } from "../icons/clock-5";
import { default as default358 } from "../icons/clock-6";
import { default as default359 } from "../icons/clock-7";
import { default as default360 } from "../icons/clock-8";
import { default as default361 } from "../icons/clock-alert";
import { default as default362 } from "../icons/clock-9";
import { default as default363 } from "../icons/clock-arrow-down";
import { default as default364 } from "../icons/clock-arrow-up";
import { default as default365 } from "../icons/clock-check";
import { default as default366 } from "../icons/clock-fading";
import { default as default367 } from "../icons/clock-plus";
import { default as default368 } from "../icons/closed-caption";
import { default as default369 } from "../icons/clock";
import { default as default370 } from "../icons/cloud-alert";
import { default as default371 } from "../icons/cloud-backup";
import { default as default372 } from "../icons/cloud-check";
import { default as default373 } from "../icons/cloud-cog";
import { default as default374 } from "../icons/cloud-drizzle";
import { default as default375 } from "../icons/cloud-fog";
import { default as default376 } from "../icons/cloud-hail";
import { default as default377 } from "../icons/cloud-lightning";
import { default as default378 } from "../icons/cloud-moon-rain";
import { default as default379 } from "../icons/cloud-moon";
import { default as default380 } from "../icons/cloud-off";
import { default as default381 } from "../icons/cloud-rain-wind";
import { default as default382 } from "../icons/cloud-rain";
import { default as default383 } from "../icons/cloud-snow";
import { default as default384 } from "../icons/cloud-sun-rain";
import { default as default385 } from "../icons/cloud-sun";
import { default as default386 } from "../icons/cloud-sync";
import { default as default387 } from "../icons/cloud";
import { default as default388 } from "../icons/cloudy";
import { default as default389 } from "../icons/clover";
import { default as default390 } from "../icons/club";
import { default as default391 } from "../icons/code";
import { default as default392 } from "../icons/coffee";
import { default as default393 } from "../icons/cog";
import { default as default394 } from "../icons/coins";
import { default as default395 } from "../icons/combine";
import { default as default396 } from "../icons/columns-4";
import { default as default397 } from "../icons/command";
import { default as default398 } from "../icons/compass";
import { default as default399 } from "../icons/component";
import { default as default400 } from "../icons/concierge-bell";
import { default as default401 } from "../icons/cone";
import { default as default402 } from "../icons/computer";
import { default as default403 } from "../icons/construction";
import { default as default404 } from "../icons/contact";
import { default as default405 } from "../icons/container";
import { default as default406 } from "../icons/contrast";
import { default as default407 } from "../icons/cookie";
import { default as default408 } from "../icons/cooking-pot";
import { default as default409 } from "../icons/copy-check";
import { default as default410 } from "../icons/copy-minus";
import { default as default411 } from "../icons/copy-plus";
import { default as default412 } from "../icons/copy-x";
import { default as default413 } from "../icons/copy-slash";
import { default as default414 } from "../icons/copy";
import { default as default415 } from "../icons/copyleft";
import { default as default416 } from "../icons/copyright";
import { default as default417 } from "../icons/corner-down-left";
import { default as default418 } from "../icons/corner-down-right";
import { default as default419 } from "../icons/corner-left-down";
import { default as default420 } from "../icons/corner-left-up";
import { default as default421 } from "../icons/corner-right-down";
import { default as default422 } from "../icons/corner-right-up";
import { default as default423 } from "../icons/corner-up-left";
import { default as default424 } from "../icons/cpu";
import { default as default425 } from "../icons/corner-up-right";
import { default as default426 } from "../icons/credit-card";
import { default as default427 } from "../icons/creative-commons";
import { default as default428 } from "../icons/croissant";
import { default as default429 } from "../icons/crop";
import { default as default430 } from "../icons/crosshair";
import { default as default431 } from "../icons/cross";
import { default as default432 } from "../icons/crown";
import { default as default433 } from "../icons/cuboid";
import { default as default434 } from "../icons/cup-soda";
import { default as default435 } from "../icons/currency";
import { default as default436 } from "../icons/cylinder";
import { default as default437 } from "../icons/dam";
import { default as default438 } from "../icons/database-backup";
import { default as default439 } from "../icons/database-search";
import { default as default440 } from "../icons/database-zap";
import { default as default441 } from "../icons/database";
import { default as default442 } from "../icons/decimals-arrow-left";
import { default as default443 } from "../icons/decimals-arrow-right";
import { default as default444 } from "../icons/dessert";
import { default as default445 } from "../icons/delete";
import { default as default446 } from "../icons/diameter";
import { default as default447 } from "../icons/diamond-minus";
import { default as default448 } from "../icons/diamond-plus";
import { default as default449 } from "../icons/diamond";
import { default as default450 } from "../icons/dice-1";
import { default as default451 } from "../icons/dice-2";
import { default as default452 } from "../icons/dice-3";
import { default as default453 } from "../icons/dice-4";
import { default as default454 } from "../icons/dice-5";
import { default as default455 } from "../icons/dice-6";
import { default as default456 } from "../icons/dices";
import { default as default457 } from "../icons/diff";
import { default as default458 } from "../icons/disc-2";
import { default as default459 } from "../icons/disc-album";
import { default as default460 } from "../icons/disc-3";
import { default as default461 } from "../icons/disc";
import { default as default462 } from "../icons/dna-off";
import { default as default463 } from "../icons/divide";
import { default as default464 } from "../icons/dna";
import { default as default465 } from "../icons/dock";
import { default as default466 } from "../icons/dog";
import { default as default467 } from "../icons/dollar-sign";
import { default as default468 } from "../icons/donut";
import { default as default469 } from "../icons/door-closed-locked";
import { default as default470 } from "../icons/door-closed";
import { default as default471 } from "../icons/door-open";
import { default as default472 } from "../icons/dot";
import { default as default473 } from "../icons/download";
import { default as default474 } from "../icons/drafting-compass";
import { default as default475 } from "../icons/drama";
import { default as default476 } from "../icons/drill";
import { default as default477 } from "../icons/drone";
import { default as default478 } from "../icons/droplet-off";
import { default as default479 } from "../icons/droplet";
import { default as default480 } from "../icons/droplets";
import { default as default481 } from "../icons/drum";
import { default as default482 } from "../icons/drumstick";
import { default as default483 } from "../icons/dumbbell";
import { default as default484 } from "../icons/ear-off";
import { default as default485 } from "../icons/ear";
import { default as default486 } from "../icons/earth-lock";
import { default as default487 } from "../icons/egg-fried";
import { default as default488 } from "../icons/egg-off";
import { default as default489 } from "../icons/eclipse";
import { default as default490 } from "../icons/egg";
import { default as default491 } from "../icons/ellipse";
import { default as default492 } from "../icons/equal-approximately";
import { default as default493 } from "../icons/equal-not";
import { default as default494 } from "../icons/equal";
import { default as default495 } from "../icons/eraser";
import { default as default496 } from "../icons/ethernet-port";
import { default as default497 } from "../icons/euro";
import { default as default498 } from "../icons/ev-charger";
import { default as default499 } from "../icons/expand";
import { default as default500 } from "../icons/external-link";
import { default as default501 } from "../icons/eye-closed";
import { default as default502 } from "../icons/eye-off";
import { default as default503 } from "../icons/factory";
import { default as default504 } from "../icons/eye";
import { default as default505 } from "../icons/fan";
import { default as default506 } from "../icons/fast-forward";
import { default as default507 } from "../icons/feather";
import { default as default508 } from "../icons/fence";
import { default as default509 } from "../icons/ferris-wheel";
import { default as default510 } from "../icons/file-archive";
import { default as default511 } from "../icons/file-box";
import { default as default512 } from "../icons/file-check";
import { default as default513 } from "../icons/file-clock";
import { default as default514 } from "../icons/file-code";
import { default as default515 } from "../icons/file-diff";
import { default as default516 } from "../icons/file-digit";
import { default as default517 } from "../icons/file-down";
import { default as default518 } from "../icons/file-heart";
import { default as default519 } from "../icons/file-image";
import { default as default520 } from "../icons/file-input";
import { default as default521 } from "../icons/file-minus";
import { default as default522 } from "../icons/file-music";
import { default as default523 } from "../icons/file-output";
import { default as default524 } from "../icons/file-plus";
import { default as default525 } from "../icons/file-scan";
import { default as default526 } from "../icons/file-search";
import { default as default527 } from "../icons/file-spreadsheet";
import { default as default528 } from "../icons/file-sliders";
import { default as default529 } from "../icons/file-stack";
import { default as default530 } from "../icons/file-symlink";
import { default as default531 } from "../icons/file-terminal";
import { default as default532 } from "../icons/file-text";
import { default as default533 } from "../icons/file-type";
import { default as default534 } from "../icons/file-up";
import { default as default535 } from "../icons/file-user";
import { default as default536 } from "../icons/file-volume";
import { default as default537 } from "../icons/file-x";
import { default as default538 } from "../icons/file";
import { default as default539 } from "../icons/files";
import { default as default540 } from "../icons/film";
import { default as default541 } from "../icons/fish-off";
import { default as default542 } from "../icons/fire-extinguisher";
import { default as default543 } from "../icons/fish-symbol";
import { default as default544 } from "../icons/fishing-hook";
import { default as default545 } from "../icons/fish";
import { default as default546 } from "../icons/fishing-rod";
import { default as default547 } from "../icons/flag-off";
import { default as default548 } from "../icons/flag-triangle-left";
import { default as default549 } from "../icons/flag-triangle-right";
import { default as default550 } from "../icons/flag";
import { default as default551 } from "../icons/flame-kindling";
import { default as default552 } from "../icons/flame";
import { default as default553 } from "../icons/flashlight-off";
import { default as default554 } from "../icons/flashlight";
import { default as default555 } from "../icons/flask-conical-off";
import { default as default556 } from "../icons/flask-conical";
import { default as default557 } from "../icons/flask-round";
import { default as default558 } from "../icons/flip-horizontal-2";
import { default as default559 } from "../icons/flip-vertical-2";
import { default as default560 } from "../icons/flower-2";
import { default as default561 } from "../icons/flower";
import { default as default562 } from "../icons/focus";
import { default as default563 } from "../icons/fold-horizontal";
import { default as default564 } from "../icons/fold-vertical";
import { default as default565 } from "../icons/folder-archive";
import { default as default566 } from "../icons/folder-check";
import { default as default567 } from "../icons/folder-bookmark";
import { default as default568 } from "../icons/folder-clock";
import { default as default569 } from "../icons/folder-closed";
import { default as default570 } from "../icons/folder-code";
import { default as default571 } from "../icons/folder-dot";
import { default as default572 } from "../icons/folder-down";
import { default as default573 } from "../icons/folder-git-2";
import { default as default574 } from "../icons/folder-git";
import { default as default575 } from "../icons/folder-heart";
import { default as default576 } from "../icons/folder-kanban";
import { default as default577 } from "../icons/folder-input";
import { default as default578 } from "../icons/folder-key";
import { default as default579 } from "../icons/folder-lock";
import { default as default580 } from "../icons/folder-minus";
import { default as default581 } from "../icons/folder-open-dot";
import { default as default582 } from "../icons/folder-open";
import { default as default583 } from "../icons/folder-output";
import { default as default584 } from "../icons/folder-plus";
import { default as default585 } from "../icons/folder-root";
import { default as default586 } from "../icons/folder-search-2";
import { default as default587 } from "../icons/folder-search";
import { default as default588 } from "../icons/folder-symlink";
import { default as default589 } from "../icons/folder-sync";
import { default as default590 } from "../icons/folder-tree";
import { default as default591 } from "../icons/folder-up";
import { default as default592 } from "../icons/folder-x";
import { default as default593 } from "../icons/folder";
import { default as default594 } from "../icons/folders";
import { default as default595 } from "../icons/footprints";
import { default as default596 } from "../icons/forklift";
import { default as default597 } from "../icons/form";
import { default as default598 } from "../icons/forward";
import { default as default599 } from "../icons/frame";
import { default as default600 } from "../icons/frown";
import { default as default601 } from "../icons/fuel";
import { default as default602 } from "../icons/fullscreen";
import { default as default603 } from "../icons/funnel-plus";
import { default as default604 } from "../icons/gallery-horizontal-end";
import { default as default605 } from "../icons/gallery-thumbnails";
import { default as default606 } from "../icons/gallery-horizontal";
import { default as default607 } from "../icons/gallery-vertical-end";
import { default as default608 } from "../icons/gallery-vertical";
import { default as default609 } from "../icons/gamepad-2";
import { default as default610 } from "../icons/gamepad-directional";
import { default as default611 } from "../icons/gamepad";
import { default as default612 } from "../icons/gauge";
import { default as default613 } from "../icons/gavel";
import { default as default614 } from "../icons/gem";
import { default as default615 } from "../icons/georgian-lari";
import { default as default616 } from "../icons/ghost";
import { default as default617 } from "../icons/gift";
import { default as default618 } from "../icons/git-branch-minus";
import { default as default619 } from "../icons/git-branch-plus";
import { default as default620 } from "../icons/git-branch";
import { default as default621 } from "../icons/git-commit-vertical";
import { default as default622 } from "../icons/git-compare-arrows";
import { default as default623 } from "../icons/git-compare";
import { default as default624 } from "../icons/git-graph";
import { default as default625 } from "../icons/git-fork";
import { default as default626 } from "../icons/git-merge-conflict";
import { default as default627 } from "../icons/git-merge";
import { default as default628 } from "../icons/git-pull-request-closed";
import { default as default629 } from "../icons/git-pull-request-arrow";
import { default as default630 } from "../icons/git-pull-request-create-arrow";
import { default as default631 } from "../icons/git-pull-request-create";
import { default as default632 } from "../icons/git-pull-request-draft";
import { default as default633 } from "../icons/git-pull-request";
import { default as default634 } from "../icons/glass-water";
import { default as default635 } from "../icons/globe-lock";
import { default as default636 } from "../icons/glasses";
import { default as default637 } from "../icons/globe-off";
import { default as default638 } from "../icons/globe-x";
import { default as default639 } from "../icons/globe";
import { default as default640 } from "../icons/goal";
import { default as default641 } from "../icons/gpu";
import { default as default642 } from "../icons/grape";
import { default as default643 } from "../icons/graduation-cap";
import { default as default644 } from "../icons/grid-3x2";
import { default as default645 } from "../icons/grip-horizontal";
import { default as default646 } from "../icons/grip-vertical";
import { default as default647 } from "../icons/group";
import { default as default648 } from "../icons/grip";
import { default as default649 } from "../icons/guitar";
import { default as default650 } from "../icons/ham";
import { default as default651 } from "../icons/hamburger";
import { default as default652 } from "../icons/hammer";
import { default as default653 } from "../icons/hand-coins";
import { default as default654 } from "../icons/hand-fist";
import { default as default655 } from "../icons/hand-heart";
import { default as default656 } from "../icons/hand-metal";
import { default as default657 } from "../icons/hand-platter";
import { default as default658 } from "../icons/hand";
import { default as default659 } from "../icons/handbag";
import { default as default660 } from "../icons/handshake";
import { default as default661 } from "../icons/hard-drive-download";
import { default as default662 } from "../icons/hard-drive-upload";
import { default as default663 } from "../icons/hard-drive";
import { default as default664 } from "../icons/hard-hat";
import { default as default665 } from "../icons/hash";
import { default as default666 } from "../icons/hat-glasses";
import { default as default667 } from "../icons/haze";
import { default as default668 } from "../icons/hd";
import { default as default669 } from "../icons/hdmi-port";
import { default as default670 } from "../icons/heading-1";
import { default as default671 } from "../icons/heading-2";
import { default as default672 } from "../icons/heading-3";
import { default as default673 } from "../icons/heading-4";
import { default as default674 } from "../icons/heading-5";
import { default as default675 } from "../icons/heading-6";
import { default as default676 } from "../icons/heading";
import { default as default677 } from "../icons/headphone-off";
import { default as default678 } from "../icons/headphones";
import { default as default679 } from "../icons/headset";
import { default as default680 } from "../icons/heart-crack";
import { default as default681 } from "../icons/heart-handshake";
import { default as default682 } from "../icons/heart-minus";
import { default as default683 } from "../icons/heart-off";
import { default as default684 } from "../icons/heart-plus";
import { default as default685 } from "../icons/heart-pulse";
import { default as default686 } from "../icons/heart-x";
import { default as default687 } from "../icons/heart";
import { default as default688 } from "../icons/helicopter";
import { default as default689 } from "../icons/heater";
import { default as default690 } from "../icons/hexagon";
import { default as default691 } from "../icons/highlighter";
import { default as default692 } from "../icons/history";
import { default as default693 } from "../icons/hop-off";
import { default as default694 } from "../icons/hop";
import { default as default695 } from "../icons/hotel";
import { default as default696 } from "../icons/hospital";
import { default as default697 } from "../icons/hourglass";
import { default as default698 } from "../icons/house-heart";
import { default as default699 } from "../icons/house-plug";
import { default as default700 } from "../icons/house-plus";
import { default as default701 } from "../icons/house-wifi";
import { default as default702 } from "../icons/id-card-lanyard";
import { default as default703 } from "../icons/id-card";
import { default as default704 } from "../icons/image-down";
import { default as default705 } from "../icons/image-minus";
import { default as default706 } from "../icons/image-off";
import { default as default707 } from "../icons/image-play";
import { default as default708 } from "../icons/image-plus";
import { default as default709 } from "../icons/image-up";
import { default as default710 } from "../icons/image-upscale";
import { default as default711 } from "../icons/image";
import { default as default712 } from "../icons/images";
import { default as default713 } from "../icons/import";
import { default as default714 } from "../icons/inbox";
import { default as default715 } from "../icons/indian-rupee";
import { default as default716 } from "../icons/info";
import { default as default717 } from "../icons/infinity";
import { default as default718 } from "../icons/inspection-panel";
import { default as default719 } from "../icons/italic";
import { default as default720 } from "../icons/iteration-ccw";
import { default as default721 } from "../icons/iteration-cw";
import { default as default722 } from "../icons/japanese-yen";
import { default as default723 } from "../icons/joystick";
import { default as default724 } from "../icons/kayak";
import { default as default725 } from "../icons/key-round";
import { default as default726 } from "../icons/kanban";
import { default as default727 } from "../icons/key-square";
import { default as default728 } from "../icons/key";
import { default as default729 } from "../icons/keyboard-music";
import { default as default730 } from "../icons/keyboard";
import { default as default731 } from "../icons/keyboard-off";
import { default as default732 } from "../icons/lamp-ceiling";
import { default as default733 } from "../icons/lamp-desk";
import { default as default734 } from "../icons/lamp-floor";
import { default as default735 } from "../icons/lamp-wall-down";
import { default as default736 } from "../icons/lamp-wall-up";
import { default as default737 } from "../icons/lamp";
import { default as default738 } from "../icons/land-plot";
import { default as default739 } from "../icons/landmark";
import { default as default740 } from "../icons/laptop-minimal-check";
import { default as default741 } from "../icons/languages";
import { default as default742 } from "../icons/laptop";
import { default as default743 } from "../icons/lasso-select";
import { default as default744 } from "../icons/lasso";
import { default as default745 } from "../icons/laugh";
import { default as default746 } from "../icons/layers-2";
import { default as default747 } from "../icons/layers-minus";
import { default as default748 } from "../icons/layers-plus";
import { default as default749 } from "../icons/layout-dashboard";
import { default as default750 } from "../icons/layout-grid";
import { default as default751 } from "../icons/layout-list";
import { default as default752 } from "../icons/layout-panel-left";
import { default as default753 } from "../icons/layout-panel-top";
import { default as default754 } from "../icons/layout-template";
import { default as default755 } from "../icons/leaf";
import { default as default756 } from "../icons/leafy-green";
import { default as default757 } from "../icons/lectern";
import { default as default758 } from "../icons/lens-concave";
import { default as default759 } from "../icons/lens-convex";
import { default as default760 } from "../icons/library";
import { default as default761 } from "../icons/library-big";
import { default as default762 } from "../icons/life-buoy";
import { default as default763 } from "../icons/ligature";
import { default as default764 } from "../icons/lightbulb-off";
import { default as default765 } from "../icons/lightbulb";
import { default as default766 } from "../icons/line-dot-right-horizontal";
import { default as default767 } from "../icons/line-squiggle";
import { default as default768 } from "../icons/line-style";
import { default as default769 } from "../icons/link-2-off";
import { default as default770 } from "../icons/link-2";
import { default as default771 } from "../icons/list-check";
import { default as default772 } from "../icons/link";
import { default as default773 } from "../icons/list-checks";
import { default as default774 } from "../icons/list-chevrons-down-up";
import { default as default775 } from "../icons/list-chevrons-up-down";
import { default as default776 } from "../icons/list-end";
import { default as default777 } from "../icons/list-collapse";
import { default as default778 } from "../icons/list-filter-plus";
import { default as default779 } from "../icons/list-filter";
import { default as default780 } from "../icons/list-minus";
import { default as default781 } from "../icons/list-music";
import { default as default782 } from "../icons/list-ordered";
import { default as default783 } from "../icons/list-plus";
import { default as default784 } from "../icons/list-restart";
import { default as default785 } from "../icons/list-start";
import { default as default786 } from "../icons/list-todo";
import { default as default787 } from "../icons/list-tree";
import { default as default788 } from "../icons/list-video";
import { default as default789 } from "../icons/list-x";
import { default as default790 } from "../icons/list";
import { default as default791 } from "../icons/loader-pinwheel";
import { default as default792 } from "../icons/loader";
import { default as default793 } from "../icons/locate-fixed";
import { default as default794 } from "../icons/locate-off";
import { default as default795 } from "../icons/locate";
import { default as default796 } from "../icons/lock-keyhole";
import { default as default797 } from "../icons/lock";
import { default as default798 } from "../icons/log-in";
import { default as default799 } from "../icons/logs";
import { default as default800 } from "../icons/log-out";
import { default as default801 } from "../icons/lollipop";
import { default as default802 } from "../icons/luggage";
import { default as default803 } from "../icons/magnet";
import { default as default804 } from "../icons/mail-check";
import { default as default805 } from "../icons/mail-minus";
import { default as default806 } from "../icons/mail-open";
import { default as default807 } from "../icons/mail-plus";
import { default as default808 } from "../icons/mail-search";
import { default as default809 } from "../icons/mail-warning";
import { default as default810 } from "../icons/mail-x";
import { default as default811 } from "../icons/mail";
import { default as default812 } from "../icons/mailbox";
import { default as default813 } from "../icons/mails";
import { default as default814 } from "../icons/map-minus";
import { default as default815 } from "../icons/map-pin-check-inside";
import { default as default816 } from "../icons/map-pin-check";
import { default as default817 } from "../icons/map-pin-house";
import { default as default818 } from "../icons/map-pin-minus-inside";
import { default as default819 } from "../icons/map-pin-minus";
import { default as default820 } from "../icons/map-pin-off";
import { default as default821 } from "../icons/map-pin-plus-inside";
import { default as default822 } from "../icons/map-pin-plus";
import { default as default823 } from "../icons/map-pin-search";
import { default as default824 } from "../icons/map-pin-x-inside";
import { default as default825 } from "../icons/map-pin-x";
import { default as default826 } from "../icons/map-pin";
import { default as default827 } from "../icons/map-pinned";
import { default as default828 } from "../icons/map-plus";
import { default as default829 } from "../icons/map";
import { default as default830 } from "../icons/mars-stroke";
import { default as default831 } from "../icons/mars";
import { default as default832 } from "../icons/martini";
import { default as default833 } from "../icons/maximize-2";
import { default as default834 } from "../icons/medal";
import { default as default835 } from "../icons/maximize";
import { default as default836 } from "../icons/megaphone";
import { default as default837 } from "../icons/megaphone-off";
import { default as default838 } from "../icons/meh";
import { default as default839 } from "../icons/menu";
import { default as default840 } from "../icons/memory-stick";
import { default as default841 } from "../icons/merge";
import { default as default842 } from "../icons/message-circle-check";
import { default as default843 } from "../icons/message-circle-code";
import { default as default844 } from "../icons/message-circle-dashed";
import { default as default845 } from "../icons/message-circle-heart";
import { default as default846 } from "../icons/message-circle-more";
import { default as default847 } from "../icons/message-circle-off";
import { default as default848 } from "../icons/message-circle-plus";
import { default as default849 } from "../icons/message-circle-reply";
import { default as default850 } from "../icons/message-circle-warning";
import { default as default851 } from "../icons/message-circle-x";
import { default as default852 } from "../icons/message-circle";
import { default as default853 } from "../icons/message-square-check";
import { default as default854 } from "../icons/message-square-dashed";
import { default as default855 } from "../icons/message-square-code";
import { default as default856 } from "../icons/message-square-diff";
import { default as default857 } from "../icons/message-square-dot";
import { default as default858 } from "../icons/message-square-heart";
import { default as default859 } from "../icons/message-square-lock";
import { default as default860 } from "../icons/message-square-more";
import { default as default861 } from "../icons/message-square-off";
import { default as default862 } from "../icons/message-square-quote";
import { default as default863 } from "../icons/message-square-reply";
import { default as default864 } from "../icons/message-square-share";
import { default as default865 } from "../icons/message-square-plus";
import { default as default866 } from "../icons/message-square-text";
import { default as default867 } from "../icons/message-square-warning";
import { default as default868 } from "../icons/message-square-x";
import { default as default869 } from "../icons/message-square";
import { default as default870 } from "../icons/messages-square";
import { default as default871 } from "../icons/metronome";
import { default as default872 } from "../icons/mic-off";
import { default as default873 } from "../icons/microchip";
import { default as default874 } from "../icons/mic";
import { default as default875 } from "../icons/microscope";
import { default as default876 } from "../icons/microwave";
import { default as default877 } from "../icons/milestone";
import { default as default878 } from "../icons/milk-off";
import { default as default879 } from "../icons/milk";
import { default as default880 } from "../icons/minimize-2";
import { default as default881 } from "../icons/minimize";
import { default as default882 } from "../icons/mirror-rectangular";
import { default as default883 } from "../icons/minus";
import { default as default884 } from "../icons/mirror-round";
import { default as default885 } from "../icons/monitor-check";
import { default as default886 } from "../icons/monitor-cloud";
import { default as default887 } from "../icons/monitor-cog";
import { default as default888 } from "../icons/monitor-dot";
import { default as default889 } from "../icons/monitor-down";
import { default as default890 } from "../icons/monitor-off";
import { default as default891 } from "../icons/monitor-pause";
import { default as default892 } from "../icons/monitor-play";
import { default as default893 } from "../icons/monitor-smartphone";
import { default as default894 } from "../icons/monitor-speaker";
import { default as default895 } from "../icons/monitor-stop";
import { default as default896 } from "../icons/monitor-up";
import { default as default897 } from "../icons/monitor-x";
import { default as default898 } from "../icons/monitor";
import { default as default899 } from "../icons/moon-star";
import { default as default900 } from "../icons/moon";
import { default as default901 } from "../icons/motorbike";
import { default as default902 } from "../icons/mountain-snow";
import { default as default903 } from "../icons/mountain";
import { default as default904 } from "../icons/mouse-left";
import { default as default905 } from "../icons/mouse-off";
import { default as default906 } from "../icons/mouse-pointer-2-off";
import { default as default907 } from "../icons/mouse-pointer-2";
import { default as default908 } from "../icons/mouse-pointer-ban";
import { default as default909 } from "../icons/mouse-pointer-click";
import { default as default910 } from "../icons/mouse-pointer";
import { default as default911 } from "../icons/mouse-right";
import { default as default912 } from "../icons/mouse";
import { default as default913 } from "../icons/move-diagonal-2";
import { default as default914 } from "../icons/move-diagonal";
import { default as default915 } from "../icons/move-down-left";
import { default as default916 } from "../icons/move-down-right";
import { default as default917 } from "../icons/move-down";
import { default as default918 } from "../icons/move-horizontal";
import { default as default919 } from "../icons/move-right";
import { default as default920 } from "../icons/move-left";
import { default as default921 } from "../icons/move-up-left";
import { default as default922 } from "../icons/move-up-right";
import { default as default923 } from "../icons/move-up";
import { default as default924 } from "../icons/move";
import { default as default925 } from "../icons/move-vertical";
import { default as default926 } from "../icons/music-2";
import { default as default927 } from "../icons/music-3";
import { default as default928 } from "../icons/music-4";
import { default as default929 } from "../icons/music";
import { default as default930 } from "../icons/navigation-2-off";
import { default as default931 } from "../icons/navigation-2";
import { default as default932 } from "../icons/navigation-off";
import { default as default933 } from "../icons/navigation";
import { default as default934 } from "../icons/network";
import { default as default935 } from "../icons/newspaper";
import { default as default936 } from "../icons/nfc";
import { default as default937 } from "../icons/non-binary";
import { default as default938 } from "../icons/notebook-pen";
import { default as default939 } from "../icons/notebook-tabs";
import { default as default940 } from "../icons/notebook-text";
import { default as default941 } from "../icons/notebook";
import { default as default942 } from "../icons/notepad-text-dashed";
import { default as default943 } from "../icons/notepad-text";
import { default as default944 } from "../icons/nut-off";
import { default as default945 } from "../icons/nut";
import { default as default946 } from "../icons/octagon-minus";
import { default as default947 } from "../icons/octagon";
import { default as default948 } from "../icons/omega";
import { default as default949 } from "../icons/option";
import { default as default950 } from "../icons/orbit";
import { default as default951 } from "../icons/origami";
import { default as default952 } from "../icons/package-2";
import { default as default953 } from "../icons/package-minus";
import { default as default954 } from "../icons/package-check";
import { default as default955 } from "../icons/package-open";
import { default as default956 } from "../icons/package-plus";
import { default as default957 } from "../icons/package-search";
import { default as default958 } from "../icons/package-x";
import { default as default959 } from "../icons/package";
import { default as default960 } from "../icons/paint-bucket";
import { default as default961 } from "../icons/paint-roller";
import { default as default962 } from "../icons/paintbrush";
import { default as default963 } from "../icons/palette";
import { default as default964 } from "../icons/panda";
import { default as default965 } from "../icons/panel-bottom-close";
import { default as default966 } from "../icons/panel-bottom-open";
import { default as default967 } from "../icons/panel-bottom";
import { default as default968 } from "../icons/panel-left-right-dashed";
import { default as default969 } from "../icons/panel-right-close";
import { default as default970 } from "../icons/panel-right-open";
import { default as default971 } from "../icons/panel-right";
import { default as default972 } from "../icons/panel-top-bottom-dashed";
import { default as default973 } from "../icons/panel-top-close";
import { default as default974 } from "../icons/panel-top-open";
import { default as default975 } from "../icons/panel-top";
import { default as default976 } from "../icons/panels-left-bottom";
import { default as default977 } from "../icons/panels-right-bottom";
import { default as default978 } from "../icons/paperclip";
import { default as default979 } from "../icons/parentheses";
import { default as default980 } from "../icons/parking-meter";
import { default as default981 } from "../icons/party-popper";
import { default as default982 } from "../icons/paw-print";
import { default as default983 } from "../icons/pause";
import { default as default984 } from "../icons/pc-case";
import { default as default985 } from "../icons/pen-off";
import { default as default986 } from "../icons/pen-tool";
import { default as default987 } from "../icons/pencil-line";
import { default as default988 } from "../icons/pencil-off";
import { default as default989 } from "../icons/pencil-ruler";
import { default as default990 } from "../icons/pencil";
import { default as default991 } from "../icons/pentagon";
import { default as default992 } from "../icons/percent";
import { default as default993 } from "../icons/person-standing";
import { default as default994 } from "../icons/philippine-peso";
import { default as default995 } from "../icons/phone-call";
import { default as default996 } from "../icons/phone-forwarded";
import { default as default997 } from "../icons/phone-incoming";
import { default as default998 } from "../icons/phone-missed";
import { default as default999 } from "../icons/phone-off";
import { default as default1000 } from "../icons/phone-outgoing";
import { default as default1001 } from "../icons/phone";
import { default as default1002 } from "../icons/pi";
import { default as default1003 } from "../icons/piano";
import { default as default1004 } from "../icons/pickaxe";
import { default as default1005 } from "../icons/picture-in-picture-2";
import { default as default1006 } from "../icons/picture-in-picture";
import { default as default1007 } from "../icons/piggy-bank";
import { default as default1008 } from "../icons/pilcrow-left";
import { default as default1009 } from "../icons/pilcrow-right";
import { default as default1010 } from "../icons/pilcrow";
import { default as default1011 } from "../icons/pill-bottle";
import { default as default1012 } from "../icons/pill";
import { default as default1013 } from "../icons/pin-off";
import { default as default1014 } from "../icons/pipette";
import { default as default1015 } from "../icons/pin";
import { default as default1016 } from "../icons/pizza";
import { default as default1017 } from "../icons/plane-landing";
import { default as default1018 } from "../icons/plane-takeoff";
import { default as default1019 } from "../icons/plane";
import { default as default1020 } from "../icons/plug-2";
import { default as default1021 } from "../icons/play";
import { default as default1022 } from "../icons/plug";
import { default as default1023 } from "../icons/plus";
import { default as default1024 } from "../icons/pocket-knife";
import { default as default1025 } from "../icons/podcast";
import { default as default1026 } from "../icons/pointer";
import { default as default1027 } from "../icons/pointer-off";
import { default as default1028 } from "../icons/popcorn";
import { default as default1029 } from "../icons/popsicle";
import { default as default1030 } from "../icons/pound-sterling";
import { default as default1031 } from "../icons/power-off";
import { default as default1032 } from "../icons/power";
import { default as default1033 } from "../icons/presentation";
import { default as default1034 } from "../icons/printer-check";
import { default as default1035 } from "../icons/projector";
import { default as default1036 } from "../icons/printer";
import { default as default1037 } from "../icons/printer-x";
import { default as default1038 } from "../icons/proportions";
import { default as default1039 } from "../icons/puzzle";
import { default as default1040 } from "../icons/pyramid";
import { default as default1041 } from "../icons/qr-code";
import { default as default1042 } from "../icons/quote";
import { default as default1043 } from "../icons/rabbit";
import { default as default1044 } from "../icons/radar";
import { default as default1045 } from "../icons/radiation";
import { default as default1046 } from "../icons/radical";
import { default as default1047 } from "../icons/radio-off";
import { default as default1048 } from "../icons/radio-receiver";
import { default as default1049 } from "../icons/radio";
import { default as default1050 } from "../icons/radio-tower";
import { default as default1051 } from "../icons/radius";
import { default as default1052 } from "../icons/rainbow";
import { default as default1053 } from "../icons/rat";
import { default as default1054 } from "../icons/ratio";
import { default as default1055 } from "../icons/receipt-cent";
import { default as default1056 } from "../icons/receipt-euro";
import { default as default1057 } from "../icons/receipt-indian-rupee";
import { default as default1058 } from "../icons/receipt-japanese-yen";
import { default as default1059 } from "../icons/receipt-pound-sterling";
import { default as default1060 } from "../icons/receipt-swiss-franc";
import { default as default1061 } from "../icons/receipt-russian-ruble";
import { default as default1062 } from "../icons/receipt-text";
import { default as default1063 } from "../icons/receipt-turkish-lira";
import { default as default1064 } from "../icons/receipt";
import { default as default1065 } from "../icons/rectangle-circle";
import { default as default1066 } from "../icons/rectangle-goggles";
import { default as default1067 } from "../icons/rectangle-horizontal";
import { default as default1068 } from "../icons/rectangle-vertical";
import { default as default1069 } from "../icons/recycle";
import { default as default1070 } from "../icons/redo-2";
import { default as default1071 } from "../icons/redo-dot";
import { default as default1072 } from "../icons/redo";
import { default as default1073 } from "../icons/refresh-ccw-dot";
import { default as default1074 } from "../icons/refresh-ccw";
import { default as default1075 } from "../icons/refresh-cw-off";
import { default as default1076 } from "../icons/refresh-cw";
import { default as default1077 } from "../icons/remove-formatting";
import { default as default1078 } from "../icons/refrigerator";
import { default as default1079 } from "../icons/regex";
import { default as default1080 } from "../icons/repeat-1";
import { default as default1081 } from "../icons/repeat-2";
import { default as default1082 } from "../icons/repeat-off";
import { default as default1083 } from "../icons/repeat";
import { default as default1084 } from "../icons/replace-all";
import { default as default1085 } from "../icons/replace";
import { default as default1086 } from "../icons/reply-all";
import { default as default1087 } from "../icons/reply";
import { default as default1088 } from "../icons/rewind";
import { default as default1089 } from "../icons/ribbon";
import { default as default1090 } from "../icons/road";
import { default as default1091 } from "../icons/rocket";
import { default as default1092 } from "../icons/roller-coaster";
import { default as default1093 } from "../icons/rocking-chair";
import { default as default1094 } from "../icons/rose";
import { default as default1095 } from "../icons/rotate-ccw-key";
import { default as default1096 } from "../icons/rotate-ccw-square";
import { default as default1097 } from "../icons/rotate-ccw";
import { default as default1098 } from "../icons/rotate-cw-square";
import { default as default1099 } from "../icons/rotate-cw";
import { default as default1100 } from "../icons/route-off";
import { default as default1101 } from "../icons/route";
import { default as default1102 } from "../icons/router";
import { default as default1103 } from "../icons/rows-4";
import { default as default1104 } from "../icons/rss";
import { default as default1105 } from "../icons/ruler-dimension-line";
import { default as default1106 } from "../icons/ruler";
import { default as default1107 } from "../icons/russian-ruble";
import { default as default1108 } from "../icons/sailboat";
import { default as default1109 } from "../icons/salad";
import { default as default1110 } from "../icons/sandwich";
import { default as default1111 } from "../icons/satellite-dish";
import { default as default1112 } from "../icons/satellite";
import { default as default1113 } from "../icons/saudi-riyal";
import { default as default1114 } from "../icons/save-all";
import { default as default1115 } from "../icons/save-off";
import { default as default1116 } from "../icons/save";
import { default as default1117 } from "../icons/scaling";
import { default as default1118 } from "../icons/scale";
import { default as default1119 } from "../icons/scan-barcode";
import { default as default1120 } from "../icons/scan-eye";
import { default as default1121 } from "../icons/scan-face";
import { default as default1122 } from "../icons/scan-heart";
import { default as default1123 } from "../icons/scan-line";
import { default as default1124 } from "../icons/scan-qr-code";
import { default as default1125 } from "../icons/scan-search";
import { default as default1126 } from "../icons/scan-text";
import { default as default1127 } from "../icons/scan";
import { default as default1128 } from "../icons/school";
import { default as default1129 } from "../icons/scissors-line-dashed";
import { default as default1130 } from "../icons/scissors";
import { default as default1131 } from "../icons/scooter";
import { default as default1132 } from "../icons/screen-share-off";
import { default as default1133 } from "../icons/screen-share";
import { default as default1134 } from "../icons/scroll-text";
import { default as default1135 } from "../icons/scroll";
import { default as default1136 } from "../icons/search-alert";
import { default as default1137 } from "../icons/search-check";
import { default as default1138 } from "../icons/search-code";
import { default as default1139 } from "../icons/search-x";
import { default as default1140 } from "../icons/search-slash";
import { default as default1141 } from "../icons/section";
import { default as default1142 } from "../icons/search";
import { default as default1143 } from "../icons/send";
import { default as default1144 } from "../icons/send-to-back";
import { default as default1145 } from "../icons/separator-horizontal";
import { default as default1146 } from "../icons/server-cog";
import { default as default1147 } from "../icons/separator-vertical";
import { default as default1148 } from "../icons/server-crash";
import { default as default1149 } from "../icons/server-off";
import { default as default1150 } from "../icons/server";
import { default as default1151 } from "../icons/settings-2";
import { default as default1152 } from "../icons/settings";
import { default as default1153 } from "../icons/shapes";
import { default as default1154 } from "../icons/share-2";
import { default as default1155 } from "../icons/share";
import { default as default1156 } from "../icons/sheet";
import { default as default1157 } from "../icons/shell";
import { default as default1158 } from "../icons/shelving-unit";
import { default as default1159 } from "../icons/shield-alert";
import { default as default1160 } from "../icons/shield-ban";
import { default as default1161 } from "../icons/shield-check";
import { default as default1162 } from "../icons/shield-cog-corner";
import { default as default1163 } from "../icons/shield-cog";
import { default as default1164 } from "../icons/shield-ellipsis";
import { default as default1165 } from "../icons/shield-half";
import { default as default1166 } from "../icons/shield-minus";
import { default as default1167 } from "../icons/shield-off";
import { default as default1168 } from "../icons/shield-plus";
import { default as default1169 } from "../icons/shield-user";
import { default as default1170 } from "../icons/ship-wheel";
import { default as default1171 } from "../icons/shield";
import { default as default1172 } from "../icons/ship";
import { default as default1173 } from "../icons/shirt";
import { default as default1174 } from "../icons/shopping-bag";
import { default as default1175 } from "../icons/shopping-basket";
import { default as default1176 } from "../icons/shopping-cart";
import { default as default1177 } from "../icons/shovel";
import { default as default1178 } from "../icons/shower-head";
import { default as default1179 } from "../icons/shredder";
import { default as default1180 } from "../icons/shrimp";
import { default as default1181 } from "../icons/shrink";
import { default as default1182 } from "../icons/shrub";
import { default as default1183 } from "../icons/shuffle";
import { default as default1184 } from "../icons/sigma";
import { default as default1185 } from "../icons/signal-high";
import { default as default1186 } from "../icons/signal-medium";
import { default as default1187 } from "../icons/signal-low";
import { default as default1188 } from "../icons/signal-zero";
import { default as default1189 } from "../icons/signal";
import { default as default1190 } from "../icons/signpost-big";
import { default as default1191 } from "../icons/signature";
import { default as default1192 } from "../icons/signpost";
import { default as default1193 } from "../icons/siren";
import { default as default1194 } from "../icons/skip-back";
import { default as default1195 } from "../icons/skip-forward";
import { default as default1196 } from "../icons/skull";
import { default as default1197 } from "../icons/slash";
import { default as default1198 } from "../icons/slice";
import { default as default1199 } from "../icons/sliders-horizontal";
import { default as default1200 } from "../icons/smartphone-charging";
import { default as default1201 } from "../icons/smartphone-nfc";
import { default as default1202 } from "../icons/smartphone";
import { default as default1203 } from "../icons/smile-plus";
import { default as default1204 } from "../icons/smile";
import { default as default1205 } from "../icons/snail";
import { default as default1206 } from "../icons/snowflake";
import { default as default1207 } from "../icons/soap-dispenser-droplet";
import { default as default1208 } from "../icons/sofa";
import { default as default1209 } from "../icons/solar-panel";
import { default as default1210 } from "../icons/soup";
import { default as default1211 } from "../icons/space";
import { default as default1212 } from "../icons/spade";
import { default as default1213 } from "../icons/sparkle";
import { default as default1214 } from "../icons/speaker";
import { default as default1215 } from "../icons/speech";
import { default as default1216 } from "../icons/spell-check-2";
import { default as default1217 } from "../icons/spell-check";
import { default as default1218 } from "../icons/spline-pointer";
import { default as default1219 } from "../icons/spline";
import { default as default1220 } from "../icons/split";
import { default as default1221 } from "../icons/spool";
import { default as default1222 } from "../icons/sport-shoe";
import { default as default1223 } from "../icons/spotlight";
import { default as default1224 } from "../icons/spray-can";
import { default as default1225 } from "../icons/sprout";
import { default as default1226 } from "../icons/square-arrow-right-enter";
import { default as default1227 } from "../icons/square-arrow-right-exit";
import { default as default1228 } from "../icons/square-dashed-bottom-code";
import { default as default1229 } from "../icons/square-dashed-bottom";
import { default as default1230 } from "../icons/square-dashed-top-solid";
import { default as default1231 } from "../icons/square-pause";
import { default as default1232 } from "../icons/square-radical";
import { default as default1233 } from "../icons/square-round-corner";
import { default as default1234 } from "../icons/square-square";
import { default as default1235 } from "../icons/square-stack";
import { default as default1236 } from "../icons/square-star";
import { default as default1237 } from "../icons/square-stop";
import { default as default1238 } from "../icons/squares-exclude";
import { default as default1239 } from "../icons/square";
import { default as default1240 } from "../icons/squares-intersect";
import { default as default1241 } from "../icons/squares-subtract";
import { default as default1242 } from "../icons/squares-unite";
import { default as default1243 } from "../icons/squircle";
import { default as default1244 } from "../icons/squircle-dashed";
import { default as default1245 } from "../icons/squirrel";
import { default as default1246 } from "../icons/stamp";
import { default as default1247 } from "../icons/star-half";
import { default as default1248 } from "../icons/star-off";
import { default as default1249 } from "../icons/step-back";
import { default as default1250 } from "../icons/star";
import { default as default1251 } from "../icons/step-forward";
import { default as default1252 } from "../icons/stethoscope";
import { default as default1253 } from "../icons/sticker";
import { default as default1254 } from "../icons/sticky-note-minus";
import { default as default1255 } from "../icons/sticky-note-check";
import { default as default1256 } from "../icons/sticky-note-plus";
import { default as default1257 } from "../icons/sticky-note-off";
import { default as default1258 } from "../icons/sticky-note-x";
import { default as default1259 } from "../icons/sticky-note";
import { default as default1260 } from "../icons/sticky-notes";
import { default as default1261 } from "../icons/stone";
import { default as default1262 } from "../icons/store";
import { default as default1263 } from "../icons/stretch-horizontal";
import { default as default1264 } from "../icons/stretch-vertical";
import { default as default1265 } from "../icons/strikethrough";
import { default as default1266 } from "../icons/subscript";
import { default as default1267 } from "../icons/sun-dim";
import { default as default1268 } from "../icons/sun-medium";
import { default as default1269 } from "../icons/sun-moon";
import { default as default1270 } from "../icons/sun-snow";
import { default as default1271 } from "../icons/sun";
import { default as default1272 } from "../icons/sunrise";
import { default as default1273 } from "../icons/sunset";
import { default as default1274 } from "../icons/superscript";
import { default as default1275 } from "../icons/swatch-book";
import { default as default1276 } from "../icons/swiss-franc";
import { default as default1277 } from "../icons/switch-camera";
import { default as default1278 } from "../icons/sword";
import { default as default1279 } from "../icons/swords";
import { default as default1280 } from "../icons/syringe";
import { default as default1281 } from "../icons/table-2";
import { default as default1282 } from "../icons/table-cells-merge";
import { default as default1283 } from "../icons/table-cells-split";
import { default as default1284 } from "../icons/table-columns-split";
import { default as default1285 } from "../icons/table-of-contents";
import { default as default1286 } from "../icons/table-properties";
import { default as default1287 } from "../icons/table-rows-split";
import { default as default1288 } from "../icons/table";
import { default as default1289 } from "../icons/tablet-smartphone";
import { default as default1290 } from "../icons/tablet";
import { default as default1291 } from "../icons/tablets";
import { default as default1292 } from "../icons/tag";
import { default as default1293 } from "../icons/tally-1";
import { default as default1294 } from "../icons/tally-2";
import { default as default1295 } from "../icons/tags";
import { default as default1296 } from "../icons/tally-3";
import { default as default1297 } from "../icons/tally-4";
import { default as default1298 } from "../icons/tally-5";
import { default as default1299 } from "../icons/tangent";
import { default as default1300 } from "../icons/target";
import { default as default1301 } from "../icons/telescope";
import { default as default1302 } from "../icons/tent";
import { default as default1303 } from "../icons/tent-tree";
import { default as default1304 } from "../icons/terminal";
import { default as default1305 } from "../icons/test-tube";
import { default as default1306 } from "../icons/test-tubes";
import { default as default1307 } from "../icons/text-cursor-input";
import { default as default1308 } from "../icons/text-cursor";
import { default as default1309 } from "../icons/text-quote";
import { default as default1310 } from "../icons/text-search";
import { default as default1311 } from "../icons/theater";
import { default as default1312 } from "../icons/thermometer-snowflake";
import { default as default1313 } from "../icons/thermometer-sun";
import { default as default1314 } from "../icons/thermometer";
import { default as default1315 } from "../icons/thumbs-down";
import { default as default1316 } from "../icons/thumbs-up";
import { default as default1317 } from "../icons/ticket-minus";
import { default as default1318 } from "../icons/ticket-check";
import { default as default1319 } from "../icons/ticket-percent";
import { default as default1320 } from "../icons/ticket-plus";
import { default as default1321 } from "../icons/ticket-x";
import { default as default1322 } from "../icons/ticket-slash";
import { default as default1323 } from "../icons/ticket";
import { default as default1324 } from "../icons/tickets-plane";
import { default as default1325 } from "../icons/timeline";
import { default as default1326 } from "../icons/tickets";
import { default as default1327 } from "../icons/timer-off";
import { default as default1328 } from "../icons/timer-reset";
import { default as default1329 } from "../icons/timer";
import { default as default1330 } from "../icons/toggle-left";
import { default as default1331 } from "../icons/tool-case";
import { default as default1332 } from "../icons/toggle-right";
import { default as default1333 } from "../icons/tornado";
import { default as default1334 } from "../icons/toilet";
import { default as default1335 } from "../icons/toolbox";
import { default as default1336 } from "../icons/torus";
import { default as default1337 } from "../icons/touchpad-off";
import { default as default1338 } from "../icons/towel-rack";
import { default as default1339 } from "../icons/touchpad";
import { default as default1340 } from "../icons/tower-control";
import { default as default1341 } from "../icons/tractor";
import { default as default1342 } from "../icons/toy-brick";
import { default as default1343 } from "../icons/traffic-cone";
import { default as default1344 } from "../icons/train-front-tunnel";
import { default as default1345 } from "../icons/train-front";
import { default as default1346 } from "../icons/train-track";
import { default as default1347 } from "../icons/transgender";
import { default as default1348 } from "../icons/trash-2";
import { default as default1349 } from "../icons/trash";
import { default as default1350 } from "../icons/tree-deciduous";
import { default as default1351 } from "../icons/tree-pine";
import { default as default1352 } from "../icons/trees";
import { default as default1353 } from "../icons/trending-down";
import { default as default1354 } from "../icons/trending-up-down";
import { default as default1355 } from "../icons/trending-up";
import { default as default1356 } from "../icons/triangle-dashed";
import { default as default1357 } from "../icons/triangle-right";
import { default as default1358 } from "../icons/triangle";
import { default as default1359 } from "../icons/trophy";
import { default as default1360 } from "../icons/truck-electric";
import { default as default1361 } from "../icons/truck";
import { default as default1362 } from "../icons/turkish-lira";
import { default as default1363 } from "../icons/turntable";
import { default as default1364 } from "../icons/turtle";
import { default as default1365 } from "../icons/tv-minimal-play";
import { default as default1366 } from "../icons/tv";
import { default as default1367 } from "../icons/type-outline";
import { default as default1368 } from "../icons/type";
import { default as default1369 } from "../icons/umbrella";
import { default as default1370 } from "../icons/umbrella-off";
import { default as default1371 } from "../icons/undo-2";
import { default as default1372 } from "../icons/underline";
import { default as default1373 } from "../icons/undo-dot";
import { default as default1374 } from "../icons/undo";
import { default as default1375 } from "../icons/unfold-horizontal";
import { default as default1376 } from "../icons/unfold-vertical";
import { default as default1377 } from "../icons/ungroup";
import { default as default1378 } from "../icons/unlink-2";
import { default as default1379 } from "../icons/unlink";
import { default as default1380 } from "../icons/unplug";
import { default as default1381 } from "../icons/upload";
import { default as default1382 } from "../icons/usb";
import { default as default1383 } from "../icons/user-check";
import { default as default1384 } from "../icons/user-cog";
import { default as default1385 } from "../icons/user-key";
import { default as default1386 } from "../icons/user-lock";
import { default as default1387 } from "../icons/user-minus";
import { default as default1388 } from "../icons/user-pen";
import { default as default1389 } from "../icons/user-plus";
import { default as default1390 } from "../icons/user-round-key";
import { default as default1391 } from "../icons/user-round-pen";
import { default as default1392 } from "../icons/user-round-search";
import { default as default1393 } from "../icons/user-search";
import { default as default1394 } from "../icons/user-star";
import { default as default1395 } from "../icons/user-x";
import { default as default1396 } from "../icons/user";
import { default as default1397 } from "../icons/users";
import { default as default1398 } from "../icons/utility-pole";
import { default as default1399 } from "../icons/van";
import { default as default1400 } from "../icons/variable";
import { default as default1401 } from "../icons/vault";
import { default as default1402 } from "../icons/vector-square";
import { default as default1403 } from "../icons/vegan";
import { default as default1404 } from "../icons/venetian-mask";
import { default as default1405 } from "../icons/venus-and-mars";
import { default as default1406 } from "../icons/venus";
import { default as default1407 } from "../icons/vibrate-off";
import { default as default1408 } from "../icons/vibrate";
import { default as default1409 } from "../icons/video-off";
import { default as default1410 } from "../icons/video";
import { default as default1411 } from "../icons/videotape";
import { default as default1412 } from "../icons/view";
import { default as default1413 } from "../icons/voicemail";
import { default as default1414 } from "../icons/volleyball";
import { default as default1415 } from "../icons/volume-1";
import { default as default1416 } from "../icons/volume-2";
import { default as default1417 } from "../icons/volume-off";
import { default as default1418 } from "../icons/volume-x";
import { default as default1419 } from "../icons/volume";
import { default as default1420 } from "../icons/vote";
import { default as default1421 } from "../icons/wallet-cards";
import { default as default1422 } from "../icons/wallpaper";
import { default as default1423 } from "../icons/wallet";
import { default as default1424 } from "../icons/warehouse";
import { default as default1425 } from "../icons/wand";
import { default as default1426 } from "../icons/washing-machine";
import { default as default1427 } from "../icons/watch";
import { default as default1428 } from "../icons/waves-arrow-down";
import { default as default1429 } from "../icons/waves-arrow-up";
import { default as default1430 } from "../icons/waves-vertical";
import { default as default1431 } from "../icons/waves-ladder";
import { default as default1432 } from "../icons/waypoints";
import { default as default1433 } from "../icons/webcam";
import { default as default1434 } from "../icons/webhook-off";
import { default as default1435 } from "../icons/webhook";
import { default as default1436 } from "../icons/weight-tilde";
import { default as default1437 } from "../icons/weight";
import { default as default1438 } from "../icons/wheat-off";
import { default as default1439 } from "../icons/whole-word";
import { default as default1440 } from "../icons/wheat";
import { default as default1441 } from "../icons/wifi-cog";
import { default as default1442 } from "../icons/wifi-high";
import { default as default1443 } from "../icons/wifi-low";
import { default as default1444 } from "../icons/wifi-off";
import { default as default1445 } from "../icons/wifi-pen";
import { default as default1446 } from "../icons/wifi-sync";
import { default as default1447 } from "../icons/wifi-zero";
import { default as default1448 } from "../icons/wifi";
import { default as default1449 } from "../icons/wind-arrow-down";
import { default as default1450 } from "../icons/wind";
import { default as default1451 } from "../icons/wine";
import { default as default1452 } from "../icons/wine-off";
import { default as default1453 } from "../icons/workflow";
import { default as default1454 } from "../icons/worm";
import { default as default1455 } from "../icons/wrench";
import { default as default1456 } from "../icons/x-line-top";
import { default as default1457 } from "../icons/x";
import { default as default1458 } from "../icons/zap-off";
import { default as default1459 } from "../icons/zap";
import { default as default1460 } from "../icons/zodiac-aries";
import { default as default1461 } from "../icons/zodiac-aquarius";
import { default as default1462 } from "../icons/zodiac-cancer";
import { default as default1463 } from "../icons/zodiac-gemini";
import { default as default1464 } from "../icons/zodiac-capricorn";
import { default as default1465 } from "../icons/zodiac-leo";
import { default as default1466 } from "../icons/zodiac-ophiuchus";
import { default as default1467 } from "../icons/zodiac-libra";
import { default as default1468 } from "../icons/zodiac-pisces";
import { default as default1469 } from "../icons/zodiac-sagittarius";
import { default as default1470 } from "../icons/zodiac-scorpio";
import { default as default1471 } from "../icons/zodiac-taurus";
import { default as default1472 } from "../icons/zodiac-virgo";
import { default as default1473 } from "../icons/zoom-in";
import { default as default1474 } from "../icons/zoom-out";
import { default as default1475 } from "../icons/fingerprint-pattern";
import {
  default as default1476
} from "../icons/fingerprint-pattern";
import { default as default1477 } from "../icons/arrow-down-0-1";
import { default as default1478 } from "../icons/arrow-down-1-0";
import { default as default1479 } from "../icons/arrow-up-0-1";
import { default as default1480 } from "../icons/arrow-up-1-0";
import { default as default1481 } from "../icons/alarm-clock-check";
import {
  default as default1482
} from "../icons/alarm-check";
import { default as default1483 } from "../icons/alarm-clock-minus";
import {
  default as default1484
} from "../icons/alarm-minus";
import { default as default1485 } from "../icons/alarm-clock-plus";
import {
  default as default1486
} from "../icons/alarm-plus";
import { default as default1487 } from "../icons/arrow-down-a-z";
import {
  default as default1488
} from "../icons/arrow-down-az";
import { default as default1489 } from "../icons/arrow-down-z-a";
import {
  default as default1490
} from "../icons/arrow-down-za";
import { default as default1491 } from "../icons/arrow-down-wide-narrow";
import {
  default as default1492
} from "../icons/sort-desc";
import { default as default1493 } from "../icons/arrow-up-a-z";
import {
  default as default1494
} from "../icons/arrow-up-az";
import { default as default1495 } from "../icons/arrow-up-narrow-wide";
import {
  default as default1496
} from "../icons/sort-asc";
import { default as default1497 } from "../icons/arrow-up-z-a";
import {
  default as default1498
} from "../icons/arrow-up-za";
import { default as default1499 } from "../icons/axis-3d";
import {
  default as default1500
} from "../icons/axis-3-d";
import { default as default1501 } from "../icons/badge-check";
import {
  default as default1502
} from "../icons/verified";
import { default as default1503 } from "../icons/badge-question-mark";
import {
  default as default1504
} from "../icons/badge-help";
import { default as default1505 } from "../icons/between-horizontal-end";
import {
  default as default1506
} from "../icons/between-horizonal-end";
import { default as default1507 } from "../icons/between-horizontal-start";
import {
  default as default1508
} from "../icons/between-horizonal-start";
import { default as default1509 } from "../icons/book-dashed";
import {
  default as default1510
} from "../icons/book-template";
import { default as default1511 } from "../icons/braces";
import {
  default as default1512
} from "../icons/curly-braces";
import { default as default1513 } from "../icons/captions";
import {
  default as default1514
} from "../icons/subtitles";
import { default as default1515 } from "../icons/chart-area";
import {
  default as default1516
} from "../icons/area-chart";
import { default as default1517 } from "../icons/chart-bar-big";
import {
  default as default1518
} from "../icons/bar-chart-horizontal-big";
import { default as default1519 } from "../icons/chart-bar";
import {
  default as default1520
} from "../icons/bar-chart-horizontal";
import { default as default1521 } from "../icons/chart-column-big";
import {
  default as default1522
} from "../icons/bar-chart-big";
import { default as default1523 } from "../icons/chart-candlestick";
import {
  default as default1524
} from "../icons/candlestick-chart";
import { default as default1525 } from "../icons/chart-column-increasing";
import {
  default as default1526
} from "../icons/bar-chart-4";
import { default as default1527 } from "../icons/chart-column";
import {
  default as default1528
} from "../icons/bar-chart-3";
import { default as default1529 } from "../icons/chart-line";
import {
  default as default1530
} from "../icons/line-chart";
import { default as default1531 } from "../icons/chart-no-axes-column-increasing";
import {
  default as default1532
} from "../icons/bar-chart";
import { default as default1533 } from "../icons/chart-no-axes-column";
import {
  default as default1534
} from "../icons/bar-chart-2";
import { default as default1535 } from "../icons/chart-no-axes-gantt";
import {
  default as default1536
} from "../icons/gantt-chart";
import { default as default1537 } from "../icons/chart-pie";
import {
  default as default1538
} from "../icons/pie-chart";
import { default as default1539 } from "../icons/chart-scatter";
import {
  default as default1540
} from "../icons/scatter-chart";
import { default as default1541 } from "../icons/circle-alert";
import {
  default as default1542
} from "../icons/alert-circle";
import { default as default1543 } from "../icons/circle-arrow-down";
import {
  default as default1544
} from "../icons/arrow-down-circle";
import { default as default1545 } from "../icons/circle-arrow-left";
import {
  default as default1546
} from "../icons/arrow-left-circle";
import { default as default1547 } from "../icons/circle-arrow-out-down-left";
import {
  default as default1548
} from "../icons/arrow-down-left-from-circle";
import { default as default1549 } from "../icons/circle-arrow-out-down-right";
import {
  default as default1550
} from "../icons/arrow-down-right-from-circle";
import { default as default1551 } from "../icons/circle-arrow-out-up-left";
import {
  default as default1552
} from "../icons/arrow-up-left-from-circle";
import { default as default1553 } from "../icons/circle-arrow-out-up-right";
import {
  default as default1554
} from "../icons/arrow-up-right-from-circle";
import { default as default1555 } from "../icons/circle-arrow-right";
import {
  default as default1556
} from "../icons/arrow-right-circle";
import { default as default1557 } from "../icons/circle-arrow-up";
import {
  default as default1558
} from "../icons/arrow-up-circle";
import { default as default1559 } from "../icons/circle-check-big";
import {
  default as default1560
} from "../icons/check-circle";
import { default as default1561 } from "../icons/circle-check";
import {
  default as default1562
} from "../icons/check-circle-2";
import { default as default1563 } from "../icons/circle-chevron-down";
import {
  default as default1564
} from "../icons/chevron-down-circle";
import { default as default1565 } from "../icons/circle-chevron-left";
import {
  default as default1566
} from "../icons/chevron-left-circle";
import { default as default1567 } from "../icons/circle-chevron-right";
import {
  default as default1568
} from "../icons/chevron-right-circle";
import { default as default1569 } from "../icons/circle-chevron-up";
import {
  default as default1570
} from "../icons/chevron-up-circle";
import { default as default1571 } from "../icons/circle-divide";
import {
  default as default1572
} from "../icons/divide-circle";
import { default as default1573 } from "../icons/circle-minus";
import {
  default as default1574
} from "../icons/minus-circle";
import { default as default1575 } from "../icons/circle-parking-off";
import {
  default as default1576
} from "../icons/parking-circle-off";
import { default as default1577 } from "../icons/circle-gauge";
import {
  default as default1578
} from "../icons/gauge-circle";
import { default as default1579 } from "../icons/circle-pause";
import {
  default as default1580
} from "../icons/pause-circle";
import { default as default1581 } from "../icons/circle-percent";
import {
  default as default1582
} from "../icons/percent-circle";
import { default as default1583 } from "../icons/circle-play";
import {
  default as default1584
} from "../icons/play-circle";
import { default as default1585 } from "../icons/circle-parking";
import {
  default as default1586
} from "../icons/parking-circle";
import { default as default1587 } from "../icons/circle-plus";
import {
  default as default1588
} from "../icons/plus-circle";
import { default as default1589 } from "../icons/circle-power";
import {
  default as default1590
} from "../icons/power-circle";
import { default as default1591 } from "../icons/circle-question-mark";
import {
  default as default1592
} from "../icons/help-circle";
import {
  default as default1593
} from "../icons/circle-help";
import { default as default1594 } from "../icons/circle-slash-2";
import {
  default as default1595
} from "../icons/circle-slashed";
import { default as default1596 } from "../icons/circle-stop";
import {
  default as default1597
} from "../icons/stop-circle";
import { default as default1598 } from "../icons/circle-user-round";
import {
  default as default1599
} from "../icons/user-circle-2";
import { default as default1600 } from "../icons/circle-user";
import {
  default as default1601
} from "../icons/user-circle";
import { default as default1602 } from "../icons/circle-x";
import {
  default as default1603
} from "../icons/x-circle";
import { default as default1604 } from "../icons/clipboard-pen-line";
import {
  default as default1605
} from "../icons/clipboard-signature";
import { default as default1606 } from "../icons/clipboard-pen";
import {
  default as default1607
} from "../icons/clipboard-edit";
import { default as default1608 } from "../icons/cloud-download";
import {
  default as default1609
} from "../icons/download-cloud";
import { default as default1610 } from "../icons/cloud-upload";
import {
  default as default1611
} from "../icons/upload-cloud";
import { default as default1612 } from "../icons/code-xml";
import {
  default as default1613
} from "../icons/code-2";
import { default as default1614 } from "../icons/columns-2";
import {
  default as default1615
} from "../icons/columns";
import { default as default1616 } from "../icons/columns-3-cog";
import {
  default as default1617
} from "../icons/columns-settings";
import {
  default as default1618
} from "../icons/table-config";
import { default as default1619 } from "../icons/columns-3";
import {
  default as default1620
} from "../icons/panels-left-right";
import { default as default1621 } from "../icons/contact-round";
import {
  default as default1622
} from "../icons/contact-2";
import { default as default1623 } from "../icons/diamond-percent";
import {
  default as default1624
} from "../icons/percent-diamond";
import { default as default1625 } from "../icons/earth";
import {
  default as default1626
} from "../icons/globe-2";
import { default as default1627 } from "../icons/ellipsis-vertical";
import {
  default as default1628
} from "../icons/more-vertical";
import { default as default1629 } from "../icons/ellipsis";
import {
  default as default1630
} from "../icons/more-horizontal";
import { default as default1631 } from "../icons/file-axis-3d";
import {
  default as default1632
} from "../icons/file-axis-3-d";
import { default as default1633 } from "../icons/file-badge";
import {
  default as default1634
} from "../icons/file-badge-2";
import { default as default1635 } from "../icons/file-braces-corner";
import {
  default as default1636
} from "../icons/file-json-2";
import { default as default1637 } from "../icons/file-braces";
import {
  default as default1638
} from "../icons/file-json";
import { default as default1639 } from "../icons/file-chart-column-increasing";
import {
  default as default1640
} from "../icons/file-bar-chart";
import { default as default1641 } from "../icons/file-chart-line";
import {
  default as default1642
} from "../icons/file-line-chart";
import { default as default1643 } from "../icons/file-chart-column";
import {
  default as default1644
} from "../icons/file-bar-chart-2";
import { default as default1645 } from "../icons/file-chart-pie";
import {
  default as default1646
} from "../icons/file-pie-chart";
import { default as default1647 } from "../icons/file-check-corner";
import {
  default as default1648
} from "../icons/file-check-2";
import { default as default1649 } from "../icons/file-code-corner";
import {
  default as default1650
} from "../icons/file-code-2";
import { default as default1651 } from "../icons/file-cog";
import {
  default as default1652
} from "../icons/file-cog-2";
import { default as default1653 } from "../icons/file-exclamation-point";
import {
  default as default1654
} from "../icons/file-warning";
import { default as default1655 } from "../icons/file-headphone";
import {
  default as default1656
} from "../icons/file-audio";
import {
  default as default1657
} from "../icons/file-audio-2";
import { default as default1658 } from "../icons/file-lock";
import {
  default as default1659
} from "../icons/file-lock-2";
import { default as default1660 } from "../icons/file-key";
import {
  default as default1661
} from "../icons/file-key-2";
import { default as default1662 } from "../icons/file-minus-corner";
import {
  default as default1663
} from "../icons/file-minus-2";
import { default as default1664 } from "../icons/file-pen";
import {
  default as default1665
} from "../icons/file-edit";
import { default as default1666 } from "../icons/file-pen-line";
import {
  default as default1667
} from "../icons/file-signature";
import { default as default1668 } from "../icons/file-play";
import {
  default as default1669
} from "../icons/file-video";
import { default as default1670 } from "../icons/file-plus-corner";
import {
  default as default1671
} from "../icons/file-plus-2";
import { default as default1672 } from "../icons/file-question-mark";
import {
  default as default1673
} from "../icons/file-question";
import { default as default1674 } from "../icons/file-search-corner";
import {
  default as default1675
} from "../icons/file-search-2";
import { default as default1676 } from "../icons/file-signal";
import {
  default as default1677
} from "../icons/file-volume-2";
import { default as default1678 } from "../icons/file-type-corner";
import {
  default as default1679
} from "../icons/file-type-2";
import { default as default1680 } from "../icons/file-video-camera";
import {
  default as default1681
} from "../icons/file-video-2";
import { default as default1682 } from "../icons/file-x-corner";
import {
  default as default1683
} from "../icons/file-x-2";
import { default as default1684 } from "../icons/folder-cog";
import {
  default as default1685
} from "../icons/folder-cog-2";
import { default as default1686 } from "../icons/folder-pen";
import {
  default as default1687
} from "../icons/folder-edit";
import { default as default1688 } from "../icons/funnel-x";
import {
  default as default1689
} from "../icons/filter-x";
import { default as default1690 } from "../icons/funnel";
import {
  default as default1691
} from "../icons/filter";
import { default as default1692 } from "../icons/git-commit-horizontal";
import {
  default as default1693
} from "../icons/git-commit";
import { default as default1694 } from "../icons/grid-2x2-check";
import {
  default as default1695
} from "../icons/grid-2-x-2-check";
import { default as default1696 } from "../icons/grid-2x2-plus";
import {
  default as default1697
} from "../icons/grid-2-x-2-plus";
import { default as default1698 } from "../icons/grid-2x2-x";
import {
  default as default1699
} from "../icons/grid-2-x-2-x";
import { default as default1700 } from "../icons/grid-2x2";
import {
  default as default1701
} from "../icons/grid-2-x-2";
import { default as default1702 } from "../icons/grid-3x3";
import {
  default as default1703
} from "../icons/grid";
import {
  default as default1704
} from "../icons/grid-3-x-3";
import { default as default1705 } from "../icons/hand-grab";
import {
  default as default1706
} from "../icons/grab";
import { default as default1707 } from "../icons/hand-helping";
import {
  default as default1708
} from "../icons/helping-hand";
import { default as default1709 } from "../icons/house";
import {
  default as default1710
} from "../icons/home";
import { default as default1711 } from "../icons/ice-cream-bowl";
import {
  default as default1712
} from "../icons/ice-cream-2";
import { default as default1713 } from "../icons/ice-cream-cone";
import {
  default as default1714
} from "../icons/ice-cream";
import { default as default1715 } from "../icons/laptop-minimal";
import {
  default as default1716
} from "../icons/laptop-2";
import { default as default1717 } from "../icons/layers";
import {
  default as default1718
} from "../icons/layers-3";
import { default as default1719 } from "../icons/list-indent-decrease";
import {
  default as default1720
} from "../icons/outdent";
import {
  default as default1721
} from "../icons/indent-decrease";
import { default as default1722 } from "../icons/list-indent-increase";
import {
  default as default1723
} from "../icons/indent";
import {
  default as default1724
} from "../icons/indent-increase";
import { default as default1725 } from "../icons/loader-circle";
import {
  default as default1726
} from "../icons/loader-2";
import { default as default1727 } from "../icons/lock-open";
import {
  default as default1728
} from "../icons/unlock";
import { default as default1729 } from "../icons/lock-keyhole-open";
import {
  default as default1730
} from "../icons/unlock-keyhole";
import { default as default1731 } from "../icons/mail-question-mark";
import {
  default as default1732
} from "../icons/mail-question";
import { default as default1733 } from "../icons/map-pin-pen";
import {
  default as default1734
} from "../icons/location-edit";
import { default as default1735 } from "../icons/message-circle-question-mark";
import {
  default as default1736
} from "../icons/message-circle-question";
import { default as default1737 } from "../icons/mic-vocal";
import {
  default as default1738
} from "../icons/mic-2";
import { default as default1739 } from "../icons/move-3d";
import {
  default as default1740
} from "../icons/move-3-d";
import { default as default1741 } from "../icons/octagon-alert";
import {
  default as default1742
} from "../icons/alert-octagon";
import { default as default1743 } from "../icons/octagon-pause";
import {
  default as default1744
} from "../icons/pause-octagon";
import { default as default1745 } from "../icons/octagon-x";
import {
  default as default1746
} from "../icons/x-octagon";
import { default as default1747 } from "../icons/paintbrush-vertical";
import {
  default as default1748
} from "../icons/paintbrush-2";
import { default as default1749 } from "../icons/panel-left-close";
import {
  default as default1750
} from "../icons/sidebar-close";
import { default as default1751 } from "../icons/panel-bottom-dashed";
import {
  default as default1752
} from "../icons/panel-bottom-inactive";
import { default as default1753 } from "../icons/panel-left-dashed";
import {
  default as default1754
} from "../icons/panel-left-inactive";
import { default as default1755 } from "../icons/panel-left-open";
import {
  default as default1756
} from "../icons/sidebar-open";
import { default as default1757 } from "../icons/panel-left";
import {
  default as default1758
} from "../icons/sidebar";
import { default as default1759 } from "../icons/panel-right-dashed";
import {
  default as default1760
} from "../icons/panel-right-inactive";
import { default as default1761 } from "../icons/panel-top-dashed";
import {
  default as default1762
} from "../icons/panel-top-inactive";
import { default as default1763 } from "../icons/panels-top-left";
import {
  default as default1764
} from "../icons/layout";
import { default as default1765 } from "../icons/pen-line";
import {
  default as default1766
} from "../icons/edit-3";
import { default as default1767 } from "../icons/pen";
import {
  default as default1768
} from "../icons/edit-2";
import { default as default1769 } from "../icons/plug-zap";
import {
  default as default1770
} from "../icons/plug-zap-2";
import { default as default1771 } from "../icons/rectangle-ellipsis";
import {
  default as default1772
} from "../icons/form-input";
import { default as default1773 } from "../icons/rotate-3d";
import {
  default as default1774
} from "../icons/rotate-3-d";
import { default as default1775 } from "../icons/rows-2";
import {
  default as default1776
} from "../icons/rows";
import { default as default1777 } from "../icons/rows-3";
import {
  default as default1778
} from "../icons/panels-top-bottom";
import { default as default1779 } from "../icons/scale-3d";
import {
  default as default1780
} from "../icons/scale-3-d";
import { default as default1781 } from "../icons/send-horizontal";
import {
  default as default1782
} from "../icons/send-horizonal";
import { default as default1783 } from "../icons/shield-question-mark";
import {
  default as default1784
} from "../icons/shield-question";
import { default as default1785 } from "../icons/shield-x";
import {
  default as default1786
} from "../icons/shield-close";
import { default as default1787 } from "../icons/sliders-vertical";
import {
  default as default1788
} from "../icons/sliders";
import { default as default1789 } from "../icons/sparkles";
import {
  default as default1790
} from "../icons/stars";
import { default as default1791 } from "../icons/square-activity";
import {
  default as default1792
} from "../icons/activity-square";
import { default as default1793 } from "../icons/square-arrow-down-left";
import {
  default as default1794
} from "../icons/arrow-down-left-square";
import { default as default1795 } from "../icons/square-arrow-down-right";
import {
  default as default1796
} from "../icons/arrow-down-right-square";
import { default as default1797 } from "../icons/square-arrow-down";
import {
  default as default1798
} from "../icons/arrow-down-square";
import { default as default1799 } from "../icons/square-arrow-left";
import {
  default as default1800
} from "../icons/arrow-left-square";
import { default as default1801 } from "../icons/square-arrow-out-down-right";
import {
  default as default1802
} from "../icons/arrow-down-right-from-square";
import { default as default1803 } from "../icons/square-arrow-out-down-left";
import {
  default as default1804
} from "../icons/arrow-down-left-from-square";
import { default as default1805 } from "../icons/square-arrow-out-up-left";
import {
  default as default1806
} from "../icons/arrow-up-left-from-square";
import { default as default1807 } from "../icons/square-arrow-out-up-right";
import {
  default as default1808
} from "../icons/arrow-up-right-from-square";
import { default as default1809 } from "../icons/square-arrow-right";
import {
  default as default1810
} from "../icons/arrow-right-square";
import { default as default1811 } from "../icons/square-arrow-up-left";
import {
  default as default1812
} from "../icons/arrow-up-left-square";
import { default as default1813 } from "../icons/square-arrow-up-right";
import {
  default as default1814
} from "../icons/arrow-up-right-square";
import { default as default1815 } from "../icons/square-arrow-up";
import {
  default as default1816
} from "../icons/arrow-up-square";
import { default as default1817 } from "../icons/square-asterisk";
import {
  default as default1818
} from "../icons/asterisk-square";
import { default as default1819 } from "../icons/square-bottom-dashed-scissors";
import {
  default as default1820
} from "../icons/scissors-square-dashed-bottom";
import { default as default1821 } from "../icons/square-centerline-dashed-vertical";
import {
  default as default1822
} from "../icons/flip-vertical";
import { default as default1823 } from "../icons/square-centerline-dashed-horizontal";
import {
  default as default1824
} from "../icons/flip-horizontal";
import { default as default1825 } from "../icons/square-chart-gantt";
import {
  default as default1826
} from "../icons/gantt-chart-square";
import {
  default as default1827
} from "../icons/square-gantt-chart";
import { default as default1828 } from "../icons/square-check-big";
import {
  default as default1829
} from "../icons/check-square";
import { default as default1830 } from "../icons/square-chevron-left";
import {
  default as default1831
} from "../icons/chevron-left-square";
import { default as default1832 } from "../icons/square-check";
import {
  default as default1833
} from "../icons/check-square-2";
import { default as default1834 } from "../icons/square-chevron-right";
import {
  default as default1835
} from "../icons/chevron-right-square";
import { default as default1836 } from "../icons/square-chevron-down";
import {
  default as default1837
} from "../icons/chevron-down-square";
import { default as default1838 } from "../icons/square-chevron-up";
import {
  default as default1839
} from "../icons/chevron-up-square";
import { default as default1840 } from "../icons/square-code";
import {
  default as default1841
} from "../icons/code-square";
import { default as default1842 } from "../icons/square-dashed-kanban";
import {
  default as default1843
} from "../icons/kanban-square-dashed";
import { default as default1844 } from "../icons/square-dashed-mouse-pointer";
import {
  default as default1845
} from "../icons/mouse-pointer-square-dashed";
import { default as default1846 } from "../icons/square-dashed-text";
import {
  default as default1847
} from "../icons/text-selection";
import {
  default as default1848
} from "../icons/text-select";
import { default as default1849 } from "../icons/square-dashed";
import {
  default as default1850
} from "../icons/box-select";
import { default as default1851 } from "../icons/square-divide";
import {
  default as default1852
} from "../icons/divide-square";
import { default as default1853 } from "../icons/square-dot";
import {
  default as default1854
} from "../icons/dot-square";
import { default as default1855 } from "../icons/square-equal";
import {
  default as default1856
} from "../icons/equal-square";
import { default as default1857 } from "../icons/square-function";
import {
  default as default1858
} from "../icons/function-square";
import { default as default1859 } from "../icons/square-library";
import {
  default as default1860
} from "../icons/library-square";
import { default as default1861 } from "../icons/square-kanban";
import {
  default as default1862
} from "../icons/kanban-square";
import { default as default1863 } from "../icons/square-m";
import {
  default as default1864
} from "../icons/m-square";
import { default as default1865 } from "../icons/square-menu";
import {
  default as default1866
} from "../icons/menu-square";
import { default as default1867 } from "../icons/square-minus";
import {
  default as default1868
} from "../icons/minus-square";
import { default as default1869 } from "../icons/square-mouse-pointer";
import {
  default as default1870
} from "../icons/inspect";
import { default as default1871 } from "../icons/square-parking-off";
import {
  default as default1872
} from "../icons/parking-square-off";
import { default as default1873 } from "../icons/square-parking";
import {
  default as default1874
} from "../icons/parking-square";
import { default as default1875 } from "../icons/square-pen";
import {
  default as default1876
} from "../icons/edit";
import {
  default as default1877
} from "../icons/pen-box";
import {
  default as default1878
} from "../icons/pen-square";
import { default as default1879 } from "../icons/square-percent";
import {
  default as default1880
} from "../icons/percent-square";
import { default as default1881 } from "../icons/square-play";
import {
  default as default1882
} from "../icons/play-square";
import { default as default1883 } from "../icons/square-pi";
import {
  default as default1884
} from "../icons/pi-square";
import { default as default1885 } from "../icons/square-pilcrow";
import {
  default as default1886
} from "../icons/pilcrow-square";
import { default as default1887 } from "../icons/square-plus";
import {
  default as default1888
} from "../icons/plus-square";
import { default as default1889 } from "../icons/square-power";
import {
  default as default1890
} from "../icons/power-square";
import { default as default1891 } from "../icons/square-scissors";
import {
  default as default1892
} from "../icons/scissors-square";
import { default as default1893 } from "../icons/square-sigma";
import {
  default as default1894
} from "../icons/sigma-square";
import { default as default1895 } from "../icons/square-slash";
import {
  default as default1896
} from "../icons/slash-square";
import { default as default1897 } from "../icons/square-split-horizontal";
import {
  default as default1898
} from "../icons/split-square-horizontal";
import { default as default1899 } from "../icons/square-split-vertical";
import {
  default as default1900
} from "../icons/split-square-vertical";
import { default as default1901 } from "../icons/square-terminal";
import {
  default as default1902
} from "../icons/terminal-square";
import { default as default1903 } from "../icons/square-user-round";
import {
  default as default1904
} from "../icons/user-square-2";
import { default as default1905 } from "../icons/square-x";
import {
  default as default1906
} from "../icons/x-square";
import { default as default1907 } from "../icons/square-user";
import {
  default as default1908
} from "../icons/user-square";
import { default as default1909 } from "../icons/test-tube-diagonal";
import {
  default as default1910
} from "../icons/test-tube-2";
import { default as default1911 } from "../icons/text-align-center";
import {
  default as default1912
} from "../icons/align-center";
import { default as default1913 } from "../icons/text-align-justify";
import {
  default as default1914
} from "../icons/align-justify";
import { default as default1915 } from "../icons/text-align-end";
import {
  default as default1916
} from "../icons/align-right";
import { default as default1917 } from "../icons/text-align-start";
import {
  default as default1918
} from "../icons/align-left";
import {
  default as default1919
} from "../icons/text";
import { default as default1920 } from "../icons/text-initial";
import {
  default as default1921
} from "../icons/letter-text";
import { default as default1922 } from "../icons/text-wrap";
import {
  default as default1923
} from "../icons/wrap-text";
import { default as default1924 } from "../icons/tram-front";
import {
  default as default1925
} from "../icons/train";
import { default as default1926 } from "../icons/tree-palm";
import {
  default as default1927
} from "../icons/palmtree";
import { default as default1928 } from "../icons/triangle-alert";
import {
  default as default1929
} from "../icons/alert-triangle";
import { default as default1930 } from "../icons/tv-minimal";
import {
  default as default1931
} from "../icons/tv-2";
import { default as default1932 } from "../icons/university";
import {
  default as default1933
} from "../icons/school-2";
import { default as default1934 } from "../icons/user-round-check";
import {
  default as default1935
} from "../icons/user-check-2";
import { default as default1936 } from "../icons/user-round-cog";
import {
  default as default1937
} from "../icons/user-cog-2";
import { default as default1938 } from "../icons/user-round-plus";
import {
  default as default1939
} from "../icons/user-plus-2";
import { default as default1940 } from "../icons/user-round-minus";
import {
  default as default1941
} from "../icons/user-minus-2";
import { default as default1942 } from "../icons/user-round-x";
import {
  default as default1943
} from "../icons/user-x-2";
import { default as default1944 } from "../icons/user-round";
import {
  default as default1945
} from "../icons/user-2";
import { default as default1946 } from "../icons/utensils-crossed";
import {
  default as default1947
} from "../icons/fork-knife-crossed";
import { default as default1948 } from "../icons/users-round";
import {
  default as default1949
} from "../icons/users-2";
import { default as default1950 } from "../icons/utensils";
import {
  default as default1951
} from "../icons/fork-knife";
import { default as default1952 } from "../icons/wallet-minimal";
import {
  default as default1953
} from "../icons/wallet-2";
import { default as default1954 } from "../icons/wand-sparkles";
import {
  default as default1955
} from "../icons/wand-2";
import { default as default1956 } from "../icons/waves-horizontal";
import {
  default as default1957
} from "../icons/waves";
export {
  default2 as LucideAArrowDown,
  default3 as LucideAArrowUp,
  default4 as LucideALargeSmall,
  default6 as LucideAccessibility,
  default5 as LucideActivity,
  default1792 as LucideActivitySquare,
  default7 as LucideAirVent,
  default8 as LucideAirplay,
  default1482 as LucideAlarmCheck,
  default10 as LucideAlarmClock,
  default1481 as LucideAlarmClockCheck,
  default1483 as LucideAlarmClockMinus,
  default9 as LucideAlarmClockOff,
  default1485 as LucideAlarmClockPlus,
  default1484 as LucideAlarmMinus,
  default1486 as LucideAlarmPlus,
  default11 as LucideAlarmSmoke,
  default12 as LucideAlbum,
  default1542 as LucideAlertCircle,
  default1742 as LucideAlertOctagon,
  default1929 as LucideAlertTriangle,
  default1912 as LucideAlignCenter,
  default13 as LucideAlignCenterHorizontal,
  default14 as LucideAlignCenterVertical,
  default15 as LucideAlignEndHorizontal,
  default16 as LucideAlignEndVertical,
  default17 as LucideAlignHorizontalDistributeCenter,
  default18 as LucideAlignHorizontalDistributeEnd,
  default19 as LucideAlignHorizontalDistributeStart,
  default20 as LucideAlignHorizontalJustifyCenter,
  default21 as LucideAlignHorizontalJustifyEnd,
  default22 as LucideAlignHorizontalJustifyStart,
  default23 as LucideAlignHorizontalSpaceAround,
  default24 as LucideAlignHorizontalSpaceBetween,
  default1914 as LucideAlignJustify,
  default1918 as LucideAlignLeft,
  default1916 as LucideAlignRight,
  default25 as LucideAlignStartHorizontal,
  default28 as LucideAlignStartVertical,
  default26 as LucideAlignVerticalDistributeCenter,
  default27 as LucideAlignVerticalDistributeEnd,
  default29 as LucideAlignVerticalDistributeStart,
  default30 as LucideAlignVerticalJustifyCenter,
  default32 as LucideAlignVerticalJustifyEnd,
  default31 as LucideAlignVerticalJustifyStart,
  default33 as LucideAlignVerticalSpaceAround,
  default34 as LucideAlignVerticalSpaceBetween,
  default35 as LucideAmbulance,
  default36 as LucideAmpersand,
  default37 as LucideAmpersands,
  default38 as LucideAmphora,
  default39 as LucideAnchor,
  default40 as LucideAngry,
  default41 as LucideAnnoyed,
  default42 as LucideAntenna,
  default43 as LucideAnvil,
  default44 as LucideAperture,
  default46 as LucideAppWindow,
  default45 as LucideAppWindowMac,
  default47 as LucideApple,
  default50 as LucideArchive,
  default49 as LucideArchiveRestore,
  default48 as LucideArchiveX,
  default1516 as LucideAreaChart,
  default51 as LucideArmchair,
  default53 as LucideArrowBigDown,
  default52 as LucideArrowBigDownDash,
  default55 as LucideArrowBigLeft,
  default54 as LucideArrowBigLeftDash,
  default57 as LucideArrowBigRight,
  default56 as LucideArrowBigRightDash,
  default59 as LucideArrowBigUp,
  default58 as LucideArrowBigUpDash,
  default67 as LucideArrowDown,
  default1477 as LucideArrowDown01,
  default1478 as LucideArrowDown10,
  default1487 as LucideArrowDownAZ,
  default1488 as LucideArrowDownAz,
  default1544 as LucideArrowDownCircle,
  default60 as LucideArrowDownFromLine,
  default61 as LucideArrowDownLeft,
  default1548 as LucideArrowDownLeftFromCircle,
  default1804 as LucideArrowDownLeftFromSquare,
  default1794 as LucideArrowDownLeftSquare,
  default62 as LucideArrowDownNarrowWide,
  default63 as LucideArrowDownRight,
  default1550 as LucideArrowDownRightFromCircle,
  default1802 as LucideArrowDownRightFromSquare,
  default1796 as LucideArrowDownRightSquare,
  default1798 as LucideArrowDownSquare,
  default64 as LucideArrowDownToDot,
  default66 as LucideArrowDownToLine,
  default65 as LucideArrowDownUp,
  default1491 as LucideArrowDownWideNarrow,
  default1489 as LucideArrowDownZA,
  default1490 as LucideArrowDownZa,
  default71 as LucideArrowLeft,
  default1546 as LucideArrowLeftCircle,
  default68 as LucideArrowLeftFromLine,
  default70 as LucideArrowLeftRight,
  default1800 as LucideArrowLeftSquare,
  default69 as LucideArrowLeftToLine,
  default75 as LucideArrowRight,
  default1556 as LucideArrowRightCircle,
  default72 as LucideArrowRightFromLine,
  default73 as LucideArrowRightLeft,
  default1810 as LucideArrowRightSquare,
  default74 as LucideArrowRightToLine,
  default83 as LucideArrowUp,
  default1479 as LucideArrowUp01,
  default1480 as LucideArrowUp10,
  default1493 as LucideArrowUpAZ,
  default1494 as LucideArrowUpAz,
  default1558 as LucideArrowUpCircle,
  default76 as LucideArrowUpDown,
  default77 as LucideArrowUpFromDot,
  default79 as LucideArrowUpFromLine,
  default78 as LucideArrowUpLeft,
  default1552 as LucideArrowUpLeftFromCircle,
  default1806 as LucideArrowUpLeftFromSquare,
  default1812 as LucideArrowUpLeftSquare,
  default1495 as LucideArrowUpNarrowWide,
  default80 as LucideArrowUpRight,
  default1554 as LucideArrowUpRightFromCircle,
  default1808 as LucideArrowUpRightFromSquare,
  default1814 as LucideArrowUpRightSquare,
  default1816 as LucideArrowUpSquare,
  default81 as LucideArrowUpToLine,
  default82 as LucideArrowUpWideNarrow,
  default1497 as LucideArrowUpZA,
  default1498 as LucideArrowUpZa,
  default84 as LucideArrowsUpFromLine,
  default85 as LucideAsterisk,
  default1818 as LucideAsteriskSquare,
  default86 as LucideAstroid,
  default87 as LucideAtSign,
  default88 as LucideAtom,
  default89 as LucideAudioLines,
  default90 as LucideAudioWaveform,
  default91 as LucideAward,
  default92 as LucideAxe,
  default1500 as LucideAxis3D,
  default1499 as LucideAxis3d,
  default93 as LucideBaby,
  default94 as LucideBackpack,
  default110 as LucideBadge,
  default96 as LucideBadgeAlert,
  default95 as LucideBadgeCent,
  default1501 as LucideBadgeCheck,
  default97 as LucideBadgeDollarSign,
  default98 as LucideBadgeEuro,
  default1504 as LucideBadgeHelp,
  default99 as LucideBadgeIndianRupee,
  default100 as LucideBadgeInfo,
  default102 as LucideBadgeJapaneseYen,
  default101 as LucideBadgeMinus,
  default103 as LucideBadgePercent,
  default104 as LucideBadgePlus,
  default105 as LucideBadgePoundSterling,
  default1503 as LucideBadgeQuestionMark,
  default106 as LucideBadgeRussianRuble,
  default107 as LucideBadgeSwissFranc,
  default109 as LucideBadgeTurkishLira,
  default108 as LucideBadgeX,
  default111 as LucideBaggageClaim,
  default112 as LucideBalloon,
  default113 as LucideBan,
  default114 as LucideBanana,
  default115 as LucideBandage,
  default119 as LucideBanknote,
  default116 as LucideBanknoteArrowDown,
  default117 as LucideBanknoteArrowUp,
  default118 as LucideBanknoteX,
  default1532 as LucideBarChart,
  default1534 as LucideBarChart2,
  default1528 as LucideBarChart3,
  default1526 as LucideBarChart4,
  default1522 as LucideBarChartBig,
  default1520 as LucideBarChartHorizontal,
  default1518 as LucideBarChartHorizontalBig,
  default120 as LucideBarcode,
  default121 as LucideBarrel,
  default122 as LucideBaseline,
  default123 as LucideBath,
  default130 as LucideBattery,
  default124 as LucideBatteryCharging,
  default126 as LucideBatteryFull,
  default125 as LucideBatteryLow,
  default127 as LucideBatteryMedium,
  default128 as LucideBatteryPlus,
  default129 as LucideBatteryWarning,
  default131 as LucideBeaker,
  default133 as LucideBean,
  default132 as LucideBeanOff,
  default136 as LucideBed,
  default134 as LucideBedDouble,
  default135 as LucideBedSingle,
  default138 as LucideBeef,
  default137 as LucideBeefOff,
  default140 as LucideBeer,
  default139 as LucideBeerOff,
  default148 as LucideBell,
  default141 as LucideBellCheck,
  default142 as LucideBellDot,
  default143 as LucideBellElectric,
  default144 as LucideBellMinus,
  default145 as LucideBellOff,
  default146 as LucideBellPlus,
  default147 as LucideBellRing,
  default1506 as LucideBetweenHorizonalEnd,
  default1508 as LucideBetweenHorizonalStart,
  default1505 as LucideBetweenHorizontalEnd,
  default1507 as LucideBetweenHorizontalStart,
  default149 as LucideBetweenVerticalEnd,
  default150 as LucideBetweenVerticalStart,
  default151 as LucideBicepsFlexed,
  default152 as LucideBike,
  default153 as LucideBinary,
  default154 as LucideBinoculars,
  default155 as LucideBiohazard,
  default156 as LucideBird,
  default157 as LucideBirdhouse,
  default158 as LucideBitcoin,
  default159 as LucideBlend,
  default160 as LucideBlender,
  default161 as LucideBlinds,
  default162 as LucideBlocks,
  default166 as LucideBluetooth,
  default163 as LucideBluetoothConnected,
  default165 as LucideBluetoothOff,
  default164 as LucideBluetoothSearching,
  default167 as LucideBold,
  default168 as LucideBolt,
  default169 as LucideBomb,
  default170 as LucideBone,
  default195 as LucideBook,
  default171 as LucideBookA,
  default172 as LucideBookAlert,
  default173 as LucideBookAudio,
  default174 as LucideBookCheck,
  default175 as LucideBookCopy,
  default1509 as LucideBookDashed,
  default176 as LucideBookDown,
  default177 as LucideBookHeadphones,
  default179 as LucideBookHeart,
  default178 as LucideBookImage,
  default180 as LucideBookKey,
  default182 as LucideBookLock,
  default181 as LucideBookMarked,
  default183 as LucideBookMinus,
  default187 as LucideBookOpen,
  default185 as LucideBookOpenCheck,
  default184 as LucideBookOpenText,
  default186 as LucideBookPlus,
  default189 as LucideBookSearch,
  default1510 as LucideBookTemplate,
  default188 as LucideBookText,
  default190 as LucideBookType,
  default192 as LucideBookUp,
  default191 as LucideBookUp2,
  default193 as LucideBookUser,
  default194 as LucideBookX,
  default201 as LucideBookmark,
  default196 as LucideBookmarkCheck,
  default197 as LucideBookmarkMinus,
  default200 as LucideBookmarkOff,
  default198 as LucideBookmarkPlus,
  default199 as LucideBookmarkX,
  default203 as LucideBoomBox,
  default205 as LucideBot,
  default202 as LucideBotMessageSquare,
  default204 as LucideBotOff,
  default206 as LucideBottleWine,
  default207 as LucideBowArrow,
  default208 as LucideBox,
  default1850 as LucideBoxSelect,
  default209 as LucideBoxes,
  default1511 as LucideBraces,
  default210 as LucideBrackets,
  default212 as LucideBrain,
  default211 as LucideBrainCircuit,
  default214 as LucideBrainCog,
  default216 as LucideBrickWall,
  default213 as LucideBrickWallFire,
  default215 as LucideBrickWallShield,
  default220 as LucideBriefcase,
  default217 as LucideBriefcaseBusiness,
  default218 as LucideBriefcaseConveyorBelt,
  default219 as LucideBriefcaseMedical,
  default221 as LucideBringToFront,
  default222 as LucideBroccoli,
  default224 as LucideBrush,
  default223 as LucideBrushCleaning,
  default225 as LucideBubbles,
  default228 as LucideBug,
  default226 as LucideBugOff,
  default227 as LucideBugPlay,
  default230 as LucideBuilding,
  default229 as LucideBuilding2,
  default233 as LucideBus,
  default231 as LucideBusFront,
  default234 as LucideCable,
  default232 as LucideCableCar,
  default236 as LucideCake,
  default235 as LucideCakeSlice,
  default237 as LucideCalculator,
  default258 as LucideCalendar,
  default238 as LucideCalendar1,
  default239 as LucideCalendarArrowDown,
  default240 as LucideCalendarArrowUp,
  default244 as LucideCalendarCheck,
  default241 as LucideCalendarCheck2,
  default242 as LucideCalendarClock,
  default243 as LucideCalendarCog,
  default245 as LucideCalendarDays,
  default246 as LucideCalendarFold,
  default247 as LucideCalendarHeart,
  default249 as LucideCalendarMinus,
  default248 as LucideCalendarMinus2,
  default250 as LucideCalendarOff,
  default252 as LucideCalendarPlus,
  default251 as LucideCalendarPlus2,
  default253 as LucideCalendarRange,
  default254 as LucideCalendarSearch,
  default255 as LucideCalendarSync,
  default257 as LucideCalendarX,
  default256 as LucideCalendarX2,
  default259 as LucideCalendars,
  default261 as LucideCamera,
  default260 as LucideCameraOff,
  default1524 as LucideCandlestickChart,
  default265 as LucideCandy,
  default262 as LucideCandyCane,
  default263 as LucideCandyOff,
  default266 as LucideCannabis,
  default264 as LucideCannabisOff,
  default1513 as LucideCaptions,
  default269 as LucideCaptionsOff,
  default270 as LucideCar,
  default267 as LucideCarFront,
  default268 as LucideCarTaxiFront,
  default271 as LucideCaravan,
  default272 as LucideCardSim,
  default274 as LucideCarrot,
  default273 as LucideCaseLower,
  default275 as LucideCaseSensitive,
  default276 as LucideCaseUpper,
  default277 as LucideCassetteTape,
  default278 as LucideCast,
  default279 as LucideCastle,
  default280 as LucideCat,
  default282 as LucideCctv,
  default281 as LucideCctvOff,
  default1515 as LucideChartArea,
  default1519 as LucideChartBar,
  default1517 as LucideChartBarBig,
  default283 as LucideChartBarDecreasing,
  default284 as LucideChartBarIncreasing,
  default285 as LucideChartBarStacked,
  default1523 as LucideChartCandlestick,
  default1527 as LucideChartColumn,
  default1521 as LucideChartColumnBig,
  default286 as LucideChartColumnDecreasing,
  default1525 as LucideChartColumnIncreasing,
  default287 as LucideChartColumnStacked,
  default288 as LucideChartGantt,
  default1529 as LucideChartLine,
  default289 as LucideChartNetwork,
  default1533 as LucideChartNoAxesColumn,
  default290 as LucideChartNoAxesColumnDecreasing,
  default1531 as LucideChartNoAxesColumnIncreasing,
  default291 as LucideChartNoAxesCombined,
  default1535 as LucideChartNoAxesGantt,
  default1537 as LucideChartPie,
  default1539 as LucideChartScatter,
  default292 as LucideChartSpline,
  default295 as LucideCheck,
  default293 as LucideCheckCheck,
  default1560 as LucideCheckCircle,
  default1562 as LucideCheckCircle2,
  default294 as LucideCheckLine,
  default1829 as LucideCheckSquare,
  default1833 as LucideCheckSquare2,
  default296 as LucideChefHat,
  default297 as LucideCherry,
  default298 as LucideChessBishop,
  default299 as LucideChessKing,
  default300 as LucideChessKnight,
  default301 as LucideChessPawn,
  default302 as LucideChessQueen,
  default304 as LucideChessRook,
  default303 as LucideChevronDown,
  default1564 as LucideChevronDownCircle,
  default1837 as LucideChevronDownSquare,
  default305 as LucideChevronFirst,
  default306 as LucideChevronLast,
  default307 as LucideChevronLeft,
  default1566 as LucideChevronLeftCircle,
  default1831 as LucideChevronLeftSquare,
  default308 as LucideChevronRight,
  default1568 as LucideChevronRightCircle,
  default1835 as LucideChevronRightSquare,
  default309 as LucideChevronUp,
  default1570 as LucideChevronUpCircle,
  default1839 as LucideChevronUpSquare,
  default311 as LucideChevronsDown,
  default310 as LucideChevronsDownUp,
  default314 as LucideChevronsLeft,
  default313 as LucideChevronsLeftRight,
  default312 as LucideChevronsLeftRightEllipsis,
  default316 as LucideChevronsRight,
  default315 as LucideChevronsRightLeft,
  default318 as LucideChevronsUp,
  default317 as LucideChevronsUpDown,
  default319 as LucideChurch,
  default321 as LucideCigarette,
  default320 as LucideCigaretteOff,
  default336 as LucideCircle,
  default1541 as LucideCircleAlert,
  default1543 as LucideCircleArrowDown,
  default1545 as LucideCircleArrowLeft,
  default1547 as LucideCircleArrowOutDownLeft,
  default1549 as LucideCircleArrowOutDownRight,
  default1551 as LucideCircleArrowOutUpLeft,
  default1553 as LucideCircleArrowOutUpRight,
  default1555 as LucideCircleArrowRight,
  default1557 as LucideCircleArrowUp,
  default1561 as LucideCircleCheck,
  default1559 as LucideCircleCheckBig,
  default1563 as LucideCircleChevronDown,
  default1565 as LucideCircleChevronLeft,
  default1567 as LucideCircleChevronRight,
  default1569 as LucideCircleChevronUp,
  default322 as LucideCircleDashed,
  default1571 as LucideCircleDivide,
  default323 as LucideCircleDollarSign,
  default325 as LucideCircleDot,
  default324 as LucideCircleDotDashed,
  default326 as LucideCircleEllipsis,
  default327 as LucideCircleEqual,
  default328 as LucideCircleFadingArrowUp,
  default329 as LucideCircleFadingPlus,
  default1577 as LucideCircleGauge,
  default1593 as LucideCircleHelp,
  default1573 as LucideCircleMinus,
  default330 as LucideCircleOff,
  default1585 as LucideCircleParking,
  default1575 as LucideCircleParkingOff,
  default1579 as LucideCirclePause,
  default1581 as LucideCirclePercent,
  default331 as LucideCirclePile,
  default1583 as LucideCirclePlay,
  default1587 as LucideCirclePlus,
  default332 as LucideCirclePoundSterling,
  default1589 as LucideCirclePower,
  default1591 as LucideCircleQuestionMark,
  default333 as LucideCircleSlash,
  default1594 as LucideCircleSlash2,
  default1595 as LucideCircleSlashed,
  default335 as LucideCircleSmall,
  default334 as LucideCircleStar,
  default1596 as LucideCircleStop,
  default1600 as LucideCircleUser,
  default1598 as LucideCircleUserRound,
  default1602 as LucideCircleX,
  default337 as LucideCircuitBoard,
  default338 as LucideCitrus,
  default341 as LucideClapperboard,
  default349 as LucideClipboard,
  default339 as LucideClipboardCheck,
  default340 as LucideClipboardClock,
  default342 as LucideClipboardCopy,
  default1607 as LucideClipboardEdit,
  default343 as LucideClipboardList,
  default344 as LucideClipboardMinus,
  default345 as LucideClipboardPaste,
  default1606 as LucideClipboardPen,
  default1604 as LucideClipboardPenLine,
  default346 as LucideClipboardPlus,
  default1605 as LucideClipboardSignature,
  default347 as LucideClipboardType,
  default348 as LucideClipboardX,
  default369 as LucideClock,
  default350 as LucideClock1,
  default351 as LucideClock10,
  default352 as LucideClock11,
  default353 as LucideClock12,
  default354 as LucideClock2,
  default355 as LucideClock3,
  default356 as LucideClock4,
  default357 as LucideClock5,
  default358 as LucideClock6,
  default359 as LucideClock7,
  default360 as LucideClock8,
  default362 as LucideClock9,
  default361 as LucideClockAlert,
  default363 as LucideClockArrowDown,
  default364 as LucideClockArrowUp,
  default365 as LucideClockCheck,
  default366 as LucideClockFading,
  default367 as LucideClockPlus,
  default368 as LucideClosedCaption,
  default387 as LucideCloud,
  default370 as LucideCloudAlert,
  default371 as LucideCloudBackup,
  default372 as LucideCloudCheck,
  default373 as LucideCloudCog,
  default1608 as LucideCloudDownload,
  default374 as LucideCloudDrizzle,
  default375 as LucideCloudFog,
  default376 as LucideCloudHail,
  default377 as LucideCloudLightning,
  default379 as LucideCloudMoon,
  default378 as LucideCloudMoonRain,
  default380 as LucideCloudOff,
  default382 as LucideCloudRain,
  default381 as LucideCloudRainWind,
  default383 as LucideCloudSnow,
  default385 as LucideCloudSun,
  default384 as LucideCloudSunRain,
  default386 as LucideCloudSync,
  default1610 as LucideCloudUpload,
  default388 as LucideCloudy,
  default389 as LucideClover,
  default390 as LucideClub,
  default391 as LucideCode,
  default1613 as LucideCode2,
  default1841 as LucideCodeSquare,
  default1612 as LucideCodeXml,
  default392 as LucideCoffee,
  default393 as LucideCog,
  default394 as LucideCoins,
  default1615 as LucideColumns,
  default1614 as LucideColumns2,
  default1619 as LucideColumns3,
  default1616 as LucideColumns3Cog,
  default396 as LucideColumns4,
  default1617 as LucideColumnsSettings,
  default395 as LucideCombine,
  default397 as LucideCommand,
  default398 as LucideCompass,
  default399 as LucideComponent,
  default402 as LucideComputer,
  default400 as LucideConciergeBell,
  default401 as LucideCone,
  default403 as LucideConstruction,
  default404 as LucideContact,
  default1622 as LucideContact2,
  default1621 as LucideContactRound,
  default405 as LucideContainer,
  default406 as LucideContrast,
  default407 as LucideCookie,
  default408 as LucideCookingPot,
  default414 as LucideCopy,
  default409 as LucideCopyCheck,
  default410 as LucideCopyMinus,
  default411 as LucideCopyPlus,
  default413 as LucideCopySlash,
  default412 as LucideCopyX,
  default415 as LucideCopyleft,
  default416 as LucideCopyright,
  default417 as LucideCornerDownLeft,
  default418 as LucideCornerDownRight,
  default419 as LucideCornerLeftDown,
  default420 as LucideCornerLeftUp,
  default421 as LucideCornerRightDown,
  default422 as LucideCornerRightUp,
  default423 as LucideCornerUpLeft,
  default425 as LucideCornerUpRight,
  default424 as LucideCpu,
  default427 as LucideCreativeCommons,
  default426 as LucideCreditCard,
  default428 as LucideCroissant,
  default429 as LucideCrop,
  default431 as LucideCross,
  default430 as LucideCrosshair,
  default432 as LucideCrown,
  default433 as LucideCuboid,
  default434 as LucideCupSoda,
  default1512 as LucideCurlyBraces,
  default435 as LucideCurrency,
  default436 as LucideCylinder,
  default437 as LucideDam,
  default441 as LucideDatabase,
  default438 as LucideDatabaseBackup,
  default439 as LucideDatabaseSearch,
  default440 as LucideDatabaseZap,
  default442 as LucideDecimalsArrowLeft,
  default443 as LucideDecimalsArrowRight,
  default445 as LucideDelete,
  default444 as LucideDessert,
  default446 as LucideDiameter,
  default449 as LucideDiamond,
  default447 as LucideDiamondMinus,
  default1623 as LucideDiamondPercent,
  default448 as LucideDiamondPlus,
  default450 as LucideDice1,
  default451 as LucideDice2,
  default452 as LucideDice3,
  default453 as LucideDice4,
  default454 as LucideDice5,
  default455 as LucideDice6,
  default456 as LucideDices,
  default457 as LucideDiff,
  default461 as LucideDisc,
  default458 as LucideDisc2,
  default460 as LucideDisc3,
  default459 as LucideDiscAlbum,
  default463 as LucideDivide,
  default1572 as LucideDivideCircle,
  default1852 as LucideDivideSquare,
  default464 as LucideDna,
  default462 as LucideDnaOff,
  default465 as LucideDock,
  default466 as LucideDog,
  default467 as LucideDollarSign,
  default468 as LucideDonut,
  default470 as LucideDoorClosed,
  default469 as LucideDoorClosedLocked,
  default471 as LucideDoorOpen,
  default472 as LucideDot,
  default1854 as LucideDotSquare,
  default473 as LucideDownload,
  default1609 as LucideDownloadCloud,
  default474 as LucideDraftingCompass,
  default475 as LucideDrama,
  default476 as LucideDrill,
  default477 as LucideDrone,
  default479 as LucideDroplet,
  default478 as LucideDropletOff,
  default480 as LucideDroplets,
  default481 as LucideDrum,
  default482 as LucideDrumstick,
  default483 as LucideDumbbell,
  default485 as LucideEar,
  default484 as LucideEarOff,
  default1625 as LucideEarth,
  default486 as LucideEarthLock,
  default489 as LucideEclipse,
  default1876 as LucideEdit,
  default1768 as LucideEdit2,
  default1766 as LucideEdit3,
  default490 as LucideEgg,
  default487 as LucideEggFried,
  default488 as LucideEggOff,
  default491 as LucideEllipse,
  default1629 as LucideEllipsis,
  default1627 as LucideEllipsisVertical,
  default494 as LucideEqual,
  default492 as LucideEqualApproximately,
  default493 as LucideEqualNot,
  default1856 as LucideEqualSquare,
  default495 as LucideEraser,
  default496 as LucideEthernetPort,
  default497 as LucideEuro,
  default498 as LucideEvCharger,
  default499 as LucideExpand,
  default500 as LucideExternalLink,
  default504 as LucideEye,
  default501 as LucideEyeClosed,
  default502 as LucideEyeOff,
  default503 as LucideFactory,
  default505 as LucideFan,
  default506 as LucideFastForward,
  default507 as LucideFeather,
  default508 as LucideFence,
  default509 as LucideFerrisWheel,
  default538 as LucideFile,
  default510 as LucideFileArchive,
  default1656 as LucideFileAudio,
  default1657 as LucideFileAudio2,
  default1632 as LucideFileAxis3D,
  default1631 as LucideFileAxis3d,
  default1633 as LucideFileBadge,
  default1634 as LucideFileBadge2,
  default1640 as LucideFileBarChart,
  default1644 as LucideFileBarChart2,
  default511 as LucideFileBox,
  default1637 as LucideFileBraces,
  default1635 as LucideFileBracesCorner,
  default1643 as LucideFileChartColumn,
  default1639 as LucideFileChartColumnIncreasing,
  default1641 as LucideFileChartLine,
  default1645 as LucideFileChartPie,
  default512 as LucideFileCheck,
  default1648 as LucideFileCheck2,
  default1647 as LucideFileCheckCorner,
  default513 as LucideFileClock,
  default514 as LucideFileCode,
  default1650 as LucideFileCode2,
  default1649 as LucideFileCodeCorner,
  default1651 as LucideFileCog,
  default1652 as LucideFileCog2,
  default515 as LucideFileDiff,
  default516 as LucideFileDigit,
  default517 as LucideFileDown,
  default1665 as LucideFileEdit,
  default1653 as LucideFileExclamationPoint,
  default1655 as LucideFileHeadphone,
  default518 as LucideFileHeart,
  default519 as LucideFileImage,
  default520 as LucideFileInput,
  default1638 as LucideFileJson,
  default1636 as LucideFileJson2,
  default1660 as LucideFileKey,
  default1661 as LucideFileKey2,
  default1642 as LucideFileLineChart,
  default1658 as LucideFileLock,
  default1659 as LucideFileLock2,
  default521 as LucideFileMinus,
  default1663 as LucideFileMinus2,
  default1662 as LucideFileMinusCorner,
  default522 as LucideFileMusic,
  default523 as LucideFileOutput,
  default1664 as LucideFilePen,
  default1666 as LucideFilePenLine,
  default1646 as LucideFilePieChart,
  default1668 as LucideFilePlay,
  default524 as LucideFilePlus,
  default1671 as LucideFilePlus2,
  default1670 as LucideFilePlusCorner,
  default1673 as LucideFileQuestion,
  default1672 as LucideFileQuestionMark,
  default525 as LucideFileScan,
  default526 as LucideFileSearch,
  default1675 as LucideFileSearch2,
  default1674 as LucideFileSearchCorner,
  default1676 as LucideFileSignal,
  default1667 as LucideFileSignature,
  default528 as LucideFileSliders,
  default527 as LucideFileSpreadsheet,
  default529 as LucideFileStack,
  default530 as LucideFileSymlink,
  default531 as LucideFileTerminal,
  default532 as LucideFileText,
  default533 as LucideFileType,
  default1679 as LucideFileType2,
  default1678 as LucideFileTypeCorner,
  default534 as LucideFileUp,
  default535 as LucideFileUser,
  default1669 as LucideFileVideo,
  default1681 as LucideFileVideo2,
  default1680 as LucideFileVideoCamera,
  default536 as LucideFileVolume,
  default1677 as LucideFileVolume2,
  default1654 as LucideFileWarning,
  default537 as LucideFileX,
  default1683 as LucideFileX2,
  default1682 as LucideFileXCorner,
  default539 as LucideFiles,
  default540 as LucideFilm,
  default1691 as LucideFilter,
  default1689 as LucideFilterX,
  default1476 as LucideFingerprint,
  default1475 as LucideFingerprintPattern,
  default542 as LucideFireExtinguisher,
  default545 as LucideFish,
  default541 as LucideFishOff,
  default543 as LucideFishSymbol,
  default544 as LucideFishingHook,
  default546 as LucideFishingRod,
  default550 as LucideFlag,
  default547 as LucideFlagOff,
  default548 as LucideFlagTriangleLeft,
  default549 as LucideFlagTriangleRight,
  default552 as LucideFlame,
  default551 as LucideFlameKindling,
  default554 as LucideFlashlight,
  default553 as LucideFlashlightOff,
  default556 as LucideFlaskConical,
  default555 as LucideFlaskConicalOff,
  default557 as LucideFlaskRound,
  default1824 as LucideFlipHorizontal,
  default558 as LucideFlipHorizontal2,
  default1822 as LucideFlipVertical,
  default559 as LucideFlipVertical2,
  default561 as LucideFlower,
  default560 as LucideFlower2,
  default562 as LucideFocus,
  default563 as LucideFoldHorizontal,
  default564 as LucideFoldVertical,
  default593 as LucideFolder,
  default565 as LucideFolderArchive,
  default567 as LucideFolderBookmark,
  default566 as LucideFolderCheck,
  default568 as LucideFolderClock,
  default569 as LucideFolderClosed,
  default570 as LucideFolderCode,
  default1684 as LucideFolderCog,
  default1685 as LucideFolderCog2,
  default571 as LucideFolderDot,
  default572 as LucideFolderDown,
  default1687 as LucideFolderEdit,
  default574 as LucideFolderGit,
  default573 as LucideFolderGit2,
  default575 as LucideFolderHeart,
  default577 as LucideFolderInput,
  default576 as LucideFolderKanban,
  default578 as LucideFolderKey,
  default579 as LucideFolderLock,
  default580 as LucideFolderMinus,
  default582 as LucideFolderOpen,
  default581 as LucideFolderOpenDot,
  default583 as LucideFolderOutput,
  default1686 as LucideFolderPen,
  default584 as LucideFolderPlus,
  default585 as LucideFolderRoot,
  default587 as LucideFolderSearch,
  default586 as LucideFolderSearch2,
  default588 as LucideFolderSymlink,
  default589 as LucideFolderSync,
  default590 as LucideFolderTree,
  default591 as LucideFolderUp,
  default592 as LucideFolderX,
  default594 as LucideFolders,
  default595 as LucideFootprints,
  default1951 as LucideForkKnife,
  default1947 as LucideForkKnifeCrossed,
  default596 as LucideForklift,
  default597 as LucideForm,
  default1772 as LucideFormInput,
  default598 as LucideForward,
  default599 as LucideFrame,
  default600 as LucideFrown,
  default601 as LucideFuel,
  default602 as LucideFullscreen,
  default1858 as LucideFunctionSquare,
  default1690 as LucideFunnel,
  default603 as LucideFunnelPlus,
  default1688 as LucideFunnelX,
  default606 as LucideGalleryHorizontal,
  default604 as LucideGalleryHorizontalEnd,
  default605 as LucideGalleryThumbnails,
  default608 as LucideGalleryVertical,
  default607 as LucideGalleryVerticalEnd,
  default611 as LucideGamepad,
  default609 as LucideGamepad2,
  default610 as LucideGamepadDirectional,
  default1536 as LucideGanttChart,
  default1826 as LucideGanttChartSquare,
  default612 as LucideGauge,
  default1578 as LucideGaugeCircle,
  default613 as LucideGavel,
  default614 as LucideGem,
  default615 as LucideGeorgianLari,
  default616 as LucideGhost,
  default617 as LucideGift,
  default620 as LucideGitBranch,
  default618 as LucideGitBranchMinus,
  default619 as LucideGitBranchPlus,
  default1693 as LucideGitCommit,
  default1692 as LucideGitCommitHorizontal,
  default621 as LucideGitCommitVertical,
  default623 as LucideGitCompare,
  default622 as LucideGitCompareArrows,
  default625 as LucideGitFork,
  default624 as LucideGitGraph,
  default627 as LucideGitMerge,
  default626 as LucideGitMergeConflict,
  default633 as LucideGitPullRequest,
  default629 as LucideGitPullRequestArrow,
  default628 as LucideGitPullRequestClosed,
  default631 as LucideGitPullRequestCreate,
  default630 as LucideGitPullRequestCreateArrow,
  default632 as LucideGitPullRequestDraft,
  default634 as LucideGlassWater,
  default636 as LucideGlasses,
  default639 as LucideGlobe,
  default1626 as LucideGlobe2,
  default635 as LucideGlobeLock,
  default637 as LucideGlobeOff,
  default638 as LucideGlobeX,
  default640 as LucideGoal,
  default641 as LucideGpu,
  default1706 as LucideGrab,
  default643 as LucideGraduationCap,
  default642 as LucideGrape,
  default1703 as LucideGrid,
  default1701 as LucideGrid2X2,
  default1695 as LucideGrid2X2Check,
  default1697 as LucideGrid2X2Plus,
  default1699 as LucideGrid2X2X,
  default1700 as LucideGrid2x2,
  default1694 as LucideGrid2x2Check,
  default1696 as LucideGrid2x2Plus,
  default1698 as LucideGrid2x2X,
  default1704 as LucideGrid3X3,
  default644 as LucideGrid3x2,
  default1702 as LucideGrid3x3,
  default648 as LucideGrip,
  default645 as LucideGripHorizontal,
  default646 as LucideGripVertical,
  default647 as LucideGroup,
  default649 as LucideGuitar,
  default650 as LucideHam,
  default651 as LucideHamburger,
  default652 as LucideHammer,
  default658 as LucideHand,
  default653 as LucideHandCoins,
  default654 as LucideHandFist,
  default1705 as LucideHandGrab,
  default655 as LucideHandHeart,
  default1707 as LucideHandHelping,
  default656 as LucideHandMetal,
  default657 as LucideHandPlatter,
  default659 as LucideHandbag,
  default660 as LucideHandshake,
  default663 as LucideHardDrive,
  default661 as LucideHardDriveDownload,
  default662 as LucideHardDriveUpload,
  default664 as LucideHardHat,
  default665 as LucideHash,
  default666 as LucideHatGlasses,
  default667 as LucideHaze,
  default668 as LucideHd,
  default669 as LucideHdmiPort,
  default676 as LucideHeading,
  default670 as LucideHeading1,
  default671 as LucideHeading2,
  default672 as LucideHeading3,
  default673 as LucideHeading4,
  default674 as LucideHeading5,
  default675 as LucideHeading6,
  default677 as LucideHeadphoneOff,
  default678 as LucideHeadphones,
  default679 as LucideHeadset,
  default687 as LucideHeart,
  default680 as LucideHeartCrack,
  default681 as LucideHeartHandshake,
  default682 as LucideHeartMinus,
  default683 as LucideHeartOff,
  default684 as LucideHeartPlus,
  default685 as LucideHeartPulse,
  default686 as LucideHeartX,
  default689 as LucideHeater,
  default688 as LucideHelicopter,
  default1592 as LucideHelpCircle,
  default1708 as LucideHelpingHand,
  default690 as LucideHexagon,
  default691 as LucideHighlighter,
  default692 as LucideHistory,
  default1710 as LucideHome,
  default694 as LucideHop,
  default693 as LucideHopOff,
  default696 as LucideHospital,
  default695 as LucideHotel,
  default697 as LucideHourglass,
  default1709 as LucideHouse,
  default698 as LucideHouseHeart,
  default699 as LucideHousePlug,
  default700 as LucideHousePlus,
  default701 as LucideHouseWifi,
  default1714 as LucideIceCream,
  default1712 as LucideIceCream2,
  default1711 as LucideIceCreamBowl,
  default1713 as LucideIceCreamCone,
  default703 as LucideIdCard,
  default702 as LucideIdCardLanyard,
  default711 as LucideImage,
  default704 as LucideImageDown,
  default705 as LucideImageMinus,
  default706 as LucideImageOff,
  default707 as LucideImagePlay,
  default708 as LucideImagePlus,
  default709 as LucideImageUp,
  default710 as LucideImageUpscale,
  default712 as LucideImages,
  default713 as LucideImport,
  default714 as LucideInbox,
  default1723 as LucideIndent,
  default1721 as LucideIndentDecrease,
  default1724 as LucideIndentIncrease,
  default715 as LucideIndianRupee,
  default717 as LucideInfinity,
  default716 as LucideInfo,
  default1870 as LucideInspect,
  default718 as LucideInspectionPanel,
  default719 as LucideItalic,
  default720 as LucideIterationCcw,
  default721 as LucideIterationCw,
  default722 as LucideJapaneseYen,
  default723 as LucideJoystick,
  default726 as LucideKanban,
  default1862 as LucideKanbanSquare,
  default1843 as LucideKanbanSquareDashed,
  default724 as LucideKayak,
  default728 as LucideKey,
  default725 as LucideKeyRound,
  default727 as LucideKeySquare,
  default730 as LucideKeyboard,
  default729 as LucideKeyboardMusic,
  default731 as LucideKeyboardOff,
  default737 as LucideLamp,
  default732 as LucideLampCeiling,
  default733 as LucideLampDesk,
  default734 as LucideLampFloor,
  default735 as LucideLampWallDown,
  default736 as LucideLampWallUp,
  default738 as LucideLandPlot,
  default739 as LucideLandmark,
  default741 as LucideLanguages,
  default742 as LucideLaptop,
  default1716 as LucideLaptop2,
  default1715 as LucideLaptopMinimal,
  default740 as LucideLaptopMinimalCheck,
  default744 as LucideLasso,
  default743 as LucideLassoSelect,
  default745 as LucideLaugh,
  default1717 as LucideLayers,
  default746 as LucideLayers2,
  default1718 as LucideLayers3,
  default747 as LucideLayersMinus,
  default748 as LucideLayersPlus,
  default1764 as LucideLayout,
  default749 as LucideLayoutDashboard,
  default750 as LucideLayoutGrid,
  default751 as LucideLayoutList,
  default752 as LucideLayoutPanelLeft,
  default753 as LucideLayoutPanelTop,
  default754 as LucideLayoutTemplate,
  default755 as LucideLeaf,
  default756 as LucideLeafyGreen,
  default757 as LucideLectern,
  default758 as LucideLensConcave,
  default759 as LucideLensConvex,
  default1921 as LucideLetterText,
  default760 as LucideLibrary,
  default761 as LucideLibraryBig,
  default1860 as LucideLibrarySquare,
  default762 as LucideLifeBuoy,
  default763 as LucideLigature,
  default765 as LucideLightbulb,
  default764 as LucideLightbulbOff,
  default1530 as LucideLineChart,
  default766 as LucideLineDotRightHorizontal,
  default767 as LucideLineSquiggle,
  default768 as LucideLineStyle,
  default772 as LucideLink,
  default770 as LucideLink2,
  default769 as LucideLink2Off,
  default790 as LucideList,
  default771 as LucideListCheck,
  default773 as LucideListChecks,
  default774 as LucideListChevronsDownUp,
  default775 as LucideListChevronsUpDown,
  default777 as LucideListCollapse,
  default776 as LucideListEnd,
  default779 as LucideListFilter,
  default778 as LucideListFilterPlus,
  default1719 as LucideListIndentDecrease,
  default1722 as LucideListIndentIncrease,
  default780 as LucideListMinus,
  default781 as LucideListMusic,
  default782 as LucideListOrdered,
  default783 as LucideListPlus,
  default784 as LucideListRestart,
  default785 as LucideListStart,
  default786 as LucideListTodo,
  default787 as LucideListTree,
  default788 as LucideListVideo,
  default789 as LucideListX,
  default792 as LucideLoader,
  default1726 as LucideLoader2,
  default1725 as LucideLoaderCircle,
  default791 as LucideLoaderPinwheel,
  default795 as LucideLocate,
  default793 as LucideLocateFixed,
  default794 as LucideLocateOff,
  default1734 as LucideLocationEdit,
  default797 as LucideLock,
  default796 as LucideLockKeyhole,
  default1729 as LucideLockKeyholeOpen,
  default1727 as LucideLockOpen,
  default798 as LucideLogIn,
  default800 as LucideLogOut,
  default799 as LucideLogs,
  default801 as LucideLollipop,
  default802 as LucideLuggage,
  default1864 as LucideMSquare,
  default803 as LucideMagnet,
  default811 as LucideMail,
  default804 as LucideMailCheck,
  default805 as LucideMailMinus,
  default806 as LucideMailOpen,
  default807 as LucideMailPlus,
  default1732 as LucideMailQuestion,
  default1731 as LucideMailQuestionMark,
  default808 as LucideMailSearch,
  default809 as LucideMailWarning,
  default810 as LucideMailX,
  default812 as LucideMailbox,
  default813 as LucideMails,
  default829 as LucideMap,
  default814 as LucideMapMinus,
  default826 as LucideMapPin,
  default816 as LucideMapPinCheck,
  default815 as LucideMapPinCheckInside,
  default817 as LucideMapPinHouse,
  default819 as LucideMapPinMinus,
  default818 as LucideMapPinMinusInside,
  default820 as LucideMapPinOff,
  default1733 as LucideMapPinPen,
  default822 as LucideMapPinPlus,
  default821 as LucideMapPinPlusInside,
  default823 as LucideMapPinSearch,
  default825 as LucideMapPinX,
  default824 as LucideMapPinXInside,
  default827 as LucideMapPinned,
  default828 as LucideMapPlus,
  default831 as LucideMars,
  default830 as LucideMarsStroke,
  default832 as LucideMartini,
  default835 as LucideMaximize,
  default833 as LucideMaximize2,
  default834 as LucideMedal,
  default836 as LucideMegaphone,
  default837 as LucideMegaphoneOff,
  default838 as LucideMeh,
  default840 as LucideMemoryStick,
  default839 as LucideMenu,
  default1866 as LucideMenuSquare,
  default841 as LucideMerge,
  default852 as LucideMessageCircle,
  default842 as LucideMessageCircleCheck,
  default843 as LucideMessageCircleCode,
  default844 as LucideMessageCircleDashed,
  default845 as LucideMessageCircleHeart,
  default846 as LucideMessageCircleMore,
  default847 as LucideMessageCircleOff,
  default848 as LucideMessageCirclePlus,
  default1736 as LucideMessageCircleQuestion,
  default1735 as LucideMessageCircleQuestionMark,
  default849 as LucideMessageCircleReply,
  default850 as LucideMessageCircleWarning,
  default851 as LucideMessageCircleX,
  default869 as LucideMessageSquare,
  default853 as LucideMessageSquareCheck,
  default855 as LucideMessageSquareCode,
  default854 as LucideMessageSquareDashed,
  default856 as LucideMessageSquareDiff,
  default857 as LucideMessageSquareDot,
  default858 as LucideMessageSquareHeart,
  default859 as LucideMessageSquareLock,
  default860 as LucideMessageSquareMore,
  default861 as LucideMessageSquareOff,
  default865 as LucideMessageSquarePlus,
  default862 as LucideMessageSquareQuote,
  default863 as LucideMessageSquareReply,
  default864 as LucideMessageSquareShare,
  default866 as LucideMessageSquareText,
  default867 as LucideMessageSquareWarning,
  default868 as LucideMessageSquareX,
  default870 as LucideMessagesSquare,
  default871 as LucideMetronome,
  default874 as LucideMic,
  default1738 as LucideMic2,
  default872 as LucideMicOff,
  default1737 as LucideMicVocal,
  default873 as LucideMicrochip,
  default875 as LucideMicroscope,
  default876 as LucideMicrowave,
  default877 as LucideMilestone,
  default879 as LucideMilk,
  default878 as LucideMilkOff,
  default881 as LucideMinimize,
  default880 as LucideMinimize2,
  default883 as LucideMinus,
  default1574 as LucideMinusCircle,
  default1868 as LucideMinusSquare,
  default882 as LucideMirrorRectangular,
  default884 as LucideMirrorRound,
  default898 as LucideMonitor,
  default885 as LucideMonitorCheck,
  default886 as LucideMonitorCloud,
  default887 as LucideMonitorCog,
  default888 as LucideMonitorDot,
  default889 as LucideMonitorDown,
  default890 as LucideMonitorOff,
  default891 as LucideMonitorPause,
  default892 as LucideMonitorPlay,
  default893 as LucideMonitorSmartphone,
  default894 as LucideMonitorSpeaker,
  default895 as LucideMonitorStop,
  default896 as LucideMonitorUp,
  default897 as LucideMonitorX,
  default900 as LucideMoon,
  default899 as LucideMoonStar,
  default1630 as LucideMoreHorizontal,
  default1628 as LucideMoreVertical,
  default901 as LucideMotorbike,
  default903 as LucideMountain,
  default902 as LucideMountainSnow,
  default912 as LucideMouse,
  default904 as LucideMouseLeft,
  default905 as LucideMouseOff,
  default910 as LucideMousePointer,
  default907 as LucideMousePointer2,
  default906 as LucideMousePointer2Off,
  default908 as LucideMousePointerBan,
  default909 as LucideMousePointerClick,
  default1845 as LucideMousePointerSquareDashed,
  default911 as LucideMouseRight,
  default924 as LucideMove,
  default1740 as LucideMove3D,
  default1739 as LucideMove3d,
  default914 as LucideMoveDiagonal,
  default913 as LucideMoveDiagonal2,
  default917 as LucideMoveDown,
  default915 as LucideMoveDownLeft,
  default916 as LucideMoveDownRight,
  default918 as LucideMoveHorizontal,
  default920 as LucideMoveLeft,
  default919 as LucideMoveRight,
  default923 as LucideMoveUp,
  default921 as LucideMoveUpLeft,
  default922 as LucideMoveUpRight,
  default925 as LucideMoveVertical,
  default929 as LucideMusic,
  default926 as LucideMusic2,
  default927 as LucideMusic3,
  default928 as LucideMusic4,
  default933 as LucideNavigation,
  default931 as LucideNavigation2,
  default930 as LucideNavigation2Off,
  default932 as LucideNavigationOff,
  default934 as LucideNetwork,
  default935 as LucideNewspaper,
  default936 as LucideNfc,
  default937 as LucideNonBinary,
  default941 as LucideNotebook,
  default938 as LucideNotebookPen,
  default939 as LucideNotebookTabs,
  default940 as LucideNotebookText,
  default943 as LucideNotepadText,
  default942 as LucideNotepadTextDashed,
  default945 as LucideNut,
  default944 as LucideNutOff,
  default947 as LucideOctagon,
  default1741 as LucideOctagonAlert,
  default946 as LucideOctagonMinus,
  default1743 as LucideOctagonPause,
  default1745 as LucideOctagonX,
  default948 as LucideOmega,
  default949 as LucideOption,
  default950 as LucideOrbit,
  default951 as LucideOrigami,
  default1720 as LucideOutdent,
  default959 as LucidePackage,
  default952 as LucidePackage2,
  default954 as LucidePackageCheck,
  default953 as LucidePackageMinus,
  default955 as LucidePackageOpen,
  default956 as LucidePackagePlus,
  default957 as LucidePackageSearch,
  default958 as LucidePackageX,
  default960 as LucidePaintBucket,
  default961 as LucidePaintRoller,
  default962 as LucidePaintbrush,
  default1748 as LucidePaintbrush2,
  default1747 as LucidePaintbrushVertical,
  default963 as LucidePalette,
  default1927 as LucidePalmtree,
  default964 as LucidePanda,
  default967 as LucidePanelBottom,
  default965 as LucidePanelBottomClose,
  default1751 as LucidePanelBottomDashed,
  default1752 as LucidePanelBottomInactive,
  default966 as LucidePanelBottomOpen,
  default1757 as LucidePanelLeft,
  default1749 as LucidePanelLeftClose,
  default1753 as LucidePanelLeftDashed,
  default1754 as LucidePanelLeftInactive,
  default1755 as LucidePanelLeftOpen,
  default968 as LucidePanelLeftRightDashed,
  default971 as LucidePanelRight,
  default969 as LucidePanelRightClose,
  default1759 as LucidePanelRightDashed,
  default1760 as LucidePanelRightInactive,
  default970 as LucidePanelRightOpen,
  default975 as LucidePanelTop,
  default972 as LucidePanelTopBottomDashed,
  default973 as LucidePanelTopClose,
  default1761 as LucidePanelTopDashed,
  default1762 as LucidePanelTopInactive,
  default974 as LucidePanelTopOpen,
  default976 as LucidePanelsLeftBottom,
  default1620 as LucidePanelsLeftRight,
  default977 as LucidePanelsRightBottom,
  default1778 as LucidePanelsTopBottom,
  default1763 as LucidePanelsTopLeft,
  default978 as LucidePaperclip,
  default979 as LucideParentheses,
  default1586 as LucideParkingCircle,
  default1576 as LucideParkingCircleOff,
  default980 as LucideParkingMeter,
  default1874 as LucideParkingSquare,
  default1872 as LucideParkingSquareOff,
  default981 as LucidePartyPopper,
  default983 as LucidePause,
  default1580 as LucidePauseCircle,
  default1744 as LucidePauseOctagon,
  default982 as LucidePawPrint,
  default984 as LucidePcCase,
  default1767 as LucidePen,
  default1877 as LucidePenBox,
  default1765 as LucidePenLine,
  default985 as LucidePenOff,
  default1878 as LucidePenSquare,
  default986 as LucidePenTool,
  default990 as LucidePencil,
  default987 as LucidePencilLine,
  default988 as LucidePencilOff,
  default989 as LucidePencilRuler,
  default991 as LucidePentagon,
  default992 as LucidePercent,
  default1582 as LucidePercentCircle,
  default1624 as LucidePercentDiamond,
  default1880 as LucidePercentSquare,
  default993 as LucidePersonStanding,
  default994 as LucidePhilippinePeso,
  default1001 as LucidePhone,
  default995 as LucidePhoneCall,
  default996 as LucidePhoneForwarded,
  default997 as LucidePhoneIncoming,
  default998 as LucidePhoneMissed,
  default999 as LucidePhoneOff,
  default1000 as LucidePhoneOutgoing,
  default1002 as LucidePi,
  default1884 as LucidePiSquare,
  default1003 as LucidePiano,
  default1004 as LucidePickaxe,
  default1006 as LucidePictureInPicture,
  default1005 as LucidePictureInPicture2,
  default1538 as LucidePieChart,
  default1007 as LucidePiggyBank,
  default1010 as LucidePilcrow,
  default1008 as LucidePilcrowLeft,
  default1009 as LucidePilcrowRight,
  default1886 as LucidePilcrowSquare,
  default1012 as LucidePill,
  default1011 as LucidePillBottle,
  default1015 as LucidePin,
  default1013 as LucidePinOff,
  default1014 as LucidePipette,
  default1016 as LucidePizza,
  default1019 as LucidePlane,
  default1017 as LucidePlaneLanding,
  default1018 as LucidePlaneTakeoff,
  default1021 as LucidePlay,
  default1584 as LucidePlayCircle,
  default1882 as LucidePlaySquare,
  default1022 as LucidePlug,
  default1020 as LucidePlug2,
  default1769 as LucidePlugZap,
  default1770 as LucidePlugZap2,
  default1023 as LucidePlus,
  default1588 as LucidePlusCircle,
  default1888 as LucidePlusSquare,
  default1024 as LucidePocketKnife,
  default1025 as LucidePodcast,
  default1026 as LucidePointer,
  default1027 as LucidePointerOff,
  default1028 as LucidePopcorn,
  default1029 as LucidePopsicle,
  default1030 as LucidePoundSterling,
  default1032 as LucidePower,
  default1590 as LucidePowerCircle,
  default1031 as LucidePowerOff,
  default1890 as LucidePowerSquare,
  default1033 as LucidePresentation,
  default1036 as LucidePrinter,
  default1034 as LucidePrinterCheck,
  default1037 as LucidePrinterX,
  default1035 as LucideProjector,
  default1038 as LucideProportions,
  default1039 as LucidePuzzle,
  default1040 as LucidePyramid,
  default1041 as LucideQrCode,
  default1042 as LucideQuote,
  default1043 as LucideRabbit,
  default1044 as LucideRadar,
  default1045 as LucideRadiation,
  default1046 as LucideRadical,
  default1049 as LucideRadio,
  default1047 as LucideRadioOff,
  default1048 as LucideRadioReceiver,
  default1050 as LucideRadioTower,
  default1051 as LucideRadius,
  default1052 as LucideRainbow,
  default1053 as LucideRat,
  default1054 as LucideRatio,
  default1064 as LucideReceipt,
  default1055 as LucideReceiptCent,
  default1056 as LucideReceiptEuro,
  default1057 as LucideReceiptIndianRupee,
  default1058 as LucideReceiptJapaneseYen,
  default1059 as LucideReceiptPoundSterling,
  default1061 as LucideReceiptRussianRuble,
  default1060 as LucideReceiptSwissFranc,
  default1062 as LucideReceiptText,
  default1063 as LucideReceiptTurkishLira,
  default1065 as LucideRectangleCircle,
  default1771 as LucideRectangleEllipsis,
  default1066 as LucideRectangleGoggles,
  default1067 as LucideRectangleHorizontal,
  default1068 as LucideRectangleVertical,
  default1069 as LucideRecycle,
  default1072 as LucideRedo,
  default1070 as LucideRedo2,
  default1071 as LucideRedoDot,
  default1074 as LucideRefreshCcw,
  default1073 as LucideRefreshCcwDot,
  default1076 as LucideRefreshCw,
  default1075 as LucideRefreshCwOff,
  default1078 as LucideRefrigerator,
  default1079 as LucideRegex,
  default1077 as LucideRemoveFormatting,
  default1083 as LucideRepeat,
  default1080 as LucideRepeat1,
  default1081 as LucideRepeat2,
  default1082 as LucideRepeatOff,
  default1085 as LucideReplace,
  default1084 as LucideReplaceAll,
  default1087 as LucideReply,
  default1086 as LucideReplyAll,
  default1088 as LucideRewind,
  default1089 as LucideRibbon,
  default1090 as LucideRoad,
  default1091 as LucideRocket,
  default1093 as LucideRockingChair,
  default1092 as LucideRollerCoaster,
  default1094 as LucideRose,
  default1774 as LucideRotate3D,
  default1773 as LucideRotate3d,
  default1097 as LucideRotateCcw,
  default1095 as LucideRotateCcwKey,
  default1096 as LucideRotateCcwSquare,
  default1099 as LucideRotateCw,
  default1098 as LucideRotateCwSquare,
  default1101 as LucideRoute,
  default1100 as LucideRouteOff,
  default1102 as LucideRouter,
  default1776 as LucideRows,
  default1775 as LucideRows2,
  default1777 as LucideRows3,
  default1103 as LucideRows4,
  default1104 as LucideRss,
  default1106 as LucideRuler,
  default1105 as LucideRulerDimensionLine,
  default1107 as LucideRussianRuble,
  default1108 as LucideSailboat,
  default1109 as LucideSalad,
  default1110 as LucideSandwich,
  default1112 as LucideSatellite,
  default1111 as LucideSatelliteDish,
  default1113 as LucideSaudiRiyal,
  default1116 as LucideSave,
  default1114 as LucideSaveAll,
  default1115 as LucideSaveOff,
  default1118 as LucideScale,
  default1780 as LucideScale3D,
  default1779 as LucideScale3d,
  default1117 as LucideScaling,
  default1127 as LucideScan,
  default1119 as LucideScanBarcode,
  default1120 as LucideScanEye,
  default1121 as LucideScanFace,
  default1122 as LucideScanHeart,
  default1123 as LucideScanLine,
  default1124 as LucideScanQrCode,
  default1125 as LucideScanSearch,
  default1126 as LucideScanText,
  default1540 as LucideScatterChart,
  default1128 as LucideSchool,
  default1933 as LucideSchool2,
  default1130 as LucideScissors,
  default1129 as LucideScissorsLineDashed,
  default1892 as LucideScissorsSquare,
  default1820 as LucideScissorsSquareDashedBottom,
  default1131 as LucideScooter,
  default1133 as LucideScreenShare,
  default1132 as LucideScreenShareOff,
  default1135 as LucideScroll,
  default1134 as LucideScrollText,
  default1142 as LucideSearch,
  default1136 as LucideSearchAlert,
  default1137 as LucideSearchCheck,
  default1138 as LucideSearchCode,
  default1140 as LucideSearchSlash,
  default1139 as LucideSearchX,
  default1141 as LucideSection,
  default1143 as LucideSend,
  default1782 as LucideSendHorizonal,
  default1781 as LucideSendHorizontal,
  default1144 as LucideSendToBack,
  default1145 as LucideSeparatorHorizontal,
  default1147 as LucideSeparatorVertical,
  default1150 as LucideServer,
  default1146 as LucideServerCog,
  default1148 as LucideServerCrash,
  default1149 as LucideServerOff,
  default1152 as LucideSettings,
  default1151 as LucideSettings2,
  default1153 as LucideShapes,
  default1155 as LucideShare,
  default1154 as LucideShare2,
  default1156 as LucideSheet,
  default1157 as LucideShell,
  default1158 as LucideShelvingUnit,
  default1171 as LucideShield,
  default1159 as LucideShieldAlert,
  default1160 as LucideShieldBan,
  default1161 as LucideShieldCheck,
  default1786 as LucideShieldClose,
  default1163 as LucideShieldCog,
  default1162 as LucideShieldCogCorner,
  default1164 as LucideShieldEllipsis,
  default1165 as LucideShieldHalf,
  default1166 as LucideShieldMinus,
  default1167 as LucideShieldOff,
  default1168 as LucideShieldPlus,
  default1784 as LucideShieldQuestion,
  default1783 as LucideShieldQuestionMark,
  default1169 as LucideShieldUser,
  default1785 as LucideShieldX,
  default1172 as LucideShip,
  default1170 as LucideShipWheel,
  default1173 as LucideShirt,
  default1174 as LucideShoppingBag,
  default1175 as LucideShoppingBasket,
  default1176 as LucideShoppingCart,
  default1177 as LucideShovel,
  default1178 as LucideShowerHead,
  default1179 as LucideShredder,
  default1180 as LucideShrimp,
  default1181 as LucideShrink,
  default1182 as LucideShrub,
  default1183 as LucideShuffle,
  default1758 as LucideSidebar,
  default1750 as LucideSidebarClose,
  default1756 as LucideSidebarOpen,
  default1184 as LucideSigma,
  default1894 as LucideSigmaSquare,
  default1189 as LucideSignal,
  default1185 as LucideSignalHigh,
  default1187 as LucideSignalLow,
  default1186 as LucideSignalMedium,
  default1188 as LucideSignalZero,
  default1191 as LucideSignature,
  default1192 as LucideSignpost,
  default1190 as LucideSignpostBig,
  default1193 as LucideSiren,
  default1194 as LucideSkipBack,
  default1195 as LucideSkipForward,
  default1196 as LucideSkull,
  default1197 as LucideSlash,
  default1896 as LucideSlashSquare,
  default1198 as LucideSlice,
  default1788 as LucideSliders,
  default1199 as LucideSlidersHorizontal,
  default1787 as LucideSlidersVertical,
  default1202 as LucideSmartphone,
  default1200 as LucideSmartphoneCharging,
  default1201 as LucideSmartphoneNfc,
  default1204 as LucideSmile,
  default1203 as LucideSmilePlus,
  default1205 as LucideSnail,
  default1206 as LucideSnowflake,
  default1207 as LucideSoapDispenserDroplet,
  default1208 as LucideSofa,
  default1209 as LucideSolarPanel,
  default1496 as LucideSortAsc,
  default1492 as LucideSortDesc,
  default1210 as LucideSoup,
  default1211 as LucideSpace,
  default1212 as LucideSpade,
  default1213 as LucideSparkle,
  default1789 as LucideSparkles,
  default1214 as LucideSpeaker,
  default1215 as LucideSpeech,
  default1217 as LucideSpellCheck,
  default1216 as LucideSpellCheck2,
  default1219 as LucideSpline,
  default1218 as LucideSplinePointer,
  default1220 as LucideSplit,
  default1898 as LucideSplitSquareHorizontal,
  default1900 as LucideSplitSquareVertical,
  default1221 as LucideSpool,
  default1222 as LucideSportShoe,
  default1223 as LucideSpotlight,
  default1224 as LucideSprayCan,
  default1225 as LucideSprout,
  default1239 as LucideSquare,
  default1791 as LucideSquareActivity,
  default1797 as LucideSquareArrowDown,
  default1793 as LucideSquareArrowDownLeft,
  default1795 as LucideSquareArrowDownRight,
  default1799 as LucideSquareArrowLeft,
  default1803 as LucideSquareArrowOutDownLeft,
  default1801 as LucideSquareArrowOutDownRight,
  default1805 as LucideSquareArrowOutUpLeft,
  default1807 as LucideSquareArrowOutUpRight,
  default1809 as LucideSquareArrowRight,
  default1226 as LucideSquareArrowRightEnter,
  default1227 as LucideSquareArrowRightExit,
  default1815 as LucideSquareArrowUp,
  default1811 as LucideSquareArrowUpLeft,
  default1813 as LucideSquareArrowUpRight,
  default1817 as LucideSquareAsterisk,
  default1819 as LucideSquareBottomDashedScissors,
  default1823 as LucideSquareCenterlineDashedHorizontal,
  default1821 as LucideSquareCenterlineDashedVertical,
  default1825 as LucideSquareChartGantt,
  default1832 as LucideSquareCheck,
  default1828 as LucideSquareCheckBig,
  default1836 as LucideSquareChevronDown,
  default1830 as LucideSquareChevronLeft,
  default1834 as LucideSquareChevronRight,
  default1838 as LucideSquareChevronUp,
  default1840 as LucideSquareCode,
  default1849 as LucideSquareDashed,
  default1229 as LucideSquareDashedBottom,
  default1228 as LucideSquareDashedBottomCode,
  default1842 as LucideSquareDashedKanban,
  default1844 as LucideSquareDashedMousePointer,
  default1846 as LucideSquareDashedText,
  default1230 as LucideSquareDashedTopSolid,
  default1851 as LucideSquareDivide,
  default1853 as LucideSquareDot,
  default1855 as LucideSquareEqual,
  default1857 as LucideSquareFunction,
  default1827 as LucideSquareGanttChart,
  default1861 as LucideSquareKanban,
  default1859 as LucideSquareLibrary,
  default1863 as LucideSquareM,
  default1865 as LucideSquareMenu,
  default1867 as LucideSquareMinus,
  default1869 as LucideSquareMousePointer,
  default1873 as LucideSquareParking,
  default1871 as LucideSquareParkingOff,
  default1231 as LucideSquarePause,
  default1875 as LucideSquarePen,
  default1879 as LucideSquarePercent,
  default1883 as LucideSquarePi,
  default1885 as LucideSquarePilcrow,
  default1881 as LucideSquarePlay,
  default1887 as LucideSquarePlus,
  default1889 as LucideSquarePower,
  default1232 as LucideSquareRadical,
  default1233 as LucideSquareRoundCorner,
  default1891 as LucideSquareScissors,
  default1893 as LucideSquareSigma,
  default1895 as LucideSquareSlash,
  default1897 as LucideSquareSplitHorizontal,
  default1899 as LucideSquareSplitVertical,
  default1234 as LucideSquareSquare,
  default1235 as LucideSquareStack,
  default1236 as LucideSquareStar,
  default1237 as LucideSquareStop,
  default1901 as LucideSquareTerminal,
  default1907 as LucideSquareUser,
  default1903 as LucideSquareUserRound,
  default1905 as LucideSquareX,
  default1238 as LucideSquaresExclude,
  default1240 as LucideSquaresIntersect,
  default1241 as LucideSquaresSubtract,
  default1242 as LucideSquaresUnite,
  default1243 as LucideSquircle,
  default1244 as LucideSquircleDashed,
  default1245 as LucideSquirrel,
  default1246 as LucideStamp,
  default1250 as LucideStar,
  default1247 as LucideStarHalf,
  default1248 as LucideStarOff,
  default1790 as LucideStars,
  default1249 as LucideStepBack,
  default1251 as LucideStepForward,
  default1252 as LucideStethoscope,
  default1253 as LucideSticker,
  default1259 as LucideStickyNote,
  default1255 as LucideStickyNoteCheck,
  default1254 as LucideStickyNoteMinus,
  default1257 as LucideStickyNoteOff,
  default1256 as LucideStickyNotePlus,
  default1258 as LucideStickyNoteX,
  default1260 as LucideStickyNotes,
  default1261 as LucideStone,
  default1597 as LucideStopCircle,
  default1262 as LucideStore,
  default1263 as LucideStretchHorizontal,
  default1264 as LucideStretchVertical,
  default1265 as LucideStrikethrough,
  default1266 as LucideSubscript,
  default1514 as LucideSubtitles,
  default1271 as LucideSun,
  default1267 as LucideSunDim,
  default1268 as LucideSunMedium,
  default1269 as LucideSunMoon,
  default1270 as LucideSunSnow,
  default1272 as LucideSunrise,
  default1273 as LucideSunset,
  default1274 as LucideSuperscript,
  default1275 as LucideSwatchBook,
  default1276 as LucideSwissFranc,
  default1277 as LucideSwitchCamera,
  default1278 as LucideSword,
  default1279 as LucideSwords,
  default1280 as LucideSyringe,
  default1288 as LucideTable,
  default1281 as LucideTable2,
  default1282 as LucideTableCellsMerge,
  default1283 as LucideTableCellsSplit,
  default1284 as LucideTableColumnsSplit,
  default1618 as LucideTableConfig,
  default1285 as LucideTableOfContents,
  default1286 as LucideTableProperties,
  default1287 as LucideTableRowsSplit,
  default1290 as LucideTablet,
  default1289 as LucideTabletSmartphone,
  default1291 as LucideTablets,
  default1292 as LucideTag,
  default1295 as LucideTags,
  default1293 as LucideTally1,
  default1294 as LucideTally2,
  default1296 as LucideTally3,
  default1297 as LucideTally4,
  default1298 as LucideTally5,
  default1299 as LucideTangent,
  default1300 as LucideTarget,
  default1301 as LucideTelescope,
  default1302 as LucideTent,
  default1303 as LucideTentTree,
  default1304 as LucideTerminal,
  default1902 as LucideTerminalSquare,
  default1305 as LucideTestTube,
  default1910 as LucideTestTube2,
  default1909 as LucideTestTubeDiagonal,
  default1306 as LucideTestTubes,
  default1919 as LucideText,
  default1911 as LucideTextAlignCenter,
  default1915 as LucideTextAlignEnd,
  default1913 as LucideTextAlignJustify,
  default1917 as LucideTextAlignStart,
  default1308 as LucideTextCursor,
  default1307 as LucideTextCursorInput,
  default1920 as LucideTextInitial,
  default1309 as LucideTextQuote,
  default1310 as LucideTextSearch,
  default1848 as LucideTextSelect,
  default1847 as LucideTextSelection,
  default1922 as LucideTextWrap,
  default1311 as LucideTheater,
  default1314 as LucideThermometer,
  default1312 as LucideThermometerSnowflake,
  default1313 as LucideThermometerSun,
  default1315 as LucideThumbsDown,
  default1316 as LucideThumbsUp,
  default1323 as LucideTicket,
  default1318 as LucideTicketCheck,
  default1317 as LucideTicketMinus,
  default1319 as LucideTicketPercent,
  default1320 as LucideTicketPlus,
  default1322 as LucideTicketSlash,
  default1321 as LucideTicketX,
  default1326 as LucideTickets,
  default1324 as LucideTicketsPlane,
  default1325 as LucideTimeline,
  default1329 as LucideTimer,
  default1327 as LucideTimerOff,
  default1328 as LucideTimerReset,
  default1330 as LucideToggleLeft,
  default1332 as LucideToggleRight,
  default1334 as LucideToilet,
  default1331 as LucideToolCase,
  default1335 as LucideToolbox,
  default1333 as LucideTornado,
  default1336 as LucideTorus,
  default1339 as LucideTouchpad,
  default1337 as LucideTouchpadOff,
  default1338 as LucideTowelRack,
  default1340 as LucideTowerControl,
  default1342 as LucideToyBrick,
  default1341 as LucideTractor,
  default1343 as LucideTrafficCone,
  default1925 as LucideTrain,
  default1345 as LucideTrainFront,
  default1344 as LucideTrainFrontTunnel,
  default1346 as LucideTrainTrack,
  default1924 as LucideTramFront,
  default1347 as LucideTransgender,
  default1349 as LucideTrash,
  default1348 as LucideTrash2,
  default1350 as LucideTreeDeciduous,
  default1926 as LucideTreePalm,
  default1351 as LucideTreePine,
  default1352 as LucideTrees,
  default1353 as LucideTrendingDown,
  default1355 as LucideTrendingUp,
  default1354 as LucideTrendingUpDown,
  default1358 as LucideTriangle,
  default1928 as LucideTriangleAlert,
  default1356 as LucideTriangleDashed,
  default1357 as LucideTriangleRight,
  default1359 as LucideTrophy,
  default1361 as LucideTruck,
  default1360 as LucideTruckElectric,
  default1362 as LucideTurkishLira,
  default1363 as LucideTurntable,
  default1364 as LucideTurtle,
  default1366 as LucideTv,
  default1931 as LucideTv2,
  default1930 as LucideTvMinimal,
  default1365 as LucideTvMinimalPlay,
  default1368 as LucideType,
  default1367 as LucideTypeOutline,
  default1369 as LucideUmbrella,
  default1370 as LucideUmbrellaOff,
  default1372 as LucideUnderline,
  default1374 as LucideUndo,
  default1371 as LucideUndo2,
  default1373 as LucideUndoDot,
  default1375 as LucideUnfoldHorizontal,
  default1376 as LucideUnfoldVertical,
  default1377 as LucideUngroup,
  default1932 as LucideUniversity,
  default1379 as LucideUnlink,
  default1378 as LucideUnlink2,
  default1728 as LucideUnlock,
  default1730 as LucideUnlockKeyhole,
  default1380 as LucideUnplug,
  default1381 as LucideUpload,
  default1611 as LucideUploadCloud,
  default1382 as LucideUsb,
  default1396 as LucideUser,
  default1945 as LucideUser2,
  default1383 as LucideUserCheck,
  default1935 as LucideUserCheck2,
  default1601 as LucideUserCircle,
  default1599 as LucideUserCircle2,
  default1384 as LucideUserCog,
  default1937 as LucideUserCog2,
  default1385 as LucideUserKey,
  default1386 as LucideUserLock,
  default1387 as LucideUserMinus,
  default1941 as LucideUserMinus2,
  default1388 as LucideUserPen,
  default1389 as LucideUserPlus,
  default1939 as LucideUserPlus2,
  default1944 as LucideUserRound,
  default1934 as LucideUserRoundCheck,
  default1936 as LucideUserRoundCog,
  default1390 as LucideUserRoundKey,
  default1940 as LucideUserRoundMinus,
  default1391 as LucideUserRoundPen,
  default1938 as LucideUserRoundPlus,
  default1392 as LucideUserRoundSearch,
  default1942 as LucideUserRoundX,
  default1393 as LucideUserSearch,
  default1908 as LucideUserSquare,
  default1904 as LucideUserSquare2,
  default1394 as LucideUserStar,
  default1395 as LucideUserX,
  default1943 as LucideUserX2,
  default1397 as LucideUsers,
  default1949 as LucideUsers2,
  default1948 as LucideUsersRound,
  default1950 as LucideUtensils,
  default1946 as LucideUtensilsCrossed,
  default1398 as LucideUtilityPole,
  default1399 as LucideVan,
  default1400 as LucideVariable,
  default1401 as LucideVault,
  default1402 as LucideVectorSquare,
  default1403 as LucideVegan,
  default1404 as LucideVenetianMask,
  default1406 as LucideVenus,
  default1405 as LucideVenusAndMars,
  default1502 as LucideVerified,
  default1408 as LucideVibrate,
  default1407 as LucideVibrateOff,
  default1410 as LucideVideo,
  default1409 as LucideVideoOff,
  default1411 as LucideVideotape,
  default1412 as LucideView,
  default1413 as LucideVoicemail,
  default1414 as LucideVolleyball,
  default1419 as LucideVolume,
  default1415 as LucideVolume1,
  default1416 as LucideVolume2,
  default1417 as LucideVolumeOff,
  default1418 as LucideVolumeX,
  default1420 as LucideVote,
  default1423 as LucideWallet,
  default1953 as LucideWallet2,
  default1421 as LucideWalletCards,
  default1952 as LucideWalletMinimal,
  default1422 as LucideWallpaper,
  default1425 as LucideWand,
  default1955 as LucideWand2,
  default1954 as LucideWandSparkles,
  default1424 as LucideWarehouse,
  default1426 as LucideWashingMachine,
  default1427 as LucideWatch,
  default1957 as LucideWaves,
  default1428 as LucideWavesArrowDown,
  default1429 as LucideWavesArrowUp,
  default1956 as LucideWavesHorizontal,
  default1431 as LucideWavesLadder,
  default1430 as LucideWavesVertical,
  default1432 as LucideWaypoints,
  default1433 as LucideWebcam,
  default1435 as LucideWebhook,
  default1434 as LucideWebhookOff,
  default1437 as LucideWeight,
  default1436 as LucideWeightTilde,
  default1440 as LucideWheat,
  default1438 as LucideWheatOff,
  default1439 as LucideWholeWord,
  default1448 as LucideWifi,
  default1441 as LucideWifiCog,
  default1442 as LucideWifiHigh,
  default1443 as LucideWifiLow,
  default1444 as LucideWifiOff,
  default1445 as LucideWifiPen,
  default1446 as LucideWifiSync,
  default1447 as LucideWifiZero,
  default1450 as LucideWind,
  default1449 as LucideWindArrowDown,
  default1451 as LucideWine,
  default1452 as LucideWineOff,
  default1453 as LucideWorkflow,
  default1454 as LucideWorm,
  default1923 as LucideWrapText,
  default1455 as LucideWrench,
  default1457 as LucideX,
  default1603 as LucideXCircle,
  default1456 as LucideXLineTop,
  default1746 as LucideXOctagon,
  default1906 as LucideXSquare,
  default1459 as LucideZap,
  default1458 as LucideZapOff,
  default1461 as LucideZodiacAquarius,
  default1460 as LucideZodiacAries,
  default1462 as LucideZodiacCancer,
  default1464 as LucideZodiacCapricorn,
  default1463 as LucideZodiacGemini,
  default1465 as LucideZodiacLeo,
  default1467 as LucideZodiacLibra,
  default1466 as LucideZodiacOphiuchus,
  default1468 as LucideZodiacPisces,
  default1469 as LucideZodiacSagittarius,
  default1470 as LucideZodiacScorpio,
  default1471 as LucideZodiacTaurus,
  default1472 as LucideZodiacVirgo,
  default1473 as LucideZoomIn,
  default1474 as LucideZoomOut
};
