// lucide-solid/dist/source/aliases/suffixed.jsx
/**
* @license lucide-solid v1.16.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/suffixed.ts
import { default as default2 } from "../icons/a-arrow-up";
import { default as default3 } from "../icons/a-arrow-down";
import { default as default4 } from "../icons/activity";
import { default as default5 } from "../icons/a-large-small";
import { default as default6 } from "../icons/accessibility";
import { default as default7 } from "../icons/air-vent";
import { default as default8 } from "../icons/airplay";
import { default as default9 } from "../icons/alarm-clock-off";
import { default as default10 } from "../icons/alarm-clock";
import { default as default11 } from "../icons/alarm-smoke";
import { default as default12 } from "../icons/album";
import { default as default13 } from "../icons/align-center-horizontal";
import { default as default14 } from "../icons/align-center-vertical";
import { default as default15 } from "../icons/align-end-horizontal";
import { default as default16 } from "../icons/align-end-vertical";
import { default as default17 } from "../icons/align-horizontal-distribute-center";
import { default as default18 } from "../icons/align-horizontal-distribute-start";
import { default as default19 } from "../icons/align-horizontal-distribute-end";
import { default as default20 } from "../icons/align-horizontal-justify-center";
import { default as default21 } from "../icons/align-horizontal-justify-end";
import { default as default22 } from "../icons/align-horizontal-justify-start";
import { default as default23 } from "../icons/align-horizontal-space-around";
import { default as default24 } from "../icons/align-start-horizontal";
import { default as default25 } from "../icons/align-vertical-distribute-center";
import { default as default26 } from "../icons/align-horizontal-space-between";
import { default as default27 } from "../icons/align-vertical-distribute-end";
import { default as default28 } from "../icons/align-start-vertical";
import { default as default29 } from "../icons/align-vertical-justify-center";
import { default as default30 } from "../icons/align-vertical-distribute-start";
import { default as default31 } from "../icons/align-vertical-justify-end";
import { default as default32 } from "../icons/align-vertical-justify-start";
import { default as default33 } from "../icons/align-vertical-space-around";
import { default as default34 } from "../icons/align-vertical-space-between";
import { default as default35 } from "../icons/ambulance";
import { default as default36 } from "../icons/ampersands";
import { default as default37 } from "../icons/amphora";
import { default as default38 } from "../icons/ampersand";
import { default as default39 } from "../icons/anchor";
import { default as default40 } from "../icons/angry";
import { default as default41 } from "../icons/annoyed";
import { default as default42 } from "../icons/anvil";
import { default as default43 } from "../icons/antenna";
import { default as default44 } from "../icons/aperture";
import { default as default45 } from "../icons/app-window";
import { default as default46 } from "../icons/app-window-mac";
import { default as default47 } from "../icons/apple";
import { default as default48 } from "../icons/archive-x";
import { default as default49 } from "../icons/archive-restore";
import { default as default50 } from "../icons/armchair";
import { default as default51 } from "../icons/archive";
import { default as default52 } from "../icons/arrow-big-down-dash";
import { default as default53 } from "../icons/arrow-big-down";
import { default as default54 } from "../icons/arrow-big-left-dash";
import { default as default55 } from "../icons/arrow-big-left";
import { default as default56 } from "../icons/arrow-big-right-dash";
import { default as default57 } from "../icons/arrow-big-right";
import { default as default58 } from "../icons/arrow-big-up-dash";
import { default as default59 } from "../icons/arrow-big-up";
import { default as default60 } from "../icons/arrow-down-from-line";
import { default as default61 } from "../icons/arrow-down-left";
import { default as default62 } from "../icons/arrow-down-narrow-wide";
import { default as default63 } from "../icons/arrow-down-right";
import { default as default64 } from "../icons/arrow-down-to-dot";
import { default as default65 } from "../icons/arrow-down-up";
import { default as default66 } from "../icons/arrow-down-to-line";
import { default as default67 } from "../icons/arrow-down";
import { default as default68 } from "../icons/arrow-left-from-line";
import { default as default69 } from "../icons/arrow-left-to-line";
import { default as default70 } from "../icons/arrow-left-right";
import { default as default71 } from "../icons/arrow-right-from-line";
import { default as default72 } from "../icons/arrow-left";
import { default as default73 } from "../icons/arrow-right-left";
import { default as default74 } from "../icons/arrow-right-to-line";
import { default as default75 } from "../icons/arrow-right";
import { default as default76 } from "../icons/arrow-up-down";
import { default as default77 } from "../icons/arrow-up-from-dot";
import { default as default78 } from "../icons/arrow-up-left";
import { default as default79 } from "../icons/arrow-up-from-line";
import { default as default80 } from "../icons/arrow-up-right";
import { default as default81 } from "../icons/arrow-up-to-line";
import { default as default82 } from "../icons/arrow-up-wide-narrow";
import { default as default83 } from "../icons/arrow-up";
import { default as default84 } from "../icons/arrows-up-from-line";
import { default as default85 } from "../icons/asterisk";
import { default as default86 } from "../icons/astroid";
import { default as default87 } from "../icons/atom";
import { default as default88 } from "../icons/at-sign";
import { default as default89 } from "../icons/audio-lines";
import { default as default90 } from "../icons/audio-waveform";
import { default as default91 } from "../icons/award";
import { default as default92 } from "../icons/axe";
import { default as default93 } from "../icons/baby";
import { default as default94 } from "../icons/backpack";
import { default as default95 } from "../icons/badge-alert";
import { default as default96 } from "../icons/badge-cent";
import { default as default97 } from "../icons/badge-dollar-sign";
import { default as default98 } from "../icons/badge-euro";
import { default as default99 } from "../icons/badge-info";
import { default as default100 } from "../icons/badge-indian-rupee";
import { default as default101 } from "../icons/badge-minus";
import { default as default102 } from "../icons/badge-japanese-yen";
import { default as default103 } from "../icons/badge-percent";
import { default as default104 } from "../icons/badge-pound-sterling";
import { default as default105 } from "../icons/badge-russian-ruble";
import { default as default106 } from "../icons/badge-plus";
import { default as default107 } from "../icons/badge-swiss-franc";
import { default as default108 } from "../icons/badge-x";
import { default as default109 } from "../icons/badge-turkish-lira";
import { default as default110 } from "../icons/badge";
import { default as default111 } from "../icons/baggage-claim";
import { default as default112 } from "../icons/balloon";
import { default as default113 } from "../icons/ban";
import { default as default114 } from "../icons/banana";
import { default as default115 } from "../icons/banknote-arrow-down";
import { default as default116 } from "../icons/bandage";
import { default as default117 } from "../icons/banknote-arrow-up";
import { default as default118 } from "../icons/banknote-x";
import { default as default119 } from "../icons/banknote";
import { default as default120 } from "../icons/barcode";
import { default as default121 } from "../icons/barrel";
import { default as default122 } from "../icons/baseline";
import { default as default123 } from "../icons/bath";
import { default as default124 } from "../icons/battery-charging";
import { default as default125 } from "../icons/battery-low";
import { default as default126 } from "../icons/battery-full";
import { default as default127 } from "../icons/battery-medium";
import { default as default128 } from "../icons/battery-plus";
import { default as default129 } from "../icons/battery-warning";
import { default as default130 } from "../icons/beaker";
import { default as default131 } from "../icons/battery";
import { default as default132 } from "../icons/bean-off";
import { default as default133 } from "../icons/bean";
import { default as default134 } from "../icons/bed-double";
import { default as default135 } from "../icons/bed-single";
import { default as default136 } from "../icons/beef-off";
import { default as default137 } from "../icons/bed";
import { default as default138 } from "../icons/beer-off";
import { default as default139 } from "../icons/beef";
import { default as default140 } from "../icons/beer";
import { default as default141 } from "../icons/bell-check";
import { default as default142 } from "../icons/bell-dot";
import { default as default143 } from "../icons/bell-electric";
import { default as default144 } from "../icons/bell-minus";
import { default as default145 } from "../icons/bell-off";
import { default as default146 } from "../icons/bell-plus";
import { default as default147 } from "../icons/bell-ring";
import { default as default148 } from "../icons/bell";
import { default as default149 } from "../icons/between-vertical-end";
import { default as default150 } from "../icons/between-vertical-start";
import { default as default151 } from "../icons/biceps-flexed";
import { default as default152 } from "../icons/bike";
import { default as default153 } from "../icons/binary";
import { default as default154 } from "../icons/binoculars";
import { default as default155 } from "../icons/biohazard";
import { default as default156 } from "../icons/birdhouse";
import { default as default157 } from "../icons/bird";
import { default as default158 } from "../icons/bitcoin";
import { default as default159 } from "../icons/blend";
import { default as default160 } from "../icons/blinds";
import { default as default161 } from "../icons/blender";
import { default as default162 } from "../icons/blocks";
import { default as default163 } from "../icons/bluetooth-connected";
import { default as default164 } from "../icons/bluetooth-searching";
import { default as default165 } from "../icons/bluetooth-off";
import { default as default166 } from "../icons/bluetooth";
import { default as default167 } from "../icons/bold";
import { default as default168 } from "../icons/bolt";
import { default as default169 } from "../icons/bomb";
import { default as default170 } from "../icons/bone";
import { default as default171 } from "../icons/book-a";
import { default as default172 } from "../icons/book-alert";
import { default as default173 } from "../icons/book-check";
import { default as default174 } from "../icons/book-audio";
import { default as default175 } from "../icons/book-copy";
import { default as default176 } from "../icons/book-down";
import { default as default177 } from "../icons/book-headphones";
import { default as default178 } from "../icons/book-image";
import { default as default179 } from "../icons/book-heart";
import { default as default180 } from "../icons/book-key";
import { default as default181 } from "../icons/book-marked";
import { default as default182 } from "../icons/book-lock";
import { default as default183 } from "../icons/book-minus";
import { default as default184 } from "../icons/book-open-text";
import { default as default185 } from "../icons/book-plus";
import { default as default186 } from "../icons/book-open-check";
import { default as default187 } from "../icons/book-open";
import { default as default188 } from "../icons/book-text";
import { default as default189 } from "../icons/book-search";
import { default as default190 } from "../icons/book-up-2";
import { default as default191 } from "../icons/book-type";
import { default as default192 } from "../icons/book-up";
import { default as default193 } from "../icons/book-user";
import { default as default194 } from "../icons/book";
import { default as default195 } from "../icons/book-x";
import { default as default196 } from "../icons/bookmark-check";
import { default as default197 } from "../icons/bookmark-minus";
import { default as default198 } from "../icons/bookmark-plus";
import { default as default199 } from "../icons/bookmark-off";
import { default as default200 } from "../icons/bookmark-x";
import { default as default201 } from "../icons/bookmark";
import { default as default202 } from "../icons/bot-message-square";
import { default as default203 } from "../icons/boom-box";
import { default as default204 } from "../icons/bot-off";
import { default as default205 } from "../icons/bot";
import { default as default206 } from "../icons/bottle-wine";
import { default as default207 } from "../icons/bow-arrow";
import { default as default208 } from "../icons/box";
import { default as default209 } from "../icons/boxes";
import { default as default210 } from "../icons/brackets";
import { default as default211 } from "../icons/brain-circuit";
import { default as default212 } from "../icons/brain";
import { default as default213 } from "../icons/brick-wall-fire";
import { default as default214 } from "../icons/brain-cog";
import { default as default215 } from "../icons/brick-wall-shield";
import { default as default216 } from "../icons/brick-wall";
import { default as default217 } from "../icons/briefcase-business";
import { default as default218 } from "../icons/briefcase-conveyor-belt";
import { default as default219 } from "../icons/briefcase";
import { default as default220 } from "../icons/briefcase-medical";
import { default as default221 } from "../icons/bring-to-front";
import { default as default222 } from "../icons/broccoli";
import { default as default223 } from "../icons/brush-cleaning";
import { default as default224 } from "../icons/brush";
import { default as default225 } from "../icons/bubbles";
import { default as default226 } from "../icons/bug-off";
import { default as default227 } from "../icons/bug-play";
import { default as default228 } from "../icons/bug";
import { default as default229 } from "../icons/building-2";
import { default as default230 } from "../icons/building";
import { default as default231 } from "../icons/bus-front";
import { default as default232 } from "../icons/cable-car";
import { default as default233 } from "../icons/cake-slice";
import { default as default234 } from "../icons/bus";
import { default as default235 } from "../icons/cable";
import { default as default236 } from "../icons/cake";
import { default as default237 } from "../icons/calculator";
import { default as default238 } from "../icons/calendar-1";
import { default as default239 } from "../icons/calendar-arrow-down";
import { default as default240 } from "../icons/calendar-arrow-up";
import { default as default241 } from "../icons/calendar-check-2";
import { default as default242 } from "../icons/calendar-clock";
import { default as default243 } from "../icons/calendar-check";
import { default as default244 } from "../icons/calendar-days";
import { default as default245 } from "../icons/calendar-cog";
import { default as default246 } from "../icons/calendar-fold";
import { default as default247 } from "../icons/calendar-heart";
import { default as default248 } from "../icons/calendar-minus";
import { default as default249 } from "../icons/calendar-off";
import { default as default250 } from "../icons/calendar-minus-2";
import { default as default251 } from "../icons/calendar-plus-2";
import { default as default252 } from "../icons/calendar-plus";
import { default as default253 } from "../icons/calendar-range";
import { default as default254 } from "../icons/calendar-sync";
import { default as default255 } from "../icons/calendar-search";
import { default as default256 } from "../icons/calendar-x-2";
import { default as default257 } from "../icons/calendar-x";
import { default as default258 } from "../icons/calendar";
import { default as default259 } from "../icons/calendars";
import { default as default260 } from "../icons/camera-off";
import { default as default261 } from "../icons/camera";
import { default as default262 } from "../icons/candy-cane";
import { default as default263 } from "../icons/candy-off";
import { default as default264 } from "../icons/cannabis-off";
import { default as default265 } from "../icons/candy";
import { default as default266 } from "../icons/cannabis";
import { default as default267 } from "../icons/car-front";
import { default as default268 } from "../icons/car-taxi-front";
import { default as default269 } from "../icons/captions-off";
import { default as default270 } from "../icons/car";
import { default as default271 } from "../icons/caravan";
import { default as default272 } from "../icons/card-sim";
import { default as default273 } from "../icons/case-lower";
import { default as default274 } from "../icons/carrot";
import { default as default275 } from "../icons/case-sensitive";
import { default as default276 } from "../icons/case-upper";
import { default as default277 } from "../icons/cassette-tape";
import { default as default278 } from "../icons/cast";
import { default as default279 } from "../icons/cat";
import { default as default280 } from "../icons/castle";
import { default as default281 } from "../icons/cctv-off";
import { default as default282 } from "../icons/cctv";
import { default as default283 } from "../icons/chart-bar-decreasing";
import { default as default284 } from "../icons/chart-bar-increasing";
import { default as default285 } from "../icons/chart-bar-stacked";
import { default as default286 } from "../icons/chart-column-decreasing";
import { default as default287 } from "../icons/chart-column-stacked";
import { default as default288 } from "../icons/chart-gantt";
import { default as default289 } from "../icons/chart-network";
import { default as default290 } from "../icons/chart-no-axes-column-decreasing";
import { default as default291 } from "../icons/chart-no-axes-combined";
import { default as default292 } from "../icons/chart-spline";
import { default as default293 } from "../icons/check-check";
import { default as default294 } from "../icons/check";
import { default as default295 } from "../icons/check-line";
import { default as default296 } from "../icons/chef-hat";
import { default as default297 } from "../icons/cherry";
import { default as default298 } from "../icons/chess-bishop";
import { default as default299 } from "../icons/chess-king";
import { default as default300 } from "../icons/chess-knight";
import { default as default301 } from "../icons/chess-pawn";
import { default as default302 } from "../icons/chevron-down";
import { default as default303 } from "../icons/chess-queen";
import { default as default304 } from "../icons/chess-rook";
import { default as default305 } from "../icons/chevron-first";
import { default as default306 } from "../icons/chevron-last";
import { default as default307 } from "../icons/chevron-left";
import { default as default308 } from "../icons/chevron-up";
import { default as default309 } from "../icons/chevron-right";
import { default as default310 } from "../icons/chevrons-down-up";
import { default as default311 } from "../icons/chevrons-left-right-ellipsis";
import { default as default312 } from "../icons/chevrons-down";
import { default as default313 } from "../icons/chevrons-left-right";
import { default as default314 } from "../icons/chevrons-left";
import { default as default315 } from "../icons/chevrons-right-left";
import { default as default316 } from "../icons/chevrons-right";
import { default as default317 } from "../icons/chevrons-up-down";
import { default as default318 } from "../icons/chevrons-up";
import { default as default319 } from "../icons/church";
import { default as default320 } from "../icons/cigarette-off";
import { default as default321 } from "../icons/cigarette";
import { default as default322 } from "../icons/circle-dashed";
import { default as default323 } from "../icons/circle-dollar-sign";
import { default as default324 } from "../icons/circle-dot-dashed";
import { default as default325 } from "../icons/circle-dot";
import { default as default326 } from "../icons/circle-ellipsis";
import { default as default327 } from "../icons/circle-fading-arrow-up";
import { default as default328 } from "../icons/circle-equal";
import { default as default329 } from "../icons/circle-fading-plus";
import { default as default330 } from "../icons/circle-off";
import { default as default331 } from "../icons/circle-pile";
import { default as default332 } from "../icons/circle-pound-sterling";
import { default as default333 } from "../icons/circle-slash";
import { default as default334 } from "../icons/circle-star";
import { default as default335 } from "../icons/circle-small";
import { default as default336 } from "../icons/circle";
import { default as default337 } from "../icons/circuit-board";
import { default as default338 } from "../icons/citrus";
import { default as default339 } from "../icons/clipboard-check";
import { default as default340 } from "../icons/clapperboard";
import { default as default341 } from "../icons/clipboard-clock";
import { default as default342 } from "../icons/clipboard-copy";
import { default as default343 } from "../icons/clipboard-list";
import { default as default344 } from "../icons/clipboard-minus";
import { default as default345 } from "../icons/clipboard-paste";
import { default as default346 } from "../icons/clipboard-plus";
import { default as default347 } from "../icons/clipboard-type";
import { default as default348 } from "../icons/clipboard-x";
import { default as default349 } from "../icons/clipboard";
import { default as default350 } from "../icons/clock-10";
import { default as default351 } from "../icons/clock-1";
import { default as default352 } from "../icons/clock-11";
import { default as default353 } from "../icons/clock-12";
import { default as default354 } from "../icons/clock-2";
import { default as default355 } from "../icons/clock-3";
import { default as default356 } from "../icons/clock-4";
import { default as default357 } from "../icons/clock-5";
import { default as default358 } from "../icons/clock-6";
import { default as default359 } from "../icons/clock-7";
import { default as default360 } from "../icons/clock-8";
import { default as default361 } from "../icons/clock-alert";
import { default as default362 } from "../icons/clock-arrow-down";
import { default as default363 } from "../icons/clock-arrow-up";
import { default as default364 } from "../icons/clock-9";
import { default as default365 } from "../icons/clock-check";
import { default as default366 } from "../icons/clock-fading";
import { default as default367 } from "../icons/clock-plus";
import { default as default368 } from "../icons/closed-caption";
import { default as default369 } from "../icons/clock";
import { default as default370 } from "../icons/cloud-alert";
import { default as default371 } from "../icons/cloud-backup";
import { default as default372 } from "../icons/cloud-check";
import { default as default373 } from "../icons/cloud-cog";
import { default as default374 } from "../icons/cloud-drizzle";
import { default as default375 } from "../icons/cloud-fog";
import { default as default376 } from "../icons/cloud-hail";
import { default as default377 } from "../icons/cloud-lightning";
import { default as default378 } from "../icons/cloud-moon-rain";
import { default as default379 } from "../icons/cloud-off";
import { default as default380 } from "../icons/cloud-moon";
import { default as default381 } from "../icons/cloud-rain-wind";
import { default as default382 } from "../icons/cloud-rain";
import { default as default383 } from "../icons/cloud-snow";
import { default as default384 } from "../icons/cloud-sun-rain";
import { default as default385 } from "../icons/cloud-sun";
import { default as default386 } from "../icons/cloud-sync";
import { default as default387 } from "../icons/cloud";
import { default as default388 } from "../icons/cloudy";
import { default as default389 } from "../icons/clover";
import { default as default390 } from "../icons/club";
import { default as default391 } from "../icons/coffee";
import { default as default392 } from "../icons/code";
import { default as default393 } from "../icons/cog";
import { default as default394 } from "../icons/coins";
import { default as default395 } from "../icons/columns-4";
import { default as default396 } from "../icons/combine";
import { default as default397 } from "../icons/command";
import { default as default398 } from "../icons/compass";
import { default as default399 } from "../icons/component";
import { default as default400 } from "../icons/concierge-bell";
import { default as default401 } from "../icons/cone";
import { default as default402 } from "../icons/computer";
import { default as default403 } from "../icons/construction";
import { default as default404 } from "../icons/contact";
import { default as default405 } from "../icons/container";
import { default as default406 } from "../icons/contrast";
import { default as default407 } from "../icons/cookie";
import { default as default408 } from "../icons/cooking-pot";
import { default as default409 } from "../icons/copy-check";
import { default as default410 } from "../icons/copy-minus";
import { default as default411 } from "../icons/copy-plus";
import { default as default412 } from "../icons/copy-x";
import { default as default413 } from "../icons/copy-slash";
import { default as default414 } from "../icons/copy";
import { default as default415 } from "../icons/copyleft";
import { default as default416 } from "../icons/corner-down-left";
import { default as default417 } from "../icons/copyright";
import { default as default418 } from "../icons/corner-down-right";
import { default as default419 } from "../icons/corner-left-down";
import { default as default420 } from "../icons/corner-right-down";
import { default as default421 } from "../icons/corner-right-up";
import { default as default422 } from "../icons/corner-left-up";
import { default as default423 } from "../icons/corner-up-left";
import { default as default424 } from "../icons/cpu";
import { default as default425 } from "../icons/corner-up-right";
import { default as default426 } from "../icons/credit-card";
import { default as default427 } from "../icons/croissant";
import { default as default428 } from "../icons/creative-commons";
import { default as default429 } from "../icons/crosshair";
import { default as default430 } from "../icons/crop";
import { default as default431 } from "../icons/crown";
import { default as default432 } from "../icons/cross";
import { default as default433 } from "../icons/cuboid";
import { default as default434 } from "../icons/cup-soda";
import { default as default435 } from "../icons/currency";
import { default as default436 } from "../icons/cylinder";
import { default as default437 } from "../icons/dam";
import { default as default438 } from "../icons/database-backup";
import { default as default439 } from "../icons/database-search";
import { default as default440 } from "../icons/database-zap";
import { default as default441 } from "../icons/database";
import { default as default442 } from "../icons/decimals-arrow-left";
import { default as default443 } from "../icons/decimals-arrow-right";
import { default as default444 } from "../icons/delete";
import { default as default445 } from "../icons/dessert";
import { default as default446 } from "../icons/diamond-minus";
import { default as default447 } from "../icons/diameter";
import { default as default448 } from "../icons/diamond-plus";
import { default as default449 } from "../icons/diamond";
import { default as default450 } from "../icons/dice-1";
import { default as default451 } from "../icons/dice-2";
import { default as default452 } from "../icons/dice-3";
import { default as default453 } from "../icons/dice-4";
import { default as default454 } from "../icons/dice-5";
import { default as default455 } from "../icons/dice-6";
import { default as default456 } from "../icons/dices";
import { default as default457 } from "../icons/diff";
import { default as default458 } from "../icons/disc-2";
import { default as default459 } from "../icons/disc-album";
import { default as default460 } from "../icons/disc-3";
import { default as default461 } from "../icons/disc";
import { default as default462 } from "../icons/dna-off";
import { default as default463 } from "../icons/divide";
import { default as default464 } from "../icons/dna";
import { default as default465 } from "../icons/dock";
import { default as default466 } from "../icons/dog";
import { default as default467 } from "../icons/dollar-sign";
import { default as default468 } from "../icons/donut";
import { default as default469 } from "../icons/door-closed-locked";
import { default as default470 } from "../icons/door-closed";
import { default as default471 } from "../icons/door-open";
import { default as default472 } from "../icons/dot";
import { default as default473 } from "../icons/download";
import { default as default474 } from "../icons/drafting-compass";
import { default as default475 } from "../icons/drama";
import { default as default476 } from "../icons/drill";
import { default as default477 } from "../icons/drone";
import { default as default478 } from "../icons/droplet-off";
import { default as default479 } from "../icons/droplet";
import { default as default480 } from "../icons/droplets";
import { default as default481 } from "../icons/drum";
import { default as default482 } from "../icons/drumstick";
import { default as default483 } from "../icons/dumbbell";
import { default as default484 } from "../icons/ear-off";
import { default as default485 } from "../icons/ear";
import { default as default486 } from "../icons/earth-lock";
import { default as default487 } from "../icons/egg-fried";
import { default as default488 } from "../icons/egg-off";
import { default as default489 } from "../icons/eclipse";
import { default as default490 } from "../icons/egg";
import { default as default491 } from "../icons/ellipse";
import { default as default492 } from "../icons/equal-approximately";
import { default as default493 } from "../icons/equal-not";
import { default as default494 } from "../icons/equal";
import { default as default495 } from "../icons/eraser";
import { default as default496 } from "../icons/ethernet-port";
import { default as default497 } from "../icons/euro";
import { default as default498 } from "../icons/expand";
import { default as default499 } from "../icons/ev-charger";
import { default as default500 } from "../icons/external-link";
import { default as default501 } from "../icons/eye-closed";
import { default as default502 } from "../icons/eye-off";
import { default as default503 } from "../icons/factory";
import { default as default504 } from "../icons/eye";
import { default as default505 } from "../icons/fan";
import { default as default506 } from "../icons/fast-forward";
import { default as default507 } from "../icons/feather";
import { default as default508 } from "../icons/fence";
import { default as default509 } from "../icons/ferris-wheel";
import { default as default510 } from "../icons/file-archive";
import { default as default511 } from "../icons/file-box";
import { default as default512 } from "../icons/file-check";
import { default as default513 } from "../icons/file-clock";
import { default as default514 } from "../icons/file-code";
import { default as default515 } from "../icons/file-diff";
import { default as default516 } from "../icons/file-down";
import { default as default517 } from "../icons/file-digit";
import { default as default518 } from "../icons/file-heart";
import { default as default519 } from "../icons/file-image";
import { default as default520 } from "../icons/file-input";
import { default as default521 } from "../icons/file-minus";
import { default as default522 } from "../icons/file-music";
import { default as default523 } from "../icons/file-output";
import { default as default524 } from "../icons/file-plus";
import { default as default525 } from "../icons/file-scan";
import { default as default526 } from "../icons/file-search";
import { default as default527 } from "../icons/file-spreadsheet";
import { default as default528 } from "../icons/file-sliders";
import { default as default529 } from "../icons/file-stack";
import { default as default530 } from "../icons/file-symlink";
import { default as default531 } from "../icons/file-terminal";
import { default as default532 } from "../icons/file-text";
import { default as default533 } from "../icons/file-type";
import { default as default534 } from "../icons/file-up";
import { default as default535 } from "../icons/file-user";
import { default as default536 } from "../icons/file-volume";
import { default as default537 } from "../icons/file-x";
import { default as default538 } from "../icons/file";
import { default as default539 } from "../icons/files";
import { default as default540 } from "../icons/film";
import { default as default541 } from "../icons/fish-off";
import { default as default542 } from "../icons/fire-extinguisher";
import { default as default543 } from "../icons/fish";
import { default as default544 } from "../icons/fishing-hook";
import { default as default545 } from "../icons/fish-symbol";
import { default as default546 } from "../icons/fishing-rod";
import { default as default547 } from "../icons/flag-off";
import { default as default548 } from "../icons/flag-triangle-left";
import { default as default549 } from "../icons/flag-triangle-right";
import { default as default550 } from "../icons/flag";
import { default as default551 } from "../icons/flame-kindling";
import { default as default552 } from "../icons/flame";
import { default as default553 } from "../icons/flashlight-off";
import { default as default554 } from "../icons/flashlight";
import { default as default555 } from "../icons/flask-conical-off";
import { default as default556 } from "../icons/flask-conical";
import { default as default557 } from "../icons/flask-round";
import { default as default558 } from "../icons/flip-horizontal-2";
import { default as default559 } from "../icons/flip-vertical-2";
import { default as default560 } from "../icons/flower-2";
import { default as default561 } from "../icons/flower";
import { default as default562 } from "../icons/focus";
import { default as default563 } from "../icons/fold-horizontal";
import { default as default564 } from "../icons/fold-vertical";
import { default as default565 } from "../icons/folder-archive";
import { default as default566 } from "../icons/folder-check";
import { default as default567 } from "../icons/folder-clock";
import { default as default568 } from "../icons/folder-bookmark";
import { default as default569 } from "../icons/folder-closed";
import { default as default570 } from "../icons/folder-code";
import { default as default571 } from "../icons/folder-dot";
import { default as default572 } from "../icons/folder-down";
import { default as default573 } from "../icons/folder-git-2";
import { default as default574 } from "../icons/folder-git";
import { default as default575 } from "../icons/folder-heart";
import { default as default576 } from "../icons/folder-input";
import { default as default577 } from "../icons/folder-kanban";
import { default as default578 } from "../icons/folder-key";
import { default as default579 } from "../icons/folder-lock";
import { default as default580 } from "../icons/folder-minus";
import { default as default581 } from "../icons/folder-open-dot";
import { default as default582 } from "../icons/folder-output";
import { default as default583 } from "../icons/folder-open";
import { default as default584 } from "../icons/folder-root";
import { default as default585 } from "../icons/folder-plus";
import { default as default586 } from "../icons/folder-search-2";
import { default as default587 } from "../icons/folder-symlink";
import { default as default588 } from "../icons/folder-search";
import { default as default589 } from "../icons/folder-sync";
import { default as default590 } from "../icons/folder-tree";
import { default as default591 } from "../icons/folder-up";
import { default as default592 } from "../icons/folder";
import { default as default593 } from "../icons/folder-x";
import { default as default594 } from "../icons/folders";
import { default as default595 } from "../icons/footprints";
import { default as default596 } from "../icons/forklift";
import { default as default597 } from "../icons/forward";
import { default as default598 } from "../icons/form";
import { default as default599 } from "../icons/frame";
import { default as default600 } from "../icons/frown";
import { default as default601 } from "../icons/fuel";
import { default as default602 } from "../icons/fullscreen";
import { default as default603 } from "../icons/funnel-plus";
import { default as default604 } from "../icons/gallery-horizontal-end";
import { default as default605 } from "../icons/gallery-thumbnails";
import { default as default606 } from "../icons/gallery-horizontal";
import { default as default607 } from "../icons/gallery-vertical";
import { default as default608 } from "../icons/gallery-vertical-end";
import { default as default609 } from "../icons/gamepad-2";
import { default as default610 } from "../icons/gamepad-directional";
import { default as default611 } from "../icons/gamepad";
import { default as default612 } from "../icons/gauge";
import { default as default613 } from "../icons/gavel";
import { default as default614 } from "../icons/gem";
import { default as default615 } from "../icons/georgian-lari";
import { default as default616 } from "../icons/ghost";
import { default as default617 } from "../icons/gift";
import { default as default618 } from "../icons/git-branch-minus";
import { default as default619 } from "../icons/git-branch-plus";
import { default as default620 } from "../icons/git-branch";
import { default as default621 } from "../icons/git-commit-vertical";
import { default as default622 } from "../icons/git-compare-arrows";
import { default as default623 } from "../icons/git-compare";
import { default as default624 } from "../icons/git-graph";
import { default as default625 } from "../icons/git-fork";
import { default as default626 } from "../icons/git-merge-conflict";
import { default as default627 } from "../icons/git-merge";
import { default as default628 } from "../icons/git-pull-request-closed";
import { default as default629 } from "../icons/git-pull-request-arrow";
import { default as default630 } from "../icons/git-pull-request-create-arrow";
import { default as default631 } from "../icons/git-pull-request-create";
import { default as default632 } from "../icons/git-pull-request-draft";
import { default as default633 } from "../icons/git-pull-request";
import { default as default634 } from "../icons/glass-water";
import { default as default635 } from "../icons/globe-lock";
import { default as default636 } from "../icons/glasses";
import { default as default637 } from "../icons/globe-off";
import { default as default638 } from "../icons/globe-x";
import { default as default639 } from "../icons/globe";
import { default as default640 } from "../icons/goal";
import { default as default641 } from "../icons/gpu";
import { default as default642 } from "../icons/grape";
import { default as default643 } from "../icons/graduation-cap";
import { default as default644 } from "../icons/grid-3x2";
import { default as default645 } from "../icons/grip-horizontal";
import { default as default646 } from "../icons/grip-vertical";
import { default as default647 } from "../icons/group";
import { default as default648 } from "../icons/grip";
import { default as default649 } from "../icons/guitar";
import { default as default650 } from "../icons/ham";
import { default as default651 } from "../icons/hamburger";
import { default as default652 } from "../icons/hammer";
import { default as default653 } from "../icons/hand-coins";
import { default as default654 } from "../icons/hand-fist";
import { default as default655 } from "../icons/hand-heart";
import { default as default656 } from "../icons/hand-metal";
import { default as default657 } from "../icons/hand-platter";
import { default as default658 } from "../icons/hand";
import { default as default659 } from "../icons/handbag";
import { default as default660 } from "../icons/handshake";
import { default as default661 } from "../icons/hard-drive-download";
import { default as default662 } from "../icons/hard-drive-upload";
import { default as default663 } from "../icons/hard-drive";
import { default as default664 } from "../icons/hard-hat";
import { default as default665 } from "../icons/hash";
import { default as default666 } from "../icons/hat-glasses";
import { default as default667 } from "../icons/haze";
import { default as default668 } from "../icons/hd";
import { default as default669 } from "../icons/hdmi-port";
import { default as default670 } from "../icons/heading-1";
import { default as default671 } from "../icons/heading-2";
import { default as default672 } from "../icons/heading-3";
import { default as default673 } from "../icons/heading-4";
import { default as default674 } from "../icons/heading-5";
import { default as default675 } from "../icons/heading-6";
import { default as default676 } from "../icons/headphone-off";
import { default as default677 } from "../icons/heading";
import { default as default678 } from "../icons/headphones";
import { default as default679 } from "../icons/headset";
import { default as default680 } from "../icons/heart-crack";
import { default as default681 } from "../icons/heart-handshake";
import { default as default682 } from "../icons/heart-off";
import { default as default683 } from "../icons/heart-minus";
import { default as default684 } from "../icons/heart-plus";
import { default as default685 } from "../icons/heart-pulse";
import { default as default686 } from "../icons/heart-x";
import { default as default687 } from "../icons/heart";
import { default as default688 } from "../icons/helicopter";
import { default as default689 } from "../icons/heater";
import { default as default690 } from "../icons/hexagon";
import { default as default691 } from "../icons/highlighter";
import { default as default692 } from "../icons/hop-off";
import { default as default693 } from "../icons/history";
import { default as default694 } from "../icons/hop";
import { default as default695 } from "../icons/hotel";
import { default as default696 } from "../icons/hospital";
import { default as default697 } from "../icons/hourglass";
import { default as default698 } from "../icons/house-heart";
import { default as default699 } from "../icons/house-plug";
import { default as default700 } from "../icons/house-wifi";
import { default as default701 } from "../icons/house-plus";
import { default as default702 } from "../icons/id-card-lanyard";
import { default as default703 } from "../icons/id-card";
import { default as default704 } from "../icons/image-down";
import { default as default705 } from "../icons/image-minus";
import { default as default706 } from "../icons/image-play";
import { default as default707 } from "../icons/image-off";
import { default as default708 } from "../icons/image-plus";
import { default as default709 } from "../icons/image-up";
import { default as default710 } from "../icons/image-upscale";
import { default as default711 } from "../icons/image";
import { default as default712 } from "../icons/images";
import { default as default713 } from "../icons/import";
import { default as default714 } from "../icons/inbox";
import { default as default715 } from "../icons/indian-rupee";
import { default as default716 } from "../icons/info";
import { default as default717 } from "../icons/infinity";
import { default as default718 } from "../icons/inspection-panel";
import { default as default719 } from "../icons/italic";
import { default as default720 } from "../icons/iteration-ccw";
import { default as default721 } from "../icons/iteration-cw";
import { default as default722 } from "../icons/japanese-yen";
import { default as default723 } from "../icons/joystick";
import { default as default724 } from "../icons/kayak";
import { default as default725 } from "../icons/key-round";
import { default as default726 } from "../icons/kanban";
import { default as default727 } from "../icons/key-square";
import { default as default728 } from "../icons/key";
import { default as default729 } from "../icons/keyboard-music";
import { default as default730 } from "../icons/keyboard";
import { default as default731 } from "../icons/keyboard-off";
import { default as default732 } from "../icons/lamp-ceiling";
import { default as default733 } from "../icons/lamp-desk";
import { default as default734 } from "../icons/lamp-floor";
import { default as default735 } from "../icons/lamp-wall-down";
import { default as default736 } from "../icons/lamp-wall-up";
import { default as default737 } from "../icons/lamp";
import { default as default738 } from "../icons/land-plot";
import { default as default739 } from "../icons/laptop-minimal-check";
import { default as default740 } from "../icons/landmark";
import { default as default741 } from "../icons/languages";
import { default as default742 } from "../icons/laptop";
import { default as default743 } from "../icons/lasso-select";
import { default as default744 } from "../icons/lasso";
import { default as default745 } from "../icons/laugh";
import { default as default746 } from "../icons/layers-2";
import { default as default747 } from "../icons/layers-plus";
import { default as default748 } from "../icons/layers-minus";
import { default as default749 } from "../icons/layout-dashboard";
import { default as default750 } from "../icons/layout-grid";
import { default as default751 } from "../icons/layout-list";
import { default as default752 } from "../icons/layout-panel-left";
import { default as default753 } from "../icons/layout-panel-top";
import { default as default754 } from "../icons/layout-template";
import { default as default755 } from "../icons/leafy-green";
import { default as default756 } from "../icons/leaf";
import { default as default757 } from "../icons/lectern";
import { default as default758 } from "../icons/lens-concave";
import { default as default759 } from "../icons/lens-convex";
import { default as default760 } from "../icons/library";
import { default as default761 } from "../icons/library-big";
import { default as default762 } from "../icons/life-buoy";
import { default as default763 } from "../icons/ligature";
import { default as default764 } from "../icons/lightbulb-off";
import { default as default765 } from "../icons/lightbulb";
import { default as default766 } from "../icons/line-dot-right-horizontal";
import { default as default767 } from "../icons/line-squiggle";
import { default as default768 } from "../icons/line-style";
import { default as default769 } from "../icons/link-2-off";
import { default as default770 } from "../icons/link-2";
import { default as default771 } from "../icons/list-check";
import { default as default772 } from "../icons/link";
import { default as default773 } from "../icons/list-checks";
import { default as default774 } from "../icons/list-chevrons-down-up";
import { default as default775 } from "../icons/list-chevrons-up-down";
import { default as default776 } from "../icons/list-end";
import { default as default777 } from "../icons/list-collapse";
import { default as default778 } from "../icons/list-filter-plus";
import { default as default779 } from "../icons/list-filter";
import { default as default780 } from "../icons/list-minus";
import { default as default781 } from "../icons/list-music";
import { default as default782 } from "../icons/list-ordered";
import { default as default783 } from "../icons/list-plus";
import { default as default784 } from "../icons/list-restart";
import { default as default785 } from "../icons/list-start";
import { default as default786 } from "../icons/list-todo";
import { default as default787 } from "../icons/list-tree";
import { default as default788 } from "../icons/list-video";
import { default as default789 } from "../icons/list-x";
import { default as default790 } from "../icons/list";
import { default as default791 } from "../icons/loader-pinwheel";
import { default as default792 } from "../icons/loader";
import { default as default793 } from "../icons/locate-fixed";
import { default as default794 } from "../icons/locate-off";
import { default as default795 } from "../icons/locate";
import { default as default796 } from "../icons/lock-keyhole";
import { default as default797 } from "../icons/lock";
import { default as default798 } from "../icons/log-in";
import { default as default799 } from "../icons/logs";
import { default as default800 } from "../icons/log-out";
import { default as default801 } from "../icons/lollipop";
import { default as default802 } from "../icons/luggage";
import { default as default803 } from "../icons/magnet";
import { default as default804 } from "../icons/mail-check";
import { default as default805 } from "../icons/mail-minus";
import { default as default806 } from "../icons/mail-open";
import { default as default807 } from "../icons/mail-plus";
import { default as default808 } from "../icons/mail-search";
import { default as default809 } from "../icons/mail-warning";
import { default as default810 } from "../icons/mail-x";
import { default as default811 } from "../icons/mail";
import { default as default812 } from "../icons/mailbox";
import { default as default813 } from "../icons/mails";
import { default as default814 } from "../icons/map-pin-check-inside";
import { default as default815 } from "../icons/map-minus";
import { default as default816 } from "../icons/map-pin-check";
import { default as default817 } from "../icons/map-pin-house";
import { default as default818 } from "../icons/map-pin-minus-inside";
import { default as default819 } from "../icons/map-pin-minus";
import { default as default820 } from "../icons/map-pin-off";
import { default as default821 } from "../icons/map-pin-plus-inside";
import { default as default822 } from "../icons/map-pin-plus";
import { default as default823 } from "../icons/map-pin-search";
import { default as default824 } from "../icons/map-pin-x-inside";
import { default as default825 } from "../icons/map-pin-x";
import { default as default826 } from "../icons/map-pin";
import { default as default827 } from "../icons/map-pinned";
import { default as default828 } from "../icons/map-plus";
import { default as default829 } from "../icons/map";
import { default as default830 } from "../icons/mars-stroke";
import { default as default831 } from "../icons/martini";
import { default as default832 } from "../icons/mars";
import { default as default833 } from "../icons/maximize-2";
import { default as default834 } from "../icons/medal";
import { default as default835 } from "../icons/maximize";
import { default as default836 } from "../icons/megaphone";
import { default as default837 } from "../icons/megaphone-off";
import { default as default838 } from "../icons/meh";
import { default as default839 } from "../icons/menu";
import { default as default840 } from "../icons/memory-stick";
import { default as default841 } from "../icons/merge";
import { default as default842 } from "../icons/message-circle-check";
import { default as default843 } from "../icons/message-circle-code";
import { default as default844 } from "../icons/message-circle-dashed";
import { default as default845 } from "../icons/message-circle-heart";
import { default as default846 } from "../icons/message-circle-off";
import { default as default847 } from "../icons/message-circle-more";
import { default as default848 } from "../icons/message-circle-plus";
import { default as default849 } from "../icons/message-circle-reply";
import { default as default850 } from "../icons/message-circle-warning";
import { default as default851 } from "../icons/message-circle-x";
import { default as default852 } from "../icons/message-circle";
import { default as default853 } from "../icons/message-square-check";
import { default as default854 } from "../icons/message-square-dashed";
import { default as default855 } from "../icons/message-square-code";
import { default as default856 } from "../icons/message-square-diff";
import { default as default857 } from "../icons/message-square-dot";
import { default as default858 } from "../icons/message-square-heart";
import { default as default859 } from "../icons/message-square-lock";
import { default as default860 } from "../icons/message-square-more";
import { default as default861 } from "../icons/message-square-quote";
import { default as default862 } from "../icons/message-square-off";
import { default as default863 } from "../icons/message-square-reply";
import { default as default864 } from "../icons/message-square-share";
import { default as default865 } from "../icons/message-square-plus";
import { default as default866 } from "../icons/message-square-text";
import { default as default867 } from "../icons/message-square-warning";
import { default as default868 } from "../icons/message-square-x";
import { default as default869 } from "../icons/message-square";
import { default as default870 } from "../icons/messages-square";
import { default as default871 } from "../icons/metronome";
import { default as default872 } from "../icons/mic-off";
import { default as default873 } from "../icons/microchip";
import { default as default874 } from "../icons/mic";
import { default as default875 } from "../icons/microscope";
import { default as default876 } from "../icons/microwave";
import { default as default877 } from "../icons/milestone";
import { default as default878 } from "../icons/milk-off";
import { default as default879 } from "../icons/milk";
import { default as default880 } from "../icons/minimize-2";
import { default as default881 } from "../icons/minimize";
import { default as default882 } from "../icons/mirror-rectangular";
import { default as default883 } from "../icons/minus";
import { default as default884 } from "../icons/mirror-round";
import { default as default885 } from "../icons/monitor-check";
import { default as default886 } from "../icons/monitor-cloud";
import { default as default887 } from "../icons/monitor-cog";
import { default as default888 } from "../icons/monitor-dot";
import { default as default889 } from "../icons/monitor-down";
import { default as default890 } from "../icons/monitor-off";
import { default as default891 } from "../icons/monitor-pause";
import { default as default892 } from "../icons/monitor-play";
import { default as default893 } from "../icons/monitor-smartphone";
import { default as default894 } from "../icons/monitor-stop";
import { default as default895 } from "../icons/monitor-speaker";
import { default as default896 } from "../icons/monitor-up";
import { default as default897 } from "../icons/monitor";
import { default as default898 } from "../icons/monitor-x";
import { default as default899 } from "../icons/moon-star";
import { default as default900 } from "../icons/moon";
import { default as default901 } from "../icons/motorbike";
import { default as default902 } from "../icons/mountain-snow";
import { default as default903 } from "../icons/mountain";
import { default as default904 } from "../icons/mouse-left";
import { default as default905 } from "../icons/mouse-pointer-2-off";
import { default as default906 } from "../icons/mouse-off";
import { default as default907 } from "../icons/mouse-pointer-2";
import { default as default908 } from "../icons/mouse-pointer-ban";
import { default as default909 } from "../icons/mouse-pointer-click";
import { default as default910 } from "../icons/mouse-pointer";
import { default as default911 } from "../icons/mouse-right";
import { default as default912 } from "../icons/mouse";
import { default as default913 } from "../icons/move-diagonal-2";
import { default as default914 } from "../icons/move-diagonal";
import { default as default915 } from "../icons/move-down-left";
import { default as default916 } from "../icons/move-down-right";
import { default as default917 } from "../icons/move-horizontal";
import { default as default918 } from "../icons/move-down";
import { default as default919 } from "../icons/move-right";
import { default as default920 } from "../icons/move-left";
import { default as default921 } from "../icons/move-up-left";
import { default as default922 } from "../icons/move-up-right";
import { default as default923 } from "../icons/move-up";
import { default as default924 } from "../icons/move";
import { default as default925 } from "../icons/move-vertical";
import { default as default926 } from "../icons/music-2";
import { default as default927 } from "../icons/music-3";
import { default as default928 } from "../icons/music-4";
import { default as default929 } from "../icons/navigation-2-off";
import { default as default930 } from "../icons/navigation-2";
import { default as default931 } from "../icons/music";
import { default as default932 } from "../icons/navigation-off";
import { default as default933 } from "../icons/navigation";
import { default as default934 } from "../icons/network";
import { default as default935 } from "../icons/newspaper";
import { default as default936 } from "../icons/nfc";
import { default as default937 } from "../icons/non-binary";
import { default as default938 } from "../icons/notebook-pen";
import { default as default939 } from "../icons/notebook-tabs";
import { default as default940 } from "../icons/notebook-text";
import { default as default941 } from "../icons/notepad-text-dashed";
import { default as default942 } from "../icons/notepad-text";
import { default as default943 } from "../icons/notebook";
import { default as default944 } from "../icons/nut-off";
import { default as default945 } from "../icons/nut";
import { default as default946 } from "../icons/octagon-minus";
import { default as default947 } from "../icons/octagon";
import { default as default948 } from "../icons/omega";
import { default as default949 } from "../icons/orbit";
import { default as default950 } from "../icons/option";
import { default as default951 } from "../icons/origami";
import { default as default952 } from "../icons/package-2";
import { default as default953 } from "../icons/package-minus";
import { default as default954 } from "../icons/package-open";
import { default as default955 } from "../icons/package-check";
import { default as default956 } from "../icons/package-plus";
import { default as default957 } from "../icons/package-search";
import { default as default958 } from "../icons/package-x";
import { default as default959 } from "../icons/paint-bucket";
import { default as default960 } from "../icons/package";
import { default as default961 } from "../icons/paint-roller";
import { default as default962 } from "../icons/paintbrush";
import { default as default963 } from "../icons/palette";
import { default as default964 } from "../icons/panda";
import { default as default965 } from "../icons/panel-bottom-close";
import { default as default966 } from "../icons/panel-bottom-open";
import { default as default967 } from "../icons/panel-bottom";
import { default as default968 } from "../icons/panel-left-right-dashed";
import { default as default969 } from "../icons/panel-right-close";
import { default as default970 } from "../icons/panel-right-open";
import { default as default971 } from "../icons/panel-top-bottom-dashed";
import { default as default972 } from "../icons/panel-top-close";
import { default as default973 } from "../icons/panel-right";
import { default as default974 } from "../icons/panel-top-open";
import { default as default975 } from "../icons/panel-top";
import { default as default976 } from "../icons/panels-left-bottom";
import { default as default977 } from "../icons/panels-right-bottom";
import { default as default978 } from "../icons/paperclip";
import { default as default979 } from "../icons/parentheses";
import { default as default980 } from "../icons/party-popper";
import { default as default981 } from "../icons/parking-meter";
import { default as default982 } from "../icons/paw-print";
import { default as default983 } from "../icons/pause";
import { default as default984 } from "../icons/pen-off";
import { default as default985 } from "../icons/pc-case";
import { default as default986 } from "../icons/pen-tool";
import { default as default987 } from "../icons/pencil-line";
import { default as default988 } from "../icons/pencil-off";
import { default as default989 } from "../icons/pencil-ruler";
import { default as default990 } from "../icons/pencil";
import { default as default991 } from "../icons/pentagon";
import { default as default992 } from "../icons/percent";
import { default as default993 } from "../icons/person-standing";
import { default as default994 } from "../icons/philippine-peso";
import { default as default995 } from "../icons/phone-call";
import { default as default996 } from "../icons/phone-forwarded";
import { default as default997 } from "../icons/phone-incoming";
import { default as default998 } from "../icons/phone-off";
import { default as default999 } from "../icons/phone-missed";
import { default as default1000 } from "../icons/phone-outgoing";
import { default as default1001 } from "../icons/phone";
import { default as default1002 } from "../icons/pi";
import { default as default1003 } from "../icons/piano";
import { default as default1004 } from "../icons/pickaxe";
import { default as default1005 } from "../icons/picture-in-picture-2";
import { default as default1006 } from "../icons/picture-in-picture";
import { default as default1007 } from "../icons/piggy-bank";
import { default as default1008 } from "../icons/pilcrow-left";
import { default as default1009 } from "../icons/pilcrow-right";
import { default as default1010 } from "../icons/pilcrow";
import { default as default1011 } from "../icons/pill-bottle";
import { default as default1012 } from "../icons/pill";
import { default as default1013 } from "../icons/pipette";
import { default as default1014 } from "../icons/pin-off";
import { default as default1015 } from "../icons/pin";
import { default as default1016 } from "../icons/pizza";
import { default as default1017 } from "../icons/plane-takeoff";
import { default as default1018 } from "../icons/plane-landing";
import { default as default1019 } from "../icons/plane";
import { default as default1020 } from "../icons/plug-2";
import { default as default1021 } from "../icons/play";
import { default as default1022 } from "../icons/plug";
import { default as default1023 } from "../icons/plus";
import { default as default1024 } from "../icons/pocket-knife";
import { default as default1025 } from "../icons/podcast";
import { default as default1026 } from "../icons/pointer";
import { default as default1027 } from "../icons/pointer-off";
import { default as default1028 } from "../icons/popcorn";
import { default as default1029 } from "../icons/popsicle";
import { default as default1030 } from "../icons/pound-sterling";
import { default as default1031 } from "../icons/power-off";
import { default as default1032 } from "../icons/power";
import { default as default1033 } from "../icons/presentation";
import { default as default1034 } from "../icons/projector";
import { default as default1035 } from "../icons/printer-check";
import { default as default1036 } from "../icons/printer";
import { default as default1037 } from "../icons/printer-x";
import { default as default1038 } from "../icons/proportions";
import { default as default1039 } from "../icons/puzzle";
import { default as default1040 } from "../icons/pyramid";
import { default as default1041 } from "../icons/qr-code";
import { default as default1042 } from "../icons/quote";
import { default as default1043 } from "../icons/rabbit";
import { default as default1044 } from "../icons/radar";
import { default as default1045 } from "../icons/radiation";
import { default as default1046 } from "../icons/radical";
import { default as default1047 } from "../icons/radio-off";
import { default as default1048 } from "../icons/radio-receiver";
import { default as default1049 } from "../icons/radio";
import { default as default1050 } from "../icons/radio-tower";
import { default as default1051 } from "../icons/radius";
import { default as default1052 } from "../icons/rainbow";
import { default as default1053 } from "../icons/rat";
import { default as default1054 } from "../icons/ratio";
import { default as default1055 } from "../icons/receipt-cent";
import { default as default1056 } from "../icons/receipt-euro";
import { default as default1057 } from "../icons/receipt-indian-rupee";
import { default as default1058 } from "../icons/receipt-japanese-yen";
import { default as default1059 } from "../icons/receipt-pound-sterling";
import { default as default1060 } from "../icons/receipt-swiss-franc";
import { default as default1061 } from "../icons/receipt-russian-ruble";
import { default as default1062 } from "../icons/receipt-text";
import { default as default1063 } from "../icons/receipt-turkish-lira";
import { default as default1064 } from "../icons/rectangle-circle";
import { default as default1065 } from "../icons/receipt";
import { default as default1066 } from "../icons/rectangle-goggles";
import { default as default1067 } from "../icons/rectangle-horizontal";
import { default as default1068 } from "../icons/rectangle-vertical";
import { default as default1069 } from "../icons/recycle";
import { default as default1070 } from "../icons/redo-dot";
import { default as default1071 } from "../icons/redo-2";
import { default as default1072 } from "../icons/redo";
import { default as default1073 } from "../icons/refresh-ccw-dot";
import { default as default1074 } from "../icons/refresh-ccw";
import { default as default1075 } from "../icons/refresh-cw";
import { default as default1076 } from "../icons/refresh-cw-off";
import { default as default1077 } from "../icons/remove-formatting";
import { default as default1078 } from "../icons/regex";
import { default as default1079 } from "../icons/refrigerator";
import { default as default1080 } from "../icons/repeat-1";
import { default as default1081 } from "../icons/repeat-2";
import { default as default1082 } from "../icons/repeat-off";
import { default as default1083 } from "../icons/repeat";
import { default as default1084 } from "../icons/replace-all";
import { default as default1085 } from "../icons/replace";
import { default as default1086 } from "../icons/reply-all";
import { default as default1087 } from "../icons/reply";
import { default as default1088 } from "../icons/rewind";
import { default as default1089 } from "../icons/ribbon";
import { default as default1090 } from "../icons/road";
import { default as default1091 } from "../icons/rocket";
import { default as default1092 } from "../icons/roller-coaster";
import { default as default1093 } from "../icons/rocking-chair";
import { default as default1094 } from "../icons/rose";
import { default as default1095 } from "../icons/rotate-ccw-key";
import { default as default1096 } from "../icons/rotate-ccw-square";
import { default as default1097 } from "../icons/rotate-ccw";
import { default as default1098 } from "../icons/rotate-cw-square";
import { default as default1099 } from "../icons/rotate-cw";
import { default as default1100 } from "../icons/route-off";
import { default as default1101 } from "../icons/route";
import { default as default1102 } from "../icons/router";
import { default as default1103 } from "../icons/rows-4";
import { default as default1104 } from "../icons/rss";
import { default as default1105 } from "../icons/ruler-dimension-line";
import { default as default1106 } from "../icons/ruler";
import { default as default1107 } from "../icons/russian-ruble";
import { default as default1108 } from "../icons/sailboat";
import { default as default1109 } from "../icons/sandwich";
import { default as default1110 } from "../icons/salad";
import { default as default1111 } from "../icons/satellite";
import { default as default1112 } from "../icons/satellite-dish";
import { default as default1113 } from "../icons/saudi-riyal";
import { default as default1114 } from "../icons/save-all";
import { default as default1115 } from "../icons/save-off";
import { default as default1116 } from "../icons/scaling";
import { default as default1117 } from "../icons/scale";
import { default as default1118 } from "../icons/save";
import { default as default1119 } from "../icons/scan-barcode";
import { default as default1120 } from "../icons/scan-eye";
import { default as default1121 } from "../icons/scan-heart";
import { default as default1122 } from "../icons/scan-face";
import { default as default1123 } from "../icons/scan-line";
import { default as default1124 } from "../icons/scan-qr-code";
import { default as default1125 } from "../icons/scan-search";
import { default as default1126 } from "../icons/scan-text";
import { default as default1127 } from "../icons/scan";
import { default as default1128 } from "../icons/school";
import { default as default1129 } from "../icons/scissors-line-dashed";
import { default as default1130 } from "../icons/scissors";
import { default as default1131 } from "../icons/scooter";
import { default as default1132 } from "../icons/screen-share-off";
import { default as default1133 } from "../icons/screen-share";
import { default as default1134 } from "../icons/scroll-text";
import { default as default1135 } from "../icons/scroll";
import { default as default1136 } from "../icons/search-alert";
import { default as default1137 } from "../icons/search-check";
import { default as default1138 } from "../icons/search-code";
import { default as default1139 } from "../icons/search-x";
import { default as default1140 } from "../icons/search-slash";
import { default as default1141 } from "../icons/section";
import { default as default1142 } from "../icons/search";
import { default as default1143 } from "../icons/send";
import { default as default1144 } from "../icons/send-to-back";
import { default as default1145 } from "../icons/separator-horizontal";
import { default as default1146 } from "../icons/server-cog";
import { default as default1147 } from "../icons/separator-vertical";
import { default as default1148 } from "../icons/server-crash";
import { default as default1149 } from "../icons/server-off";
import { default as default1150 } from "../icons/server";
import { default as default1151 } from "../icons/settings-2";
import { default as default1152 } from "../icons/settings";
import { default as default1153 } from "../icons/share-2";
import { default as default1154 } from "../icons/shapes";
import { default as default1155 } from "../icons/share";
import { default as default1156 } from "../icons/sheet";
import { default as default1157 } from "../icons/shelving-unit";
import { default as default1158 } from "../icons/shell";
import { default as default1159 } from "../icons/shield-alert";
import { default as default1160 } from "../icons/shield-ban";
import { default as default1161 } from "../icons/shield-check";
import { default as default1162 } from "../icons/shield-cog-corner";
import { default as default1163 } from "../icons/shield-cog";
import { default as default1164 } from "../icons/shield-ellipsis";
import { default as default1165 } from "../icons/shield-half";
import { default as default1166 } from "../icons/shield-minus";
import { default as default1167 } from "../icons/shield-off";
import { default as default1168 } from "../icons/shield-plus";
import { default as default1169 } from "../icons/shield-user";
import { default as default1170 } from "../icons/shield";
import { default as default1171 } from "../icons/ship-wheel";
import { default as default1172 } from "../icons/ship";
import { default as default1173 } from "../icons/shirt";
import { default as default1174 } from "../icons/shopping-bag";
import { default as default1175 } from "../icons/shopping-basket";
import { default as default1176 } from "../icons/shopping-cart";
import { default as default1177 } from "../icons/shovel";
import { default as default1178 } from "../icons/shower-head";
import { default as default1179 } from "../icons/shredder";
import { default as default1180 } from "../icons/shrimp";
import { default as default1181 } from "../icons/shrink";
import { default as default1182 } from "../icons/shrub";
import { default as default1183 } from "../icons/sigma";
import { default as default1184 } from "../icons/shuffle";
import { default as default1185 } from "../icons/signal-high";
import { default as default1186 } from "../icons/signal-medium";
import { default as default1187 } from "../icons/signal-zero";
import { default as default1188 } from "../icons/signal-low";
import { default as default1189 } from "../icons/signal";
import { default as default1190 } from "../icons/signpost-big";
import { default as default1191 } from "../icons/signature";
import { default as default1192 } from "../icons/signpost";
import { default as default1193 } from "../icons/siren";
import { default as default1194 } from "../icons/skip-back";
import { default as default1195 } from "../icons/skip-forward";
import { default as default1196 } from "../icons/skull";
import { default as default1197 } from "../icons/slash";
import { default as default1198 } from "../icons/slice";
import { default as default1199 } from "../icons/sliders-horizontal";
import { default as default1200 } from "../icons/smartphone-charging";
import { default as default1201 } from "../icons/smartphone-nfc";
import { default as default1202 } from "../icons/smartphone";
import { default as default1203 } from "../icons/smile-plus";
import { default as default1204 } from "../icons/smile";
import { default as default1205 } from "../icons/snail";
import { default as default1206 } from "../icons/snowflake";
import { default as default1207 } from "../icons/soap-dispenser-droplet";
import { default as default1208 } from "../icons/sofa";
import { default as default1209 } from "../icons/soup";
import { default as default1210 } from "../icons/solar-panel";
import { default as default1211 } from "../icons/space";
import { default as default1212 } from "../icons/spade";
import { default as default1213 } from "../icons/sparkle";
import { default as default1214 } from "../icons/speaker";
import { default as default1215 } from "../icons/speech";
import { default as default1216 } from "../icons/spell-check-2";
import { default as default1217 } from "../icons/spell-check";
import { default as default1218 } from "../icons/spline-pointer";
import { default as default1219 } from "../icons/spline";
import { default as default1220 } from "../icons/split";
import { default as default1221 } from "../icons/spool";
import { default as default1222 } from "../icons/spotlight";
import { default as default1223 } from "../icons/sport-shoe";
import { default as default1224 } from "../icons/spray-can";
import { default as default1225 } from "../icons/sprout";
import { default as default1226 } from "../icons/square-arrow-right-enter";
import { default as default1227 } from "../icons/square-arrow-right-exit";
import { default as default1228 } from "../icons/square-dashed-bottom-code";
import { default as default1229 } from "../icons/square-dashed-bottom";
import { default as default1230 } from "../icons/square-dashed-top-solid";
import { default as default1231 } from "../icons/square-pause";
import { default as default1232 } from "../icons/square-radical";
import { default as default1233 } from "../icons/square-round-corner";
import { default as default1234 } from "../icons/square-square";
import { default as default1235 } from "../icons/square-stack";
import { default as default1236 } from "../icons/square-star";
import { default as default1237 } from "../icons/square-stop";
import { default as default1238 } from "../icons/squares-exclude";
import { default as default1239 } from "../icons/square";
import { default as default1240 } from "../icons/squares-intersect";
import { default as default1241 } from "../icons/squares-subtract";
import { default as default1242 } from "../icons/squares-unite";
import { default as default1243 } from "../icons/squircle";
import { default as default1244 } from "../icons/squircle-dashed";
import { default as default1245 } from "../icons/squirrel";
import { default as default1246 } from "../icons/stamp";
import { default as default1247 } from "../icons/star-half";
import { default as default1248 } from "../icons/star-off";
import { default as default1249 } from "../icons/step-back";
import { default as default1250 } from "../icons/star";
import { default as default1251 } from "../icons/step-forward";
import { default as default1252 } from "../icons/stethoscope";
import { default as default1253 } from "../icons/sticker";
import { default as default1254 } from "../icons/sticky-note-minus";
import { default as default1255 } from "../icons/sticky-note-check";
import { default as default1256 } from "../icons/sticky-note-plus";
import { default as default1257 } from "../icons/sticky-note-off";
import { default as default1258 } from "../icons/sticky-note-x";
import { default as default1259 } from "../icons/sticky-note";
import { default as default1260 } from "../icons/sticky-notes";
import { default as default1261 } from "../icons/stone";
import { default as default1262 } from "../icons/store";
import { default as default1263 } from "../icons/stretch-horizontal";
import { default as default1264 } from "../icons/stretch-vertical";
import { default as default1265 } from "../icons/strikethrough";
import { default as default1266 } from "../icons/subscript";
import { default as default1267 } from "../icons/sun-dim";
import { default as default1268 } from "../icons/sun-medium";
import { default as default1269 } from "../icons/sun-moon";
import { default as default1270 } from "../icons/sun-snow";
import { default as default1271 } from "../icons/sun";
import { default as default1272 } from "../icons/sunrise";
import { default as default1273 } from "../icons/superscript";
import { default as default1274 } from "../icons/swatch-book";
import { default as default1275 } from "../icons/sunset";
import { default as default1276 } from "../icons/swiss-franc";
import { default as default1277 } from "../icons/switch-camera";
import { default as default1278 } from "../icons/sword";
import { default as default1279 } from "../icons/swords";
import { default as default1280 } from "../icons/syringe";
import { default as default1281 } from "../icons/table-2";
import { default as default1282 } from "../icons/table-cells-merge";
import { default as default1283 } from "../icons/table-cells-split";
import { default as default1284 } from "../icons/table-columns-split";
import { default as default1285 } from "../icons/table-of-contents";
import { default as default1286 } from "../icons/table-rows-split";
import { default as default1287 } from "../icons/table";
import { default as default1288 } from "../icons/table-properties";
import { default as default1289 } from "../icons/tablet-smartphone";
import { default as default1290 } from "../icons/tablets";
import { default as default1291 } from "../icons/tablet";
import { default as default1292 } from "../icons/tag";
import { default as default1293 } from "../icons/tally-2";
import { default as default1294 } from "../icons/tally-1";
import { default as default1295 } from "../icons/tags";
import { default as default1296 } from "../icons/tally-3";
import { default as default1297 } from "../icons/tally-4";
import { default as default1298 } from "../icons/tally-5";
import { default as default1299 } from "../icons/tangent";
import { default as default1300 } from "../icons/telescope";
import { default as default1301 } from "../icons/target";
import { default as default1302 } from "../icons/tent";
import { default as default1303 } from "../icons/tent-tree";
import { default as default1304 } from "../icons/terminal";
import { default as default1305 } from "../icons/test-tube";
import { default as default1306 } from "../icons/test-tubes";
import { default as default1307 } from "../icons/text-cursor-input";
import { default as default1308 } from "../icons/text-cursor";
import { default as default1309 } from "../icons/text-quote";
import { default as default1310 } from "../icons/text-search";
import { default as default1311 } from "../icons/theater";
import { default as default1312 } from "../icons/thermometer-snowflake";
import { default as default1313 } from "../icons/thermometer-sun";
import { default as default1314 } from "../icons/thermometer";
import { default as default1315 } from "../icons/thumbs-down";
import { default as default1316 } from "../icons/thumbs-up";
import { default as default1317 } from "../icons/ticket-minus";
import { default as default1318 } from "../icons/ticket-check";
import { default as default1319 } from "../icons/ticket-plus";
import { default as default1320 } from "../icons/ticket-percent";
import { default as default1321 } from "../icons/ticket-x";
import { default as default1322 } from "../icons/ticket";
import { default as default1323 } from "../icons/ticket-slash";
import { default as default1324 } from "../icons/tickets-plane";
import { default as default1325 } from "../icons/tickets";
import { default as default1326 } from "../icons/timeline";
import { default as default1327 } from "../icons/timer-off";
import { default as default1328 } from "../icons/timer-reset";
import { default as default1329 } from "../icons/timer";
import { default as default1330 } from "../icons/toggle-left";
import { default as default1331 } from "../icons/tool-case";
import { default as default1332 } from "../icons/toggle-right";
import { default as default1333 } from "../icons/tornado";
import { default as default1334 } from "../icons/toilet";
import { default as default1335 } from "../icons/toolbox";
import { default as default1336 } from "../icons/torus";
import { default as default1337 } from "../icons/touchpad-off";
import { default as default1338 } from "../icons/towel-rack";
import { default as default1339 } from "../icons/touchpad";
import { default as default1340 } from "../icons/tower-control";
import { default as default1341 } from "../icons/tractor";
import { default as default1342 } from "../icons/toy-brick";
import { default as default1343 } from "../icons/traffic-cone";
import { default as default1344 } from "../icons/train-front-tunnel";
import { default as default1345 } from "../icons/train-front";
import { default as default1346 } from "../icons/train-track";
import { default as default1347 } from "../icons/transgender";
import { default as default1348 } from "../icons/trash-2";
import { default as default1349 } from "../icons/tree-deciduous";
import { default as default1350 } from "../icons/trash";
import { default as default1351 } from "../icons/tree-pine";
import { default as default1352 } from "../icons/trees";
import { default as default1353 } from "../icons/trending-down";
import { default as default1354 } from "../icons/trending-up-down";
import { default as default1355 } from "../icons/trending-up";
import { default as default1356 } from "../icons/triangle-dashed";
import { default as default1357 } from "../icons/triangle-right";
import { default as default1358 } from "../icons/triangle";
import { default as default1359 } from "../icons/trophy";
import { default as default1360 } from "../icons/truck-electric";
import { default as default1361 } from "../icons/truck";
import { default as default1362 } from "../icons/turkish-lira";
import { default as default1363 } from "../icons/turntable";
import { default as default1364 } from "../icons/turtle";
import { default as default1365 } from "../icons/tv-minimal-play";
import { default as default1366 } from "../icons/tv";
import { default as default1367 } from "../icons/type-outline";
import { default as default1368 } from "../icons/type";
import { default as default1369 } from "../icons/umbrella";
import { default as default1370 } from "../icons/umbrella-off";
import { default as default1371 } from "../icons/undo-2";
import { default as default1372 } from "../icons/underline";
import { default as default1373 } from "../icons/undo-dot";
import { default as default1374 } from "../icons/undo";
import { default as default1375 } from "../icons/unfold-vertical";
import { default as default1376 } from "../icons/unfold-horizontal";
import { default as default1377 } from "../icons/ungroup";
import { default as default1378 } from "../icons/unlink-2";
import { default as default1379 } from "../icons/unlink";
import { default as default1380 } from "../icons/unplug";
import { default as default1381 } from "../icons/upload";
import { default as default1382 } from "../icons/usb";
import { default as default1383 } from "../icons/user-check";
import { default as default1384 } from "../icons/user-cog";
import { default as default1385 } from "../icons/user-key";
import { default as default1386 } from "../icons/user-minus";
import { default as default1387 } from "../icons/user-lock";
import { default as default1388 } from "../icons/user-pen";
import { default as default1389 } from "../icons/user-plus";
import { default as default1390 } from "../icons/user-round-key";
import { default as default1391 } from "../icons/user-round-pen";
import { default as default1392 } from "../icons/user-round-search";
import { default as default1393 } from "../icons/user-search";
import { default as default1394 } from "../icons/user-star";
import { default as default1395 } from "../icons/user-x";
import { default as default1396 } from "../icons/user";
import { default as default1397 } from "../icons/users";
import { default as default1398 } from "../icons/utility-pole";
import { default as default1399 } from "../icons/van";
import { default as default1400 } from "../icons/variable";
import { default as default1401 } from "../icons/vault";
import { default as default1402 } from "../icons/vector-square";
import { default as default1403 } from "../icons/vegan";
import { default as default1404 } from "../icons/venetian-mask";
import { default as default1405 } from "../icons/venus-and-mars";
import { default as default1406 } from "../icons/venus";
import { default as default1407 } from "../icons/vibrate-off";
import { default as default1408 } from "../icons/vibrate";
import { default as default1409 } from "../icons/video-off";
import { default as default1410 } from "../icons/video";
import { default as default1411 } from "../icons/videotape";
import { default as default1412 } from "../icons/view";
import { default as default1413 } from "../icons/voicemail";
import { default as default1414 } from "../icons/volleyball";
import { default as default1415 } from "../icons/volume-1";
import { default as default1416 } from "../icons/volume-off";
import { default as default1417 } from "../icons/volume-2";
import { default as default1418 } from "../icons/volume-x";
import { default as default1419 } from "../icons/volume";
import { default as default1420 } from "../icons/vote";
import { default as default1421 } from "../icons/wallet-cards";
import { default as default1422 } from "../icons/wallpaper";
import { default as default1423 } from "../icons/wallet";
import { default as default1424 } from "../icons/warehouse";
import { default as default1425 } from "../icons/wand";
import { default as default1426 } from "../icons/washing-machine";
import { default as default1427 } from "../icons/watch";
import { default as default1428 } from "../icons/waves-arrow-down";
import { default as default1429 } from "../icons/waves-arrow-up";
import { default as default1430 } from "../icons/waves-vertical";
import { default as default1431 } from "../icons/waves-ladder";
import { default as default1432 } from "../icons/waypoints";
import { default as default1433 } from "../icons/webcam";
import { default as default1434 } from "../icons/webhook-off";
import { default as default1435 } from "../icons/webhook";
import { default as default1436 } from "../icons/weight-tilde";
import { default as default1437 } from "../icons/weight";
import { default as default1438 } from "../icons/wheat-off";
import { default as default1439 } from "../icons/wheat";
import { default as default1440 } from "../icons/whole-word";
import { default as default1441 } from "../icons/wifi-cog";
import { default as default1442 } from "../icons/wifi-high";
import { default as default1443 } from "../icons/wifi-low";
import { default as default1444 } from "../icons/wifi-off";
import { default as default1445 } from "../icons/wifi-pen";
import { default as default1446 } from "../icons/wifi-sync";
import { default as default1447 } from "../icons/wifi";
import { default as default1448 } from "../icons/wifi-zero";
import { default as default1449 } from "../icons/wind-arrow-down";
import { default as default1450 } from "../icons/wind";
import { default as default1451 } from "../icons/wine";
import { default as default1452 } from "../icons/wine-off";
import { default as default1453 } from "../icons/workflow";
import { default as default1454 } from "../icons/worm";
import { default as default1455 } from "../icons/wrench";
import { default as default1456 } from "../icons/x-line-top";
import { default as default1457 } from "../icons/x";
import { default as default1458 } from "../icons/zap-off";
import { default as default1459 } from "../icons/zodiac-aries";
import { default as default1460 } from "../icons/zap";
import { default as default1461 } from "../icons/zodiac-aquarius";
import { default as default1462 } from "../icons/zodiac-cancer";
import { default as default1463 } from "../icons/zodiac-gemini";
import { default as default1464 } from "../icons/zodiac-leo";
import { default as default1465 } from "../icons/zodiac-ophiuchus";
import { default as default1466 } from "../icons/zodiac-capricorn";
import { default as default1467 } from "../icons/zodiac-libra";
import { default as default1468 } from "../icons/zodiac-pisces";
import { default as default1469 } from "../icons/zodiac-scorpio";
import { default as default1470 } from "../icons/zodiac-sagittarius";
import { default as default1471 } from "../icons/zodiac-taurus";
import { default as default1472 } from "../icons/zodiac-virgo";
import { default as default1473 } from "../icons/zoom-out";
import { default as default1474 } from "../icons/zoom-in";
import { default as default1475 } from "../icons/fingerprint-pattern";
import {
  default as default1476
} from "../icons/fingerprint-pattern";
import { default as default1477 } from "../icons/arrow-down-0-1";
import { default as default1478 } from "../icons/arrow-down-1-0";
import { default as default1479 } from "../icons/arrow-up-0-1";
import { default as default1480 } from "../icons/arrow-up-1-0";
import { default as default1481 } from "../icons/alarm-clock-check";
import {
  default as default1482
} from "../icons/alarm-check";
import { default as default1483 } from "../icons/alarm-clock-minus";
import {
  default as default1484
} from "../icons/alarm-minus";
import { default as default1485 } from "../icons/alarm-clock-plus";
import {
  default as default1486
} from "../icons/alarm-plus";
import { default as default1487 } from "../icons/arrow-down-a-z";
import {
  default as default1488
} from "../icons/arrow-down-az";
import { default as default1489 } from "../icons/arrow-down-z-a";
import {
  default as default1490
} from "../icons/arrow-down-za";
import { default as default1491 } from "../icons/arrow-down-wide-narrow";
import {
  default as default1492
} from "../icons/sort-desc";
import { default as default1493 } from "../icons/arrow-up-a-z";
import {
  default as default1494
} from "../icons/arrow-up-az";
import { default as default1495 } from "../icons/arrow-up-narrow-wide";
import {
  default as default1496
} from "../icons/sort-asc";
import { default as default1497 } from "../icons/arrow-up-z-a";
import {
  default as default1498
} from "../icons/arrow-up-za";
import { default as default1499 } from "../icons/axis-3d";
import {
  default as default1500
} from "../icons/axis-3-d";
import { default as default1501 } from "../icons/badge-check";
import {
  default as default1502
} from "../icons/verified";
import { default as default1503 } from "../icons/badge-question-mark";
import {
  default as default1504
} from "../icons/badge-help";
import { default as default1505 } from "../icons/between-horizontal-end";
import {
  default as default1506
} from "../icons/between-horizonal-end";
import { default as default1507 } from "../icons/between-horizontal-start";
import {
  default as default1508
} from "../icons/between-horizonal-start";
import { default as default1509 } from "../icons/book-dashed";
import {
  default as default1510
} from "../icons/book-template";
import { default as default1511 } from "../icons/braces";
import {
  default as default1512
} from "../icons/curly-braces";
import { default as default1513 } from "../icons/captions";
import {
  default as default1514
} from "../icons/subtitles";
import { default as default1515 } from "../icons/chart-area";
import {
  default as default1516
} from "../icons/area-chart";
import { default as default1517 } from "../icons/chart-bar-big";
import {
  default as default1518
} from "../icons/bar-chart-horizontal-big";
import { default as default1519 } from "../icons/chart-bar";
import {
  default as default1520
} from "../icons/bar-chart-horizontal";
import { default as default1521 } from "../icons/chart-candlestick";
import {
  default as default1522
} from "../icons/candlestick-chart";
import { default as default1523 } from "../icons/chart-column-big";
import {
  default as default1524
} from "../icons/bar-chart-big";
import { default as default1525 } from "../icons/chart-column-increasing";
import {
  default as default1526
} from "../icons/bar-chart-4";
import { default as default1527 } from "../icons/chart-column";
import {
  default as default1528
} from "../icons/bar-chart-3";
import { default as default1529 } from "../icons/chart-line";
import {
  default as default1530
} from "../icons/line-chart";
import { default as default1531 } from "../icons/chart-no-axes-column-increasing";
import {
  default as default1532
} from "../icons/bar-chart";
import { default as default1533 } from "../icons/chart-no-axes-column";
import {
  default as default1534
} from "../icons/bar-chart-2";
import { default as default1535 } from "../icons/chart-no-axes-gantt";
import {
  default as default1536
} from "../icons/gantt-chart";
import { default as default1537 } from "../icons/chart-pie";
import {
  default as default1538
} from "../icons/pie-chart";
import { default as default1539 } from "../icons/circle-alert";
import {
  default as default1540
} from "../icons/alert-circle";
import { default as default1541 } from "../icons/chart-scatter";
import {
  default as default1542
} from "../icons/scatter-chart";
import { default as default1543 } from "../icons/circle-arrow-down";
import {
  default as default1544
} from "../icons/arrow-down-circle";
import { default as default1545 } from "../icons/circle-arrow-left";
import {
  default as default1546
} from "../icons/arrow-left-circle";
import { default as default1547 } from "../icons/circle-arrow-out-down-left";
import {
  default as default1548
} from "../icons/arrow-down-left-from-circle";
import { default as default1549 } from "../icons/circle-arrow-out-down-right";
import {
  default as default1550
} from "../icons/arrow-down-right-from-circle";
import { default as default1551 } from "../icons/circle-arrow-out-up-left";
import {
  default as default1552
} from "../icons/arrow-up-left-from-circle";
import { default as default1553 } from "../icons/circle-arrow-out-up-right";
import {
  default as default1554
} from "../icons/arrow-up-right-from-circle";
import { default as default1555 } from "../icons/circle-arrow-right";
import {
  default as default1556
} from "../icons/arrow-right-circle";
import { default as default1557 } from "../icons/circle-arrow-up";
import {
  default as default1558
} from "../icons/arrow-up-circle";
import { default as default1559 } from "../icons/circle-check-big";
import {
  default as default1560
} from "../icons/check-circle";
import { default as default1561 } from "../icons/circle-chevron-down";
import {
  default as default1562
} from "../icons/chevron-down-circle";
import { default as default1563 } from "../icons/circle-check";
import {
  default as default1564
} from "../icons/check-circle-2";
import { default as default1565 } from "../icons/circle-chevron-left";
import {
  default as default1566
} from "../icons/chevron-left-circle";
import { default as default1567 } from "../icons/circle-chevron-right";
import {
  default as default1568
} from "../icons/chevron-right-circle";
import { default as default1569 } from "../icons/circle-divide";
import {
  default as default1570
} from "../icons/divide-circle";
import { default as default1571 } from "../icons/circle-chevron-up";
import {
  default as default1572
} from "../icons/chevron-up-circle";
import { default as default1573 } from "../icons/circle-minus";
import {
  default as default1574
} from "../icons/minus-circle";
import { default as default1575 } from "../icons/circle-parking-off";
import {
  default as default1576
} from "../icons/parking-circle-off";
import { default as default1577 } from "../icons/circle-gauge";
import {
  default as default1578
} from "../icons/gauge-circle";
import { default as default1579 } from "../icons/circle-pause";
import {
  default as default1580
} from "../icons/pause-circle";
import { default as default1581 } from "../icons/circle-play";
import {
  default as default1582
} from "../icons/play-circle";
import { default as default1583 } from "../icons/circle-percent";
import {
  default as default1584
} from "../icons/percent-circle";
import { default as default1585 } from "../icons/circle-parking";
import {
  default as default1586
} from "../icons/parking-circle";
import { default as default1587 } from "../icons/circle-plus";
import {
  default as default1588
} from "../icons/plus-circle";
import { default as default1589 } from "../icons/circle-power";
import {
  default as default1590
} from "../icons/power-circle";
import { default as default1591 } from "../icons/circle-slash-2";
import {
  default as default1592
} from "../icons/circle-slashed";
import { default as default1593 } from "../icons/circle-question-mark";
import {
  default as default1594
} from "../icons/help-circle";
import {
  default as default1595
} from "../icons/circle-help";
import { default as default1596 } from "../icons/circle-stop";
import {
  default as default1597
} from "../icons/stop-circle";
import { default as default1598 } from "../icons/circle-user-round";
import {
  default as default1599
} from "../icons/user-circle-2";
import { default as default1600 } from "../icons/circle-user";
import {
  default as default1601
} from "../icons/user-circle";
import { default as default1602 } from "../icons/circle-x";
import {
  default as default1603
} from "../icons/x-circle";
import { default as default1604 } from "../icons/clipboard-pen-line";
import {
  default as default1605
} from "../icons/clipboard-signature";
import { default as default1606 } from "../icons/clipboard-pen";
import {
  default as default1607
} from "../icons/clipboard-edit";
import { default as default1608 } from "../icons/cloud-download";
import {
  default as default1609
} from "../icons/download-cloud";
import { default as default1610 } from "../icons/cloud-upload";
import {
  default as default1611
} from "../icons/upload-cloud";
import { default as default1612 } from "../icons/code-xml";
import {
  default as default1613
} from "../icons/code-2";
import { default as default1614 } from "../icons/columns-2";
import {
  default as default1615
} from "../icons/columns";
import { default as default1616 } from "../icons/columns-3-cog";
import {
  default as default1617
} from "../icons/columns-settings";
import {
  default as default1618
} from "../icons/table-config";
import { default as default1619 } from "../icons/columns-3";
import {
  default as default1620
} from "../icons/panels-left-right";
import { default as default1621 } from "../icons/contact-round";
import {
  default as default1622
} from "../icons/contact-2";
import { default as default1623 } from "../icons/diamond-percent";
import {
  default as default1624
} from "../icons/percent-diamond";
import { default as default1625 } from "../icons/earth";
import {
  default as default1626
} from "../icons/globe-2";
import { default as default1627 } from "../icons/ellipsis-vertical";
import {
  default as default1628
} from "../icons/more-vertical";
import { default as default1629 } from "../icons/ellipsis";
import {
  default as default1630
} from "../icons/more-horizontal";
import { default as default1631 } from "../icons/file-axis-3d";
import {
  default as default1632
} from "../icons/file-axis-3-d";
import { default as default1633 } from "../icons/file-badge";
import {
  default as default1634
} from "../icons/file-badge-2";
import { default as default1635 } from "../icons/file-braces-corner";
import {
  default as default1636
} from "../icons/file-json-2";
import { default as default1637 } from "../icons/file-braces";
import {
  default as default1638
} from "../icons/file-json";
import { default as default1639 } from "../icons/file-chart-line";
import {
  default as default1640
} from "../icons/file-line-chart";
import { default as default1641 } from "../icons/file-chart-column-increasing";
import {
  default as default1642
} from "../icons/file-bar-chart";
import { default as default1643 } from "../icons/file-chart-column";
import {
  default as default1644
} from "../icons/file-bar-chart-2";
import { default as default1645 } from "../icons/file-chart-pie";
import {
  default as default1646
} from "../icons/file-pie-chart";
import { default as default1647 } from "../icons/file-check-corner";
import {
  default as default1648
} from "../icons/file-check-2";
import { default as default1649 } from "../icons/file-code-corner";
import {
  default as default1650
} from "../icons/file-code-2";
import { default as default1651 } from "../icons/file-cog";
import {
  default as default1652
} from "../icons/file-cog-2";
import { default as default1653 } from "../icons/file-exclamation-point";
import {
  default as default1654
} from "../icons/file-warning";
import { default as default1655 } from "../icons/file-headphone";
import {
  default as default1656
} from "../icons/file-audio";
import {
  default as default1657
} from "../icons/file-audio-2";
import { default as default1658 } from "../icons/file-lock";
import {
  default as default1659
} from "../icons/file-lock-2";
import { default as default1660 } from "../icons/file-minus-corner";
import {
  default as default1661
} from "../icons/file-minus-2";
import { default as default1662 } from "../icons/file-key";
import {
  default as default1663
} from "../icons/file-key-2";
import { default as default1664 } from "../icons/file-pen";
import {
  default as default1665
} from "../icons/file-edit";
import { default as default1666 } from "../icons/file-pen-line";
import {
  default as default1667
} from "../icons/file-signature";
import { default as default1668 } from "../icons/file-play";
import {
  default as default1669
} from "../icons/file-video";
import { default as default1670 } from "../icons/file-plus-corner";
import {
  default as default1671
} from "../icons/file-plus-2";
import { default as default1672 } from "../icons/file-search-corner";
import {
  default as default1673
} from "../icons/file-search-2";
import { default as default1674 } from "../icons/file-question-mark";
import {
  default as default1675
} from "../icons/file-question";
import { default as default1676 } from "../icons/file-signal";
import {
  default as default1677
} from "../icons/file-volume-2";
import { default as default1678 } from "../icons/file-type-corner";
import {
  default as default1679
} from "../icons/file-type-2";
import { default as default1680 } from "../icons/file-video-camera";
import {
  default as default1681
} from "../icons/file-video-2";
import { default as default1682 } from "../icons/file-x-corner";
import {
  default as default1683
} from "../icons/file-x-2";
import { default as default1684 } from "../icons/folder-cog";
import {
  default as default1685
} from "../icons/folder-cog-2";
import { default as default1686 } from "../icons/folder-pen";
import {
  default as default1687
} from "../icons/folder-edit";
import { default as default1688 } from "../icons/funnel-x";
import {
  default as default1689
} from "../icons/filter-x";
import { default as default1690 } from "../icons/funnel";
import {
  default as default1691
} from "../icons/filter";
import { default as default1692 } from "../icons/git-commit-horizontal";
import {
  default as default1693
} from "../icons/git-commit";
import { default as default1694 } from "../icons/grid-2x2-check";
import {
  default as default1695
} from "../icons/grid-2-x-2-check";
import { default as default1696 } from "../icons/grid-2x2-plus";
import {
  default as default1697
} from "../icons/grid-2-x-2-plus";
import { default as default1698 } from "../icons/grid-2x2-x";
import {
  default as default1699
} from "../icons/grid-2-x-2-x";
import { default as default1700 } from "../icons/grid-2x2";
import {
  default as default1701
} from "../icons/grid-2-x-2";
import { default as default1702 } from "../icons/grid-3x3";
import {
  default as default1703
} from "../icons/grid";
import {
  default as default1704
} from "../icons/grid-3-x-3";
import { default as default1705 } from "../icons/hand-grab";
import {
  default as default1706
} from "../icons/grab";
import { default as default1707 } from "../icons/hand-helping";
import {
  default as default1708
} from "../icons/helping-hand";
import { default as default1709 } from "../icons/house";
import {
  default as default1710
} from "../icons/home";
import { default as default1711 } from "../icons/ice-cream-bowl";
import {
  default as default1712
} from "../icons/ice-cream-2";
import { default as default1713 } from "../icons/ice-cream-cone";
import {
  default as default1714
} from "../icons/ice-cream";
import { default as default1715 } from "../icons/laptop-minimal";
import {
  default as default1716
} from "../icons/laptop-2";
import { default as default1717 } from "../icons/layers";
import {
  default as default1718
} from "../icons/layers-3";
import { default as default1719 } from "../icons/list-indent-decrease";
import {
  default as default1720
} from "../icons/outdent";
import {
  default as default1721
} from "../icons/indent-decrease";
import { default as default1722 } from "../icons/list-indent-increase";
import {
  default as default1723
} from "../icons/indent";
import {
  default as default1724
} from "../icons/indent-increase";
import { default as default1725 } from "../icons/loader-circle";
import {
  default as default1726
} from "../icons/loader-2";
import { default as default1727 } from "../icons/lock-open";
import {
  default as default1728
} from "../icons/unlock";
import { default as default1729 } from "../icons/lock-keyhole-open";
import {
  default as default1730
} from "../icons/unlock-keyhole";
import { default as default1731 } from "../icons/mail-question-mark";
import {
  default as default1732
} from "../icons/mail-question";
import { default as default1733 } from "../icons/map-pin-pen";
import {
  default as default1734
} from "../icons/location-edit";
import { default as default1735 } from "../icons/message-circle-question-mark";
import {
  default as default1736
} from "../icons/message-circle-question";
import { default as default1737 } from "../icons/mic-vocal";
import {
  default as default1738
} from "../icons/mic-2";
import { default as default1739 } from "../icons/octagon-pause";
import {
  default as default1740
} from "../icons/pause-octagon";
import { default as default1741 } from "../icons/octagon-alert";
import {
  default as default1742
} from "../icons/alert-octagon";
import { default as default1743 } from "../icons/move-3d";
import {
  default as default1744
} from "../icons/move-3-d";
import { default as default1745 } from "../icons/octagon-x";
import {
  default as default1746
} from "../icons/x-octagon";
import { default as default1747 } from "../icons/panel-left-close";
import {
  default as default1748
} from "../icons/sidebar-close";
import { default as default1749 } from "../icons/paintbrush-vertical";
import {
  default as default1750
} from "../icons/paintbrush-2";
import { default as default1751 } from "../icons/panel-bottom-dashed";
import {
  default as default1752
} from "../icons/panel-bottom-inactive";
import { default as default1753 } from "../icons/panel-left-dashed";
import {
  default as default1754
} from "../icons/panel-left-inactive";
import { default as default1755 } from "../icons/panel-left-open";
import {
  default as default1756
} from "../icons/sidebar-open";
import { default as default1757 } from "../icons/panel-left";
import {
  default as default1758
} from "../icons/sidebar";
import { default as default1759 } from "../icons/panel-right-dashed";
import {
  default as default1760
} from "../icons/panel-right-inactive";
import { default as default1761 } from "../icons/panel-top-dashed";
import {
  default as default1762
} from "../icons/panel-top-inactive";
import { default as default1763 } from "../icons/panels-top-left";
import {
  default as default1764
} from "../icons/layout";
import { default as default1765 } from "../icons/pen-line";
import {
  default as default1766
} from "../icons/edit-3";
import { default as default1767 } from "../icons/pen";
import {
  default as default1768
} from "../icons/edit-2";
import { default as default1769 } from "../icons/plug-zap";
import {
  default as default1770
} from "../icons/plug-zap-2";
import { default as default1771 } from "../icons/rectangle-ellipsis";
import {
  default as default1772
} from "../icons/form-input";
import { default as default1773 } from "../icons/rotate-3d";
import {
  default as default1774
} from "../icons/rotate-3-d";
import { default as default1775 } from "../icons/rows-2";
import {
  default as default1776
} from "../icons/rows";
import { default as default1777 } from "../icons/scale-3d";
import {
  default as default1778
} from "../icons/scale-3-d";
import { default as default1779 } from "../icons/rows-3";
import {
  default as default1780
} from "../icons/panels-top-bottom";
import { default as default1781 } from "../icons/send-horizontal";
import {
  default as default1782
} from "../icons/send-horizonal";
import { default as default1783 } from "../icons/shield-question-mark";
import {
  default as default1784
} from "../icons/shield-question";
import { default as default1785 } from "../icons/shield-x";
import {
  default as default1786
} from "../icons/shield-close";
import { default as default1787 } from "../icons/sliders-vertical";
import {
  default as default1788
} from "../icons/sliders";
import { default as default1789 } from "../icons/sparkles";
import {
  default as default1790
} from "../icons/stars";
import { default as default1791 } from "../icons/square-activity";
import {
  default as default1792
} from "../icons/activity-square";
import { default as default1793 } from "../icons/square-arrow-down-left";
import {
  default as default1794
} from "../icons/arrow-down-left-square";
import { default as default1795 } from "../icons/square-arrow-down";
import {
  default as default1796
} from "../icons/arrow-down-square";
import { default as default1797 } from "../icons/square-arrow-down-right";
import {
  default as default1798
} from "../icons/arrow-down-right-square";
import { default as default1799 } from "../icons/square-arrow-left";
import {
  default as default1800
} from "../icons/arrow-left-square";
import { default as default1801 } from "../icons/square-arrow-out-down-left";
import {
  default as default1802
} from "../icons/arrow-down-left-from-square";
import { default as default1803 } from "../icons/square-arrow-out-down-right";
import {
  default as default1804
} from "../icons/arrow-down-right-from-square";
import { default as default1805 } from "../icons/square-arrow-out-up-left";
import {
  default as default1806
} from "../icons/arrow-up-left-from-square";
import { default as default1807 } from "../icons/square-arrow-out-up-right";
import {
  default as default1808
} from "../icons/arrow-up-right-from-square";
import { default as default1809 } from "../icons/square-arrow-right";
import {
  default as default1810
} from "../icons/arrow-right-square";
import { default as default1811 } from "../icons/square-arrow-up-left";
import {
  default as default1812
} from "../icons/arrow-up-left-square";
import { default as default1813 } from "../icons/square-arrow-up";
import {
  default as default1814
} from "../icons/arrow-up-square";
import { default as default1815 } from "../icons/square-arrow-up-right";
import {
  default as default1816
} from "../icons/arrow-up-right-square";
import { default as default1817 } from "../icons/square-asterisk";
import {
  default as default1818
} from "../icons/asterisk-square";
import { default as default1819 } from "../icons/square-centerline-dashed-vertical";
import {
  default as default1820
} from "../icons/flip-vertical";
import { default as default1821 } from "../icons/square-centerline-dashed-horizontal";
import {
  default as default1822
} from "../icons/flip-horizontal";
import { default as default1823 } from "../icons/square-bottom-dashed-scissors";
import {
  default as default1824
} from "../icons/scissors-square-dashed-bottom";
import { default as default1825 } from "../icons/square-chart-gantt";
import {
  default as default1826
} from "../icons/gantt-chart-square";
import {
  default as default1827
} from "../icons/square-gantt-chart";
import { default as default1828 } from "../icons/square-check-big";
import {
  default as default1829
} from "../icons/check-square";
import { default as default1830 } from "../icons/square-check";
import {
  default as default1831
} from "../icons/check-square-2";
import { default as default1832 } from "../icons/square-chevron-right";
import {
  default as default1833
} from "../icons/chevron-right-square";
import { default as default1834 } from "../icons/square-chevron-left";
import {
  default as default1835
} from "../icons/chevron-left-square";
import { default as default1836 } from "../icons/square-chevron-down";
import {
  default as default1837
} from "../icons/chevron-down-square";
import { default as default1838 } from "../icons/square-chevron-up";
import {
  default as default1839
} from "../icons/chevron-up-square";
import { default as default1840 } from "../icons/square-code";
import {
  default as default1841
} from "../icons/code-square";
import { default as default1842 } from "../icons/square-dashed-kanban";
import {
  default as default1843
} from "../icons/kanban-square-dashed";
import { default as default1844 } from "../icons/square-dashed-mouse-pointer";
import {
  default as default1845
} from "../icons/mouse-pointer-square-dashed";
import { default as default1846 } from "../icons/square-dashed-text";
import {
  default as default1847
} from "../icons/text-selection";
import {
  default as default1848
} from "../icons/text-select";
import { default as default1849 } from "../icons/square-dashed";
import {
  default as default1850
} from "../icons/box-select";
import { default as default1851 } from "../icons/square-divide";
import {
  default as default1852
} from "../icons/divide-square";
import { default as default1853 } from "../icons/square-dot";
import {
  default as default1854
} from "../icons/dot-square";
import { default as default1855 } from "../icons/square-function";
import {
  default as default1856
} from "../icons/function-square";
import { default as default1857 } from "../icons/square-equal";
import {
  default as default1858
} from "../icons/equal-square";
import { default as default1859 } from "../icons/square-library";
import {
  default as default1860
} from "../icons/library-square";
import { default as default1861 } from "../icons/square-kanban";
import {
  default as default1862
} from "../icons/kanban-square";
import { default as default1863 } from "../icons/square-m";
import {
  default as default1864
} from "../icons/m-square";
import { default as default1865 } from "../icons/square-menu";
import {
  default as default1866
} from "../icons/menu-square";
import { default as default1867 } from "../icons/square-minus";
import {
  default as default1868
} from "../icons/minus-square";
import { default as default1869 } from "../icons/square-mouse-pointer";
import {
  default as default1870
} from "../icons/inspect";
import { default as default1871 } from "../icons/square-parking-off";
import {
  default as default1872
} from "../icons/parking-square-off";
import { default as default1873 } from "../icons/square-parking";
import {
  default as default1874
} from "../icons/parking-square";
import { default as default1875 } from "../icons/square-percent";
import {
  default as default1876
} from "../icons/percent-square";
import { default as default1877 } from "../icons/square-pen";
import {
  default as default1878
} from "../icons/edit";
import {
  default as default1879
} from "../icons/pen-box";
import {
  default as default1880
} from "../icons/pen-square";
import { default as default1881 } from "../icons/square-play";
import {
  default as default1882
} from "../icons/play-square";
import { default as default1883 } from "../icons/square-pi";
import {
  default as default1884
} from "../icons/pi-square";
import { default as default1885 } from "../icons/square-pilcrow";
import {
  default as default1886
} from "../icons/pilcrow-square";
import { default as default1887 } from "../icons/square-plus";
import {
  default as default1888
} from "../icons/plus-square";
import { default as default1889 } from "../icons/square-power";
import {
  default as default1890
} from "../icons/power-square";
import { default as default1891 } from "../icons/square-scissors";
import {
  default as default1892
} from "../icons/scissors-square";
import { default as default1893 } from "../icons/square-sigma";
import {
  default as default1894
} from "../icons/sigma-square";
import { default as default1895 } from "../icons/square-slash";
import {
  default as default1896
} from "../icons/slash-square";
import { default as default1897 } from "../icons/square-split-horizontal";
import {
  default as default1898
} from "../icons/split-square-horizontal";
import { default as default1899 } from "../icons/square-split-vertical";
import {
  default as default1900
} from "../icons/split-square-vertical";
import { default as default1901 } from "../icons/square-terminal";
import {
  default as default1902
} from "../icons/terminal-square";
import { default as default1903 } from "../icons/square-user-round";
import {
  default as default1904
} from "../icons/user-square-2";
import { default as default1905 } from "../icons/square-x";
import {
  default as default1906
} from "../icons/x-square";
import { default as default1907 } from "../icons/square-user";
import {
  default as default1908
} from "../icons/user-square";
import { default as default1909 } from "../icons/test-tube-diagonal";
import {
  default as default1910
} from "../icons/test-tube-2";
import { default as default1911 } from "../icons/text-align-justify";
import {
  default as default1912
} from "../icons/align-justify";
import { default as default1913 } from "../icons/text-align-end";
import {
  default as default1914
} from "../icons/align-right";
import { default as default1915 } from "../icons/text-align-center";
import {
  default as default1916
} from "../icons/align-center";
import { default as default1917 } from "../icons/text-align-start";
import {
  default as default1918
} from "../icons/align-left";
import {
  default as default1919
} from "../icons/text";
import { default as default1920 } from "../icons/text-wrap";
import {
  default as default1921
} from "../icons/wrap-text";
import { default as default1922 } from "../icons/text-initial";
import {
  default as default1923
} from "../icons/letter-text";
import { default as default1924 } from "../icons/tram-front";
import {
  default as default1925
} from "../icons/train";
import { default as default1926 } from "../icons/tree-palm";
import {
  default as default1927
} from "../icons/palmtree";
import { default as default1928 } from "../icons/triangle-alert";
import {
  default as default1929
} from "../icons/alert-triangle";
import { default as default1930 } from "../icons/tv-minimal";
import {
  default as default1931
} from "../icons/tv-2";
import { default as default1932 } from "../icons/university";
import {
  default as default1933
} from "../icons/school-2";
import { default as default1934 } from "../icons/user-round-cog";
import {
  default as default1935
} from "../icons/user-cog-2";
import { default as default1936 } from "../icons/user-round-check";
import {
  default as default1937
} from "../icons/user-check-2";
import { default as default1938 } from "../icons/user-round-plus";
import {
  default as default1939
} from "../icons/user-plus-2";
import { default as default1940 } from "../icons/user-round-minus";
import {
  default as default1941
} from "../icons/user-minus-2";
import { default as default1942 } from "../icons/user-round-x";
import {
  default as default1943
} from "../icons/user-x-2";
import { default as default1944 } from "../icons/user-round";
import {
  default as default1945
} from "../icons/user-2";
import { default as default1946 } from "../icons/utensils-crossed";
import {
  default as default1947
} from "../icons/fork-knife-crossed";
import { default as default1948 } from "../icons/users-round";
import {
  default as default1949
} from "../icons/users-2";
import { default as default1950 } from "../icons/utensils";
import {
  default as default1951
} from "../icons/fork-knife";
import { default as default1952 } from "../icons/wallet-minimal";
import {
  default as default1953
} from "../icons/wallet-2";
import { default as default1954 } from "../icons/wand-sparkles";
import {
  default as default1955
} from "../icons/wand-2";
import { default as default1956 } from "../icons/waves-horizontal";
import {
  default as default1957
} from "../icons/waves";
export {
  default3 as AArrowDownIcon,
  default2 as AArrowUpIcon,
  default5 as ALargeSmallIcon,
  default6 as AccessibilityIcon,
  default4 as ActivityIcon,
  default1792 as ActivitySquareIcon,
  default7 as AirVentIcon,
  default8 as AirplayIcon,
  default1482 as AlarmCheckIcon,
  default1481 as AlarmClockCheckIcon,
  default10 as AlarmClockIcon,
  default1483 as AlarmClockMinusIcon,
  default9 as AlarmClockOffIcon,
  default1485 as AlarmClockPlusIcon,
  default1484 as AlarmMinusIcon,
  default1486 as AlarmPlusIcon,
  default11 as AlarmSmokeIcon,
  default12 as AlbumIcon,
  default1540 as AlertCircleIcon,
  default1742 as AlertOctagonIcon,
  default1929 as AlertTriangleIcon,
  default13 as AlignCenterHorizontalIcon,
  default1916 as AlignCenterIcon,
  default14 as AlignCenterVerticalIcon,
  default15 as AlignEndHorizontalIcon,
  default16 as AlignEndVerticalIcon,
  default17 as AlignHorizontalDistributeCenterIcon,
  default19 as AlignHorizontalDistributeEndIcon,
  default18 as AlignHorizontalDistributeStartIcon,
  default20 as AlignHorizontalJustifyCenterIcon,
  default21 as AlignHorizontalJustifyEndIcon,
  default22 as AlignHorizontalJustifyStartIcon,
  default23 as AlignHorizontalSpaceAroundIcon,
  default26 as AlignHorizontalSpaceBetweenIcon,
  default1912 as AlignJustifyIcon,
  default1918 as AlignLeftIcon,
  default1914 as AlignRightIcon,
  default24 as AlignStartHorizontalIcon,
  default28 as AlignStartVerticalIcon,
  default25 as AlignVerticalDistributeCenterIcon,
  default27 as AlignVerticalDistributeEndIcon,
  default30 as AlignVerticalDistributeStartIcon,
  default29 as AlignVerticalJustifyCenterIcon,
  default31 as AlignVerticalJustifyEndIcon,
  default32 as AlignVerticalJustifyStartIcon,
  default33 as AlignVerticalSpaceAroundIcon,
  default34 as AlignVerticalSpaceBetweenIcon,
  default35 as AmbulanceIcon,
  default38 as AmpersandIcon,
  default36 as AmpersandsIcon,
  default37 as AmphoraIcon,
  default39 as AnchorIcon,
  default40 as AngryIcon,
  default41 as AnnoyedIcon,
  default43 as AntennaIcon,
  default42 as AnvilIcon,
  default44 as ApertureIcon,
  default45 as AppWindowIcon,
  default46 as AppWindowMacIcon,
  default47 as AppleIcon,
  default51 as ArchiveIcon,
  default49 as ArchiveRestoreIcon,
  default48 as ArchiveXIcon,
  default1516 as AreaChartIcon,
  default50 as ArmchairIcon,
  default52 as ArrowBigDownDashIcon,
  default53 as ArrowBigDownIcon,
  default54 as ArrowBigLeftDashIcon,
  default55 as ArrowBigLeftIcon,
  default56 as ArrowBigRightDashIcon,
  default57 as ArrowBigRightIcon,
  default58 as ArrowBigUpDashIcon,
  default59 as ArrowBigUpIcon,
  default1477 as ArrowDown01Icon,
  default1478 as ArrowDown10Icon,
  default1487 as ArrowDownAZIcon,
  default1488 as ArrowDownAzIcon,
  default1544 as ArrowDownCircleIcon,
  default60 as ArrowDownFromLineIcon,
  default67 as ArrowDownIcon,
  default1548 as ArrowDownLeftFromCircleIcon,
  default1802 as ArrowDownLeftFromSquareIcon,
  default61 as ArrowDownLeftIcon,
  default1794 as ArrowDownLeftSquareIcon,
  default62 as ArrowDownNarrowWideIcon,
  default1550 as ArrowDownRightFromCircleIcon,
  default1804 as ArrowDownRightFromSquareIcon,
  default63 as ArrowDownRightIcon,
  default1798 as ArrowDownRightSquareIcon,
  default1796 as ArrowDownSquareIcon,
  default64 as ArrowDownToDotIcon,
  default66 as ArrowDownToLineIcon,
  default65 as ArrowDownUpIcon,
  default1491 as ArrowDownWideNarrowIcon,
  default1489 as ArrowDownZAIcon,
  default1490 as ArrowDownZaIcon,
  default1546 as ArrowLeftCircleIcon,
  default68 as ArrowLeftFromLineIcon,
  default72 as ArrowLeftIcon,
  default70 as ArrowLeftRightIcon,
  default1800 as ArrowLeftSquareIcon,
  default69 as ArrowLeftToLineIcon,
  default1556 as ArrowRightCircleIcon,
  default71 as ArrowRightFromLineIcon,
  default75 as ArrowRightIcon,
  default73 as ArrowRightLeftIcon,
  default1810 as ArrowRightSquareIcon,
  default74 as ArrowRightToLineIcon,
  default1479 as ArrowUp01Icon,
  default1480 as ArrowUp10Icon,
  default1493 as ArrowUpAZIcon,
  default1494 as ArrowUpAzIcon,
  default1558 as ArrowUpCircleIcon,
  default76 as ArrowUpDownIcon,
  default77 as ArrowUpFromDotIcon,
  default79 as ArrowUpFromLineIcon,
  default83 as ArrowUpIcon,
  default1552 as ArrowUpLeftFromCircleIcon,
  default1806 as ArrowUpLeftFromSquareIcon,
  default78 as ArrowUpLeftIcon,
  default1812 as ArrowUpLeftSquareIcon,
  default1495 as ArrowUpNarrowWideIcon,
  default1554 as ArrowUpRightFromCircleIcon,
  default1808 as ArrowUpRightFromSquareIcon,
  default80 as ArrowUpRightIcon,
  default1816 as ArrowUpRightSquareIcon,
  default1814 as ArrowUpSquareIcon,
  default81 as ArrowUpToLineIcon,
  default82 as ArrowUpWideNarrowIcon,
  default1497 as ArrowUpZAIcon,
  default1498 as ArrowUpZaIcon,
  default84 as ArrowsUpFromLineIcon,
  default85 as AsteriskIcon,
  default1818 as AsteriskSquareIcon,
  default86 as AstroidIcon,
  default88 as AtSignIcon,
  default87 as AtomIcon,
  default89 as AudioLinesIcon,
  default90 as AudioWaveformIcon,
  default91 as AwardIcon,
  default92 as AxeIcon,
  default1500 as Axis3DIcon,
  default1499 as Axis3dIcon,
  default93 as BabyIcon,
  default94 as BackpackIcon,
  default95 as BadgeAlertIcon,
  default96 as BadgeCentIcon,
  default1501 as BadgeCheckIcon,
  default97 as BadgeDollarSignIcon,
  default98 as BadgeEuroIcon,
  default1504 as BadgeHelpIcon,
  default110 as BadgeIcon,
  default100 as BadgeIndianRupeeIcon,
  default99 as BadgeInfoIcon,
  default102 as BadgeJapaneseYenIcon,
  default101 as BadgeMinusIcon,
  default103 as BadgePercentIcon,
  default106 as BadgePlusIcon,
  default104 as BadgePoundSterlingIcon,
  default1503 as BadgeQuestionMarkIcon,
  default105 as BadgeRussianRubleIcon,
  default107 as BadgeSwissFrancIcon,
  default109 as BadgeTurkishLiraIcon,
  default108 as BadgeXIcon,
  default111 as BaggageClaimIcon,
  default112 as BalloonIcon,
  default113 as BanIcon,
  default114 as BananaIcon,
  default116 as BandageIcon,
  default115 as BanknoteArrowDownIcon,
  default117 as BanknoteArrowUpIcon,
  default119 as BanknoteIcon,
  default118 as BanknoteXIcon,
  default1534 as BarChart2Icon,
  default1528 as BarChart3Icon,
  default1526 as BarChart4Icon,
  default1524 as BarChartBigIcon,
  default1518 as BarChartHorizontalBigIcon,
  default1520 as BarChartHorizontalIcon,
  default1532 as BarChartIcon,
  default120 as BarcodeIcon,
  default121 as BarrelIcon,
  default122 as BaselineIcon,
  default123 as BathIcon,
  default124 as BatteryChargingIcon,
  default126 as BatteryFullIcon,
  default131 as BatteryIcon,
  default125 as BatteryLowIcon,
  default127 as BatteryMediumIcon,
  default128 as BatteryPlusIcon,
  default129 as BatteryWarningIcon,
  default130 as BeakerIcon,
  default133 as BeanIcon,
  default132 as BeanOffIcon,
  default134 as BedDoubleIcon,
  default137 as BedIcon,
  default135 as BedSingleIcon,
  default139 as BeefIcon,
  default136 as BeefOffIcon,
  default140 as BeerIcon,
  default138 as BeerOffIcon,
  default141 as BellCheckIcon,
  default142 as BellDotIcon,
  default143 as BellElectricIcon,
  default148 as BellIcon,
  default144 as BellMinusIcon,
  default145 as BellOffIcon,
  default146 as BellPlusIcon,
  default147 as BellRingIcon,
  default1506 as BetweenHorizonalEndIcon,
  default1508 as BetweenHorizonalStartIcon,
  default1505 as BetweenHorizontalEndIcon,
  default1507 as BetweenHorizontalStartIcon,
  default149 as BetweenVerticalEndIcon,
  default150 as BetweenVerticalStartIcon,
  default151 as BicepsFlexedIcon,
  default152 as BikeIcon,
  default153 as BinaryIcon,
  default154 as BinocularsIcon,
  default155 as BiohazardIcon,
  default157 as BirdIcon,
  default156 as BirdhouseIcon,
  default158 as BitcoinIcon,
  default159 as BlendIcon,
  default161 as BlenderIcon,
  default160 as BlindsIcon,
  default162 as BlocksIcon,
  default163 as BluetoothConnectedIcon,
  default166 as BluetoothIcon,
  default165 as BluetoothOffIcon,
  default164 as BluetoothSearchingIcon,
  default167 as BoldIcon,
  default168 as BoltIcon,
  default169 as BombIcon,
  default170 as BoneIcon,
  default171 as BookAIcon,
  default172 as BookAlertIcon,
  default174 as BookAudioIcon,
  default173 as BookCheckIcon,
  default175 as BookCopyIcon,
  default1509 as BookDashedIcon,
  default176 as BookDownIcon,
  default177 as BookHeadphonesIcon,
  default179 as BookHeartIcon,
  default194 as BookIcon,
  default178 as BookImageIcon,
  default180 as BookKeyIcon,
  default182 as BookLockIcon,
  default181 as BookMarkedIcon,
  default183 as BookMinusIcon,
  default186 as BookOpenCheckIcon,
  default187 as BookOpenIcon,
  default184 as BookOpenTextIcon,
  default185 as BookPlusIcon,
  default189 as BookSearchIcon,
  default1510 as BookTemplateIcon,
  default188 as BookTextIcon,
  default191 as BookTypeIcon,
  default190 as BookUp2Icon,
  default192 as BookUpIcon,
  default193 as BookUserIcon,
  default195 as BookXIcon,
  default196 as BookmarkCheckIcon,
  default201 as BookmarkIcon,
  default197 as BookmarkMinusIcon,
  default199 as BookmarkOffIcon,
  default198 as BookmarkPlusIcon,
  default200 as BookmarkXIcon,
  default203 as BoomBoxIcon,
  default205 as BotIcon,
  default202 as BotMessageSquareIcon,
  default204 as BotOffIcon,
  default206 as BottleWineIcon,
  default207 as BowArrowIcon,
  default208 as BoxIcon,
  default1850 as BoxSelectIcon,
  default209 as BoxesIcon,
  default1511 as BracesIcon,
  default210 as BracketsIcon,
  default211 as BrainCircuitIcon,
  default214 as BrainCogIcon,
  default212 as BrainIcon,
  default213 as BrickWallFireIcon,
  default216 as BrickWallIcon,
  default215 as BrickWallShieldIcon,
  default217 as BriefcaseBusinessIcon,
  default218 as BriefcaseConveyorBeltIcon,
  default219 as BriefcaseIcon,
  default220 as BriefcaseMedicalIcon,
  default221 as BringToFrontIcon,
  default222 as BroccoliIcon,
  default223 as BrushCleaningIcon,
  default224 as BrushIcon,
  default225 as BubblesIcon,
  default228 as BugIcon,
  default226 as BugOffIcon,
  default227 as BugPlayIcon,
  default229 as Building2Icon,
  default230 as BuildingIcon,
  default231 as BusFrontIcon,
  default234 as BusIcon,
  default232 as CableCarIcon,
  default235 as CableIcon,
  default236 as CakeIcon,
  default233 as CakeSliceIcon,
  default237 as CalculatorIcon,
  default238 as Calendar1Icon,
  default239 as CalendarArrowDownIcon,
  default240 as CalendarArrowUpIcon,
  default241 as CalendarCheck2Icon,
  default243 as CalendarCheckIcon,
  default242 as CalendarClockIcon,
  default245 as CalendarCogIcon,
  default244 as CalendarDaysIcon,
  default246 as CalendarFoldIcon,
  default247 as CalendarHeartIcon,
  default258 as CalendarIcon,
  default250 as CalendarMinus2Icon,
  default248 as CalendarMinusIcon,
  default249 as CalendarOffIcon,
  default251 as CalendarPlus2Icon,
  default252 as CalendarPlusIcon,
  default253 as CalendarRangeIcon,
  default255 as CalendarSearchIcon,
  default254 as CalendarSyncIcon,
  default256 as CalendarX2Icon,
  default257 as CalendarXIcon,
  default259 as CalendarsIcon,
  default261 as CameraIcon,
  default260 as CameraOffIcon,
  default1522 as CandlestickChartIcon,
  default262 as CandyCaneIcon,
  default265 as CandyIcon,
  default263 as CandyOffIcon,
  default266 as CannabisIcon,
  default264 as CannabisOffIcon,
  default1513 as CaptionsIcon,
  default269 as CaptionsOffIcon,
  default267 as CarFrontIcon,
  default270 as CarIcon,
  default268 as CarTaxiFrontIcon,
  default271 as CaravanIcon,
  default272 as CardSimIcon,
  default274 as CarrotIcon,
  default273 as CaseLowerIcon,
  default275 as CaseSensitiveIcon,
  default276 as CaseUpperIcon,
  default277 as CassetteTapeIcon,
  default278 as CastIcon,
  default280 as CastleIcon,
  default279 as CatIcon,
  default282 as CctvIcon,
  default281 as CctvOffIcon,
  default1515 as ChartAreaIcon,
  default1517 as ChartBarBigIcon,
  default283 as ChartBarDecreasingIcon,
  default1519 as ChartBarIcon,
  default284 as ChartBarIncreasingIcon,
  default285 as ChartBarStackedIcon,
  default1521 as ChartCandlestickIcon,
  default1523 as ChartColumnBigIcon,
  default286 as ChartColumnDecreasingIcon,
  default1527 as ChartColumnIcon,
  default1525 as ChartColumnIncreasingIcon,
  default287 as ChartColumnStackedIcon,
  default288 as ChartGanttIcon,
  default1529 as ChartLineIcon,
  default289 as ChartNetworkIcon,
  default290 as ChartNoAxesColumnDecreasingIcon,
  default1533 as ChartNoAxesColumnIcon,
  default1531 as ChartNoAxesColumnIncreasingIcon,
  default291 as ChartNoAxesCombinedIcon,
  default1535 as ChartNoAxesGanttIcon,
  default1537 as ChartPieIcon,
  default1541 as ChartScatterIcon,
  default292 as ChartSplineIcon,
  default293 as CheckCheckIcon,
  default1564 as CheckCircle2Icon,
  default1560 as CheckCircleIcon,
  default294 as CheckIcon,
  default295 as CheckLineIcon,
  default1831 as CheckSquare2Icon,
  default1829 as CheckSquareIcon,
  default296 as ChefHatIcon,
  default297 as CherryIcon,
  default298 as ChessBishopIcon,
  default299 as ChessKingIcon,
  default300 as ChessKnightIcon,
  default301 as ChessPawnIcon,
  default303 as ChessQueenIcon,
  default304 as ChessRookIcon,
  default1562 as ChevronDownCircleIcon,
  default302 as ChevronDownIcon,
  default1837 as ChevronDownSquareIcon,
  default305 as ChevronFirstIcon,
  default306 as ChevronLastIcon,
  default1566 as ChevronLeftCircleIcon,
  default307 as ChevronLeftIcon,
  default1835 as ChevronLeftSquareIcon,
  default1568 as ChevronRightCircleIcon,
  default309 as ChevronRightIcon,
  default1833 as ChevronRightSquareIcon,
  default1572 as ChevronUpCircleIcon,
  default308 as ChevronUpIcon,
  default1839 as ChevronUpSquareIcon,
  default312 as ChevronsDownIcon,
  default310 as ChevronsDownUpIcon,
  default314 as ChevronsLeftIcon,
  default311 as ChevronsLeftRightEllipsisIcon,
  default313 as ChevronsLeftRightIcon,
  default316 as ChevronsRightIcon,
  default315 as ChevronsRightLeftIcon,
  default317 as ChevronsUpDownIcon,
  default318 as ChevronsUpIcon,
  default319 as ChurchIcon,
  default321 as CigaretteIcon,
  default320 as CigaretteOffIcon,
  default1539 as CircleAlertIcon,
  default1543 as CircleArrowDownIcon,
  default1545 as CircleArrowLeftIcon,
  default1547 as CircleArrowOutDownLeftIcon,
  default1549 as CircleArrowOutDownRightIcon,
  default1551 as CircleArrowOutUpLeftIcon,
  default1553 as CircleArrowOutUpRightIcon,
  default1555 as CircleArrowRightIcon,
  default1557 as CircleArrowUpIcon,
  default1559 as CircleCheckBigIcon,
  default1563 as CircleCheckIcon,
  default1561 as CircleChevronDownIcon,
  default1565 as CircleChevronLeftIcon,
  default1567 as CircleChevronRightIcon,
  default1571 as CircleChevronUpIcon,
  default322 as CircleDashedIcon,
  default1569 as CircleDivideIcon,
  default323 as CircleDollarSignIcon,
  default324 as CircleDotDashedIcon,
  default325 as CircleDotIcon,
  default326 as CircleEllipsisIcon,
  default328 as CircleEqualIcon,
  default327 as CircleFadingArrowUpIcon,
  default329 as CircleFadingPlusIcon,
  default1577 as CircleGaugeIcon,
  default1595 as CircleHelpIcon,
  default336 as CircleIcon,
  default1573 as CircleMinusIcon,
  default330 as CircleOffIcon,
  default1585 as CircleParkingIcon,
  default1575 as CircleParkingOffIcon,
  default1579 as CirclePauseIcon,
  default1583 as CirclePercentIcon,
  default331 as CirclePileIcon,
  default1581 as CirclePlayIcon,
  default1587 as CirclePlusIcon,
  default332 as CirclePoundSterlingIcon,
  default1589 as CirclePowerIcon,
  default1593 as CircleQuestionMarkIcon,
  default1591 as CircleSlash2Icon,
  default333 as CircleSlashIcon,
  default1592 as CircleSlashedIcon,
  default335 as CircleSmallIcon,
  default334 as CircleStarIcon,
  default1596 as CircleStopIcon,
  default1600 as CircleUserIcon,
  default1598 as CircleUserRoundIcon,
  default1602 as CircleXIcon,
  default337 as CircuitBoardIcon,
  default338 as CitrusIcon,
  default340 as ClapperboardIcon,
  default339 as ClipboardCheckIcon,
  default341 as ClipboardClockIcon,
  default342 as ClipboardCopyIcon,
  default1607 as ClipboardEditIcon,
  default349 as ClipboardIcon,
  default343 as ClipboardListIcon,
  default344 as ClipboardMinusIcon,
  default345 as ClipboardPasteIcon,
  default1606 as ClipboardPenIcon,
  default1604 as ClipboardPenLineIcon,
  default346 as ClipboardPlusIcon,
  default1605 as ClipboardSignatureIcon,
  default347 as ClipboardTypeIcon,
  default348 as ClipboardXIcon,
  default350 as Clock10Icon,
  default352 as Clock11Icon,
  default353 as Clock12Icon,
  default351 as Clock1Icon,
  default354 as Clock2Icon,
  default355 as Clock3Icon,
  default356 as Clock4Icon,
  default357 as Clock5Icon,
  default358 as Clock6Icon,
  default359 as Clock7Icon,
  default360 as Clock8Icon,
  default364 as Clock9Icon,
  default361 as ClockAlertIcon,
  default362 as ClockArrowDownIcon,
  default363 as ClockArrowUpIcon,
  default365 as ClockCheckIcon,
  default366 as ClockFadingIcon,
  default369 as ClockIcon,
  default367 as ClockPlusIcon,
  default368 as ClosedCaptionIcon,
  default370 as CloudAlertIcon,
  default371 as CloudBackupIcon,
  default372 as CloudCheckIcon,
  default373 as CloudCogIcon,
  default1608 as CloudDownloadIcon,
  default374 as CloudDrizzleIcon,
  default375 as CloudFogIcon,
  default376 as CloudHailIcon,
  default387 as CloudIcon,
  default377 as CloudLightningIcon,
  default380 as CloudMoonIcon,
  default378 as CloudMoonRainIcon,
  default379 as CloudOffIcon,
  default382 as CloudRainIcon,
  default381 as CloudRainWindIcon,
  default383 as CloudSnowIcon,
  default385 as CloudSunIcon,
  default384 as CloudSunRainIcon,
  default386 as CloudSyncIcon,
  default1610 as CloudUploadIcon,
  default388 as CloudyIcon,
  default389 as CloverIcon,
  default390 as ClubIcon,
  default1613 as Code2Icon,
  default392 as CodeIcon,
  default1841 as CodeSquareIcon,
  default1612 as CodeXmlIcon,
  default391 as CoffeeIcon,
  default393 as CogIcon,
  default394 as CoinsIcon,
  default1614 as Columns2Icon,
  default1616 as Columns3CogIcon,
  default1619 as Columns3Icon,
  default395 as Columns4Icon,
  default1615 as ColumnsIcon,
  default1617 as ColumnsSettingsIcon,
  default396 as CombineIcon,
  default397 as CommandIcon,
  default398 as CompassIcon,
  default399 as ComponentIcon,
  default402 as ComputerIcon,
  default400 as ConciergeBellIcon,
  default401 as ConeIcon,
  default403 as ConstructionIcon,
  default1622 as Contact2Icon,
  default404 as ContactIcon,
  default1621 as ContactRoundIcon,
  default405 as ContainerIcon,
  default406 as ContrastIcon,
  default407 as CookieIcon,
  default408 as CookingPotIcon,
  default409 as CopyCheckIcon,
  default414 as CopyIcon,
  default410 as CopyMinusIcon,
  default411 as CopyPlusIcon,
  default413 as CopySlashIcon,
  default412 as CopyXIcon,
  default415 as CopyleftIcon,
  default417 as CopyrightIcon,
  default416 as CornerDownLeftIcon,
  default418 as CornerDownRightIcon,
  default419 as CornerLeftDownIcon,
  default422 as CornerLeftUpIcon,
  default420 as CornerRightDownIcon,
  default421 as CornerRightUpIcon,
  default423 as CornerUpLeftIcon,
  default425 as CornerUpRightIcon,
  default424 as CpuIcon,
  default428 as CreativeCommonsIcon,
  default426 as CreditCardIcon,
  default427 as CroissantIcon,
  default430 as CropIcon,
  default432 as CrossIcon,
  default429 as CrosshairIcon,
  default431 as CrownIcon,
  default433 as CuboidIcon,
  default434 as CupSodaIcon,
  default1512 as CurlyBracesIcon,
  default435 as CurrencyIcon,
  default436 as CylinderIcon,
  default437 as DamIcon,
  default438 as DatabaseBackupIcon,
  default441 as DatabaseIcon,
  default439 as DatabaseSearchIcon,
  default440 as DatabaseZapIcon,
  default442 as DecimalsArrowLeftIcon,
  default443 as DecimalsArrowRightIcon,
  default444 as DeleteIcon,
  default445 as DessertIcon,
  default447 as DiameterIcon,
  default449 as DiamondIcon,
  default446 as DiamondMinusIcon,
  default1623 as DiamondPercentIcon,
  default448 as DiamondPlusIcon,
  default450 as Dice1Icon,
  default451 as Dice2Icon,
  default452 as Dice3Icon,
  default453 as Dice4Icon,
  default454 as Dice5Icon,
  default455 as Dice6Icon,
  default456 as DicesIcon,
  default457 as DiffIcon,
  default458 as Disc2Icon,
  default460 as Disc3Icon,
  default459 as DiscAlbumIcon,
  default461 as DiscIcon,
  default1570 as DivideCircleIcon,
  default463 as DivideIcon,
  default1852 as DivideSquareIcon,
  default464 as DnaIcon,
  default462 as DnaOffIcon,
  default465 as DockIcon,
  default466 as DogIcon,
  default467 as DollarSignIcon,
  default468 as DonutIcon,
  default470 as DoorClosedIcon,
  default469 as DoorClosedLockedIcon,
  default471 as DoorOpenIcon,
  default472 as DotIcon,
  default1854 as DotSquareIcon,
  default1609 as DownloadCloudIcon,
  default473 as DownloadIcon,
  default474 as DraftingCompassIcon,
  default475 as DramaIcon,
  default476 as DrillIcon,
  default477 as DroneIcon,
  default479 as DropletIcon,
  default478 as DropletOffIcon,
  default480 as DropletsIcon,
  default481 as DrumIcon,
  default482 as DrumstickIcon,
  default483 as DumbbellIcon,
  default485 as EarIcon,
  default484 as EarOffIcon,
  default1625 as EarthIcon,
  default486 as EarthLockIcon,
  default489 as EclipseIcon,
  default1768 as Edit2Icon,
  default1766 as Edit3Icon,
  default1878 as EditIcon,
  default487 as EggFriedIcon,
  default490 as EggIcon,
  default488 as EggOffIcon,
  default491 as EllipseIcon,
  default1629 as EllipsisIcon,
  default1627 as EllipsisVerticalIcon,
  default492 as EqualApproximatelyIcon,
  default494 as EqualIcon,
  default493 as EqualNotIcon,
  default1858 as EqualSquareIcon,
  default495 as EraserIcon,
  default496 as EthernetPortIcon,
  default497 as EuroIcon,
  default499 as EvChargerIcon,
  default498 as ExpandIcon,
  default500 as ExternalLinkIcon,
  default501 as EyeClosedIcon,
  default504 as EyeIcon,
  default502 as EyeOffIcon,
  default503 as FactoryIcon,
  default505 as FanIcon,
  default506 as FastForwardIcon,
  default507 as FeatherIcon,
  default508 as FenceIcon,
  default509 as FerrisWheelIcon,
  default510 as FileArchiveIcon,
  default1657 as FileAudio2Icon,
  default1656 as FileAudioIcon,
  default1632 as FileAxis3DIcon,
  default1631 as FileAxis3dIcon,
  default1634 as FileBadge2Icon,
  default1633 as FileBadgeIcon,
  default1644 as FileBarChart2Icon,
  default1642 as FileBarChartIcon,
  default511 as FileBoxIcon,
  default1635 as FileBracesCornerIcon,
  default1637 as FileBracesIcon,
  default1643 as FileChartColumnIcon,
  default1641 as FileChartColumnIncreasingIcon,
  default1639 as FileChartLineIcon,
  default1645 as FileChartPieIcon,
  default1648 as FileCheck2Icon,
  default1647 as FileCheckCornerIcon,
  default512 as FileCheckIcon,
  default513 as FileClockIcon,
  default1650 as FileCode2Icon,
  default1649 as FileCodeCornerIcon,
  default514 as FileCodeIcon,
  default1652 as FileCog2Icon,
  default1651 as FileCogIcon,
  default515 as FileDiffIcon,
  default517 as FileDigitIcon,
  default516 as FileDownIcon,
  default1665 as FileEditIcon,
  default1653 as FileExclamationPointIcon,
  default1655 as FileHeadphoneIcon,
  default518 as FileHeartIcon,
  default538 as FileIcon,
  default519 as FileImageIcon,
  default520 as FileInputIcon,
  default1636 as FileJson2Icon,
  default1638 as FileJsonIcon,
  default1663 as FileKey2Icon,
  default1662 as FileKeyIcon,
  default1640 as FileLineChartIcon,
  default1659 as FileLock2Icon,
  default1658 as FileLockIcon,
  default1661 as FileMinus2Icon,
  default1660 as FileMinusCornerIcon,
  default521 as FileMinusIcon,
  default522 as FileMusicIcon,
  default523 as FileOutputIcon,
  default1664 as FilePenIcon,
  default1666 as FilePenLineIcon,
  default1646 as FilePieChartIcon,
  default1668 as FilePlayIcon,
  default1671 as FilePlus2Icon,
  default1670 as FilePlusCornerIcon,
  default524 as FilePlusIcon,
  default1675 as FileQuestionIcon,
  default1674 as FileQuestionMarkIcon,
  default525 as FileScanIcon,
  default1673 as FileSearch2Icon,
  default1672 as FileSearchCornerIcon,
  default526 as FileSearchIcon,
  default1676 as FileSignalIcon,
  default1667 as FileSignatureIcon,
  default528 as FileSlidersIcon,
  default527 as FileSpreadsheetIcon,
  default529 as FileStackIcon,
  default530 as FileSymlinkIcon,
  default531 as FileTerminalIcon,
  default532 as FileTextIcon,
  default1679 as FileType2Icon,
  default1678 as FileTypeCornerIcon,
  default533 as FileTypeIcon,
  default534 as FileUpIcon,
  default535 as FileUserIcon,
  default1681 as FileVideo2Icon,
  default1680 as FileVideoCameraIcon,
  default1669 as FileVideoIcon,
  default1677 as FileVolume2Icon,
  default536 as FileVolumeIcon,
  default1654 as FileWarningIcon,
  default1683 as FileX2Icon,
  default1682 as FileXCornerIcon,
  default537 as FileXIcon,
  default539 as FilesIcon,
  default540 as FilmIcon,
  default1691 as FilterIcon,
  default1689 as FilterXIcon,
  default1476 as FingerprintIcon,
  default1475 as FingerprintPatternIcon,
  default542 as FireExtinguisherIcon,
  default543 as FishIcon,
  default541 as FishOffIcon,
  default545 as FishSymbolIcon,
  default544 as FishingHookIcon,
  default546 as FishingRodIcon,
  default550 as FlagIcon,
  default547 as FlagOffIcon,
  default548 as FlagTriangleLeftIcon,
  default549 as FlagTriangleRightIcon,
  default552 as FlameIcon,
  default551 as FlameKindlingIcon,
  default554 as FlashlightIcon,
  default553 as FlashlightOffIcon,
  default556 as FlaskConicalIcon,
  default555 as FlaskConicalOffIcon,
  default557 as FlaskRoundIcon,
  default558 as FlipHorizontal2Icon,
  default1822 as FlipHorizontalIcon,
  default559 as FlipVertical2Icon,
  default1820 as FlipVerticalIcon,
  default560 as Flower2Icon,
  default561 as FlowerIcon,
  default562 as FocusIcon,
  default563 as FoldHorizontalIcon,
  default564 as FoldVerticalIcon,
  default565 as FolderArchiveIcon,
  default568 as FolderBookmarkIcon,
  default566 as FolderCheckIcon,
  default567 as FolderClockIcon,
  default569 as FolderClosedIcon,
  default570 as FolderCodeIcon,
  default1685 as FolderCog2Icon,
  default1684 as FolderCogIcon,
  default571 as FolderDotIcon,
  default572 as FolderDownIcon,
  default1687 as FolderEditIcon,
  default573 as FolderGit2Icon,
  default574 as FolderGitIcon,
  default575 as FolderHeartIcon,
  default592 as FolderIcon,
  default576 as FolderInputIcon,
  default577 as FolderKanbanIcon,
  default578 as FolderKeyIcon,
  default579 as FolderLockIcon,
  default580 as FolderMinusIcon,
  default581 as FolderOpenDotIcon,
  default583 as FolderOpenIcon,
  default582 as FolderOutputIcon,
  default1686 as FolderPenIcon,
  default585 as FolderPlusIcon,
  default584 as FolderRootIcon,
  default586 as FolderSearch2Icon,
  default588 as FolderSearchIcon,
  default587 as FolderSymlinkIcon,
  default589 as FolderSyncIcon,
  default590 as FolderTreeIcon,
  default591 as FolderUpIcon,
  default593 as FolderXIcon,
  default594 as FoldersIcon,
  default595 as FootprintsIcon,
  default1947 as ForkKnifeCrossedIcon,
  default1951 as ForkKnifeIcon,
  default596 as ForkliftIcon,
  default598 as FormIcon,
  default1772 as FormInputIcon,
  default597 as ForwardIcon,
  default599 as FrameIcon,
  default600 as FrownIcon,
  default601 as FuelIcon,
  default602 as FullscreenIcon,
  default1856 as FunctionSquareIcon,
  default1690 as FunnelIcon,
  default603 as FunnelPlusIcon,
  default1688 as FunnelXIcon,
  default604 as GalleryHorizontalEndIcon,
  default606 as GalleryHorizontalIcon,
  default605 as GalleryThumbnailsIcon,
  default608 as GalleryVerticalEndIcon,
  default607 as GalleryVerticalIcon,
  default609 as Gamepad2Icon,
  default610 as GamepadDirectionalIcon,
  default611 as GamepadIcon,
  default1536 as GanttChartIcon,
  default1826 as GanttChartSquareIcon,
  default1578 as GaugeCircleIcon,
  default612 as GaugeIcon,
  default613 as GavelIcon,
  default614 as GemIcon,
  default615 as GeorgianLariIcon,
  default616 as GhostIcon,
  default617 as GiftIcon,
  default620 as GitBranchIcon,
  default618 as GitBranchMinusIcon,
  default619 as GitBranchPlusIcon,
  default1692 as GitCommitHorizontalIcon,
  default1693 as GitCommitIcon,
  default621 as GitCommitVerticalIcon,
  default622 as GitCompareArrowsIcon,
  default623 as GitCompareIcon,
  default625 as GitForkIcon,
  default624 as GitGraphIcon,
  default626 as GitMergeConflictIcon,
  default627 as GitMergeIcon,
  default629 as GitPullRequestArrowIcon,
  default628 as GitPullRequestClosedIcon,
  default630 as GitPullRequestCreateArrowIcon,
  default631 as GitPullRequestCreateIcon,
  default632 as GitPullRequestDraftIcon,
  default633 as GitPullRequestIcon,
  default634 as GlassWaterIcon,
  default636 as GlassesIcon,
  default1626 as Globe2Icon,
  default639 as GlobeIcon,
  default635 as GlobeLockIcon,
  default637 as GlobeOffIcon,
  default638 as GlobeXIcon,
  default640 as GoalIcon,
  default641 as GpuIcon,
  default1706 as GrabIcon,
  default643 as GraduationCapIcon,
  default642 as GrapeIcon,
  default1695 as Grid2X2CheckIcon,
  default1701 as Grid2X2Icon,
  default1697 as Grid2X2PlusIcon,
  default1699 as Grid2X2XIcon,
  default1694 as Grid2x2CheckIcon,
  default1700 as Grid2x2Icon,
  default1696 as Grid2x2PlusIcon,
  default1698 as Grid2x2XIcon,
  default1704 as Grid3X3Icon,
  default644 as Grid3x2Icon,
  default1702 as Grid3x3Icon,
  default1703 as GridIcon,
  default645 as GripHorizontalIcon,
  default648 as GripIcon,
  default646 as GripVerticalIcon,
  default647 as GroupIcon,
  default649 as GuitarIcon,
  default650 as HamIcon,
  default651 as HamburgerIcon,
  default652 as HammerIcon,
  default653 as HandCoinsIcon,
  default654 as HandFistIcon,
  default1705 as HandGrabIcon,
  default655 as HandHeartIcon,
  default1707 as HandHelpingIcon,
  default658 as HandIcon,
  default656 as HandMetalIcon,
  default657 as HandPlatterIcon,
  default659 as HandbagIcon,
  default660 as HandshakeIcon,
  default661 as HardDriveDownloadIcon,
  default663 as HardDriveIcon,
  default662 as HardDriveUploadIcon,
  default664 as HardHatIcon,
  default665 as HashIcon,
  default666 as HatGlassesIcon,
  default667 as HazeIcon,
  default668 as HdIcon,
  default669 as HdmiPortIcon,
  default670 as Heading1Icon,
  default671 as Heading2Icon,
  default672 as Heading3Icon,
  default673 as Heading4Icon,
  default674 as Heading5Icon,
  default675 as Heading6Icon,
  default677 as HeadingIcon,
  default676 as HeadphoneOffIcon,
  default678 as HeadphonesIcon,
  default679 as HeadsetIcon,
  default680 as HeartCrackIcon,
  default681 as HeartHandshakeIcon,
  default687 as HeartIcon,
  default683 as HeartMinusIcon,
  default682 as HeartOffIcon,
  default684 as HeartPlusIcon,
  default685 as HeartPulseIcon,
  default686 as HeartXIcon,
  default689 as HeaterIcon,
  default688 as HelicopterIcon,
  default1594 as HelpCircleIcon,
  default1708 as HelpingHandIcon,
  default690 as HexagonIcon,
  default691 as HighlighterIcon,
  default693 as HistoryIcon,
  default1710 as HomeIcon,
  default694 as HopIcon,
  default692 as HopOffIcon,
  default696 as HospitalIcon,
  default695 as HotelIcon,
  default697 as HourglassIcon,
  default698 as HouseHeartIcon,
  default1709 as HouseIcon,
  default699 as HousePlugIcon,
  default701 as HousePlusIcon,
  default700 as HouseWifiIcon,
  default1712 as IceCream2Icon,
  default1711 as IceCreamBowlIcon,
  default1713 as IceCreamConeIcon,
  default1714 as IceCreamIcon,
  default703 as IdCardIcon,
  default702 as IdCardLanyardIcon,
  default704 as ImageDownIcon,
  default711 as ImageIcon,
  default705 as ImageMinusIcon,
  default707 as ImageOffIcon,
  default706 as ImagePlayIcon,
  default708 as ImagePlusIcon,
  default709 as ImageUpIcon,
  default710 as ImageUpscaleIcon,
  default712 as ImagesIcon,
  default713 as ImportIcon,
  default714 as InboxIcon,
  default1721 as IndentDecreaseIcon,
  default1723 as IndentIcon,
  default1724 as IndentIncreaseIcon,
  default715 as IndianRupeeIcon,
  default717 as InfinityIcon,
  default716 as InfoIcon,
  default1870 as InspectIcon,
  default718 as InspectionPanelIcon,
  default719 as ItalicIcon,
  default720 as IterationCcwIcon,
  default721 as IterationCwIcon,
  default722 as JapaneseYenIcon,
  default723 as JoystickIcon,
  default726 as KanbanIcon,
  default1843 as KanbanSquareDashedIcon,
  default1862 as KanbanSquareIcon,
  default724 as KayakIcon,
  default728 as KeyIcon,
  default725 as KeyRoundIcon,
  default727 as KeySquareIcon,
  default730 as KeyboardIcon,
  default729 as KeyboardMusicIcon,
  default731 as KeyboardOffIcon,
  default732 as LampCeilingIcon,
  default733 as LampDeskIcon,
  default734 as LampFloorIcon,
  default737 as LampIcon,
  default735 as LampWallDownIcon,
  default736 as LampWallUpIcon,
  default738 as LandPlotIcon,
  default740 as LandmarkIcon,
  default741 as LanguagesIcon,
  default1716 as Laptop2Icon,
  default742 as LaptopIcon,
  default739 as LaptopMinimalCheckIcon,
  default1715 as LaptopMinimalIcon,
  default744 as LassoIcon,
  default743 as LassoSelectIcon,
  default745 as LaughIcon,
  default746 as Layers2Icon,
  default1718 as Layers3Icon,
  default1717 as LayersIcon,
  default748 as LayersMinusIcon,
  default747 as LayersPlusIcon,
  default749 as LayoutDashboardIcon,
  default750 as LayoutGridIcon,
  default1764 as LayoutIcon,
  default751 as LayoutListIcon,
  default752 as LayoutPanelLeftIcon,
  default753 as LayoutPanelTopIcon,
  default754 as LayoutTemplateIcon,
  default756 as LeafIcon,
  default755 as LeafyGreenIcon,
  default757 as LecternIcon,
  default758 as LensConcaveIcon,
  default759 as LensConvexIcon,
  default1923 as LetterTextIcon,
  default761 as LibraryBigIcon,
  default760 as LibraryIcon,
  default1860 as LibrarySquareIcon,
  default762 as LifeBuoyIcon,
  default763 as LigatureIcon,
  default765 as LightbulbIcon,
  default764 as LightbulbOffIcon,
  default1530 as LineChartIcon,
  default766 as LineDotRightHorizontalIcon,
  default767 as LineSquiggleIcon,
  default768 as LineStyleIcon,
  default770 as Link2Icon,
  default769 as Link2OffIcon,
  default772 as LinkIcon,
  default771 as ListCheckIcon,
  default773 as ListChecksIcon,
  default774 as ListChevronsDownUpIcon,
  default775 as ListChevronsUpDownIcon,
  default777 as ListCollapseIcon,
  default776 as ListEndIcon,
  default779 as ListFilterIcon,
  default778 as ListFilterPlusIcon,
  default790 as ListIcon,
  default1719 as ListIndentDecreaseIcon,
  default1722 as ListIndentIncreaseIcon,
  default780 as ListMinusIcon,
  default781 as ListMusicIcon,
  default782 as ListOrderedIcon,
  default783 as ListPlusIcon,
  default784 as ListRestartIcon,
  default785 as ListStartIcon,
  default786 as ListTodoIcon,
  default787 as ListTreeIcon,
  default788 as ListVideoIcon,
  default789 as ListXIcon,
  default1726 as Loader2Icon,
  default1725 as LoaderCircleIcon,
  default792 as LoaderIcon,
  default791 as LoaderPinwheelIcon,
  default793 as LocateFixedIcon,
  default795 as LocateIcon,
  default794 as LocateOffIcon,
  default1734 as LocationEditIcon,
  default797 as LockIcon,
  default796 as LockKeyholeIcon,
  default1729 as LockKeyholeOpenIcon,
  default1727 as LockOpenIcon,
  default798 as LogInIcon,
  default800 as LogOutIcon,
  default799 as LogsIcon,
  default801 as LollipopIcon,
  default802 as LuggageIcon,
  default1864 as MSquareIcon,
  default803 as MagnetIcon,
  default804 as MailCheckIcon,
  default811 as MailIcon,
  default805 as MailMinusIcon,
  default806 as MailOpenIcon,
  default807 as MailPlusIcon,
  default1732 as MailQuestionIcon,
  default1731 as MailQuestionMarkIcon,
  default808 as MailSearchIcon,
  default809 as MailWarningIcon,
  default810 as MailXIcon,
  default812 as MailboxIcon,
  default813 as MailsIcon,
  default829 as MapIcon,
  default815 as MapMinusIcon,
  default816 as MapPinCheckIcon,
  default814 as MapPinCheckInsideIcon,
  default817 as MapPinHouseIcon,
  default826 as MapPinIcon,
  default819 as MapPinMinusIcon,
  default818 as MapPinMinusInsideIcon,
  default820 as MapPinOffIcon,
  default1733 as MapPinPenIcon,
  default822 as MapPinPlusIcon,
  default821 as MapPinPlusInsideIcon,
  default823 as MapPinSearchIcon,
  default825 as MapPinXIcon,
  default824 as MapPinXInsideIcon,
  default827 as MapPinnedIcon,
  default828 as MapPlusIcon,
  default832 as MarsIcon,
  default830 as MarsStrokeIcon,
  default831 as MartiniIcon,
  default833 as Maximize2Icon,
  default835 as MaximizeIcon,
  default834 as MedalIcon,
  default836 as MegaphoneIcon,
  default837 as MegaphoneOffIcon,
  default838 as MehIcon,
  default840 as MemoryStickIcon,
  default839 as MenuIcon,
  default1866 as MenuSquareIcon,
  default841 as MergeIcon,
  default842 as MessageCircleCheckIcon,
  default843 as MessageCircleCodeIcon,
  default844 as MessageCircleDashedIcon,
  default845 as MessageCircleHeartIcon,
  default852 as MessageCircleIcon,
  default847 as MessageCircleMoreIcon,
  default846 as MessageCircleOffIcon,
  default848 as MessageCirclePlusIcon,
  default1736 as MessageCircleQuestionIcon,
  default1735 as MessageCircleQuestionMarkIcon,
  default849 as MessageCircleReplyIcon,
  default850 as MessageCircleWarningIcon,
  default851 as MessageCircleXIcon,
  default853 as MessageSquareCheckIcon,
  default855 as MessageSquareCodeIcon,
  default854 as MessageSquareDashedIcon,
  default856 as MessageSquareDiffIcon,
  default857 as MessageSquareDotIcon,
  default858 as MessageSquareHeartIcon,
  default869 as MessageSquareIcon,
  default859 as MessageSquareLockIcon,
  default860 as MessageSquareMoreIcon,
  default862 as MessageSquareOffIcon,
  default865 as MessageSquarePlusIcon,
  default861 as MessageSquareQuoteIcon,
  default863 as MessageSquareReplyIcon,
  default864 as MessageSquareShareIcon,
  default866 as MessageSquareTextIcon,
  default867 as MessageSquareWarningIcon,
  default868 as MessageSquareXIcon,
  default870 as MessagesSquareIcon,
  default871 as MetronomeIcon,
  default1738 as Mic2Icon,
  default874 as MicIcon,
  default872 as MicOffIcon,
  default1737 as MicVocalIcon,
  default873 as MicrochipIcon,
  default875 as MicroscopeIcon,
  default876 as MicrowaveIcon,
  default877 as MilestoneIcon,
  default879 as MilkIcon,
  default878 as MilkOffIcon,
  default880 as Minimize2Icon,
  default881 as MinimizeIcon,
  default1574 as MinusCircleIcon,
  default883 as MinusIcon,
  default1868 as MinusSquareIcon,
  default882 as MirrorRectangularIcon,
  default884 as MirrorRoundIcon,
  default885 as MonitorCheckIcon,
  default886 as MonitorCloudIcon,
  default887 as MonitorCogIcon,
  default888 as MonitorDotIcon,
  default889 as MonitorDownIcon,
  default897 as MonitorIcon,
  default890 as MonitorOffIcon,
  default891 as MonitorPauseIcon,
  default892 as MonitorPlayIcon,
  default893 as MonitorSmartphoneIcon,
  default895 as MonitorSpeakerIcon,
  default894 as MonitorStopIcon,
  default896 as MonitorUpIcon,
  default898 as MonitorXIcon,
  default900 as MoonIcon,
  default899 as MoonStarIcon,
  default1630 as MoreHorizontalIcon,
  default1628 as MoreVerticalIcon,
  default901 as MotorbikeIcon,
  default903 as MountainIcon,
  default902 as MountainSnowIcon,
  default912 as MouseIcon,
  default904 as MouseLeftIcon,
  default906 as MouseOffIcon,
  default907 as MousePointer2Icon,
  default905 as MousePointer2OffIcon,
  default908 as MousePointerBanIcon,
  default909 as MousePointerClickIcon,
  default910 as MousePointerIcon,
  default1845 as MousePointerSquareDashedIcon,
  default911 as MouseRightIcon,
  default1744 as Move3DIcon,
  default1743 as Move3dIcon,
  default913 as MoveDiagonal2Icon,
  default914 as MoveDiagonalIcon,
  default918 as MoveDownIcon,
  default915 as MoveDownLeftIcon,
  default916 as MoveDownRightIcon,
  default917 as MoveHorizontalIcon,
  default924 as MoveIcon,
  default920 as MoveLeftIcon,
  default919 as MoveRightIcon,
  default923 as MoveUpIcon,
  default921 as MoveUpLeftIcon,
  default922 as MoveUpRightIcon,
  default925 as MoveVerticalIcon,
  default926 as Music2Icon,
  default927 as Music3Icon,
  default928 as Music4Icon,
  default931 as MusicIcon,
  default930 as Navigation2Icon,
  default929 as Navigation2OffIcon,
  default933 as NavigationIcon,
  default932 as NavigationOffIcon,
  default934 as NetworkIcon,
  default935 as NewspaperIcon,
  default936 as NfcIcon,
  default937 as NonBinaryIcon,
  default943 as NotebookIcon,
  default938 as NotebookPenIcon,
  default939 as NotebookTabsIcon,
  default940 as NotebookTextIcon,
  default941 as NotepadTextDashedIcon,
  default942 as NotepadTextIcon,
  default945 as NutIcon,
  default944 as NutOffIcon,
  default1741 as OctagonAlertIcon,
  default947 as OctagonIcon,
  default946 as OctagonMinusIcon,
  default1739 as OctagonPauseIcon,
  default1745 as OctagonXIcon,
  default948 as OmegaIcon,
  default950 as OptionIcon,
  default949 as OrbitIcon,
  default951 as OrigamiIcon,
  default1720 as OutdentIcon,
  default952 as Package2Icon,
  default955 as PackageCheckIcon,
  default960 as PackageIcon,
  default953 as PackageMinusIcon,
  default954 as PackageOpenIcon,
  default956 as PackagePlusIcon,
  default957 as PackageSearchIcon,
  default958 as PackageXIcon,
  default959 as PaintBucketIcon,
  default961 as PaintRollerIcon,
  default1750 as Paintbrush2Icon,
  default962 as PaintbrushIcon,
  default1749 as PaintbrushVerticalIcon,
  default963 as PaletteIcon,
  default1927 as PalmtreeIcon,
  default964 as PandaIcon,
  default965 as PanelBottomCloseIcon,
  default1751 as PanelBottomDashedIcon,
  default967 as PanelBottomIcon,
  default1752 as PanelBottomInactiveIcon,
  default966 as PanelBottomOpenIcon,
  default1747 as PanelLeftCloseIcon,
  default1753 as PanelLeftDashedIcon,
  default1757 as PanelLeftIcon,
  default1754 as PanelLeftInactiveIcon,
  default1755 as PanelLeftOpenIcon,
  default968 as PanelLeftRightDashedIcon,
  default969 as PanelRightCloseIcon,
  default1759 as PanelRightDashedIcon,
  default973 as PanelRightIcon,
  default1760 as PanelRightInactiveIcon,
  default970 as PanelRightOpenIcon,
  default971 as PanelTopBottomDashedIcon,
  default972 as PanelTopCloseIcon,
  default1761 as PanelTopDashedIcon,
  default975 as PanelTopIcon,
  default1762 as PanelTopInactiveIcon,
  default974 as PanelTopOpenIcon,
  default976 as PanelsLeftBottomIcon,
  default1620 as PanelsLeftRightIcon,
  default977 as PanelsRightBottomIcon,
  default1780 as PanelsTopBottomIcon,
  default1763 as PanelsTopLeftIcon,
  default978 as PaperclipIcon,
  default979 as ParenthesesIcon,
  default1586 as ParkingCircleIcon,
  default1576 as ParkingCircleOffIcon,
  default981 as ParkingMeterIcon,
  default1874 as ParkingSquareIcon,
  default1872 as ParkingSquareOffIcon,
  default980 as PartyPopperIcon,
  default1580 as PauseCircleIcon,
  default983 as PauseIcon,
  default1740 as PauseOctagonIcon,
  default982 as PawPrintIcon,
  default985 as PcCaseIcon,
  default1879 as PenBoxIcon,
  default1767 as PenIcon,
  default1765 as PenLineIcon,
  default984 as PenOffIcon,
  default1880 as PenSquareIcon,
  default986 as PenToolIcon,
  default990 as PencilIcon,
  default987 as PencilLineIcon,
  default988 as PencilOffIcon,
  default989 as PencilRulerIcon,
  default991 as PentagonIcon,
  default1584 as PercentCircleIcon,
  default1624 as PercentDiamondIcon,
  default992 as PercentIcon,
  default1876 as PercentSquareIcon,
  default993 as PersonStandingIcon,
  default994 as PhilippinePesoIcon,
  default995 as PhoneCallIcon,
  default996 as PhoneForwardedIcon,
  default1001 as PhoneIcon,
  default997 as PhoneIncomingIcon,
  default999 as PhoneMissedIcon,
  default998 as PhoneOffIcon,
  default1000 as PhoneOutgoingIcon,
  default1002 as PiIcon,
  default1884 as PiSquareIcon,
  default1003 as PianoIcon,
  default1004 as PickaxeIcon,
  default1005 as PictureInPicture2Icon,
  default1006 as PictureInPictureIcon,
  default1538 as PieChartIcon,
  default1007 as PiggyBankIcon,
  default1010 as PilcrowIcon,
  default1008 as PilcrowLeftIcon,
  default1009 as PilcrowRightIcon,
  default1886 as PilcrowSquareIcon,
  default1011 as PillBottleIcon,
  default1012 as PillIcon,
  default1015 as PinIcon,
  default1014 as PinOffIcon,
  default1013 as PipetteIcon,
  default1016 as PizzaIcon,
  default1019 as PlaneIcon,
  default1018 as PlaneLandingIcon,
  default1017 as PlaneTakeoffIcon,
  default1582 as PlayCircleIcon,
  default1021 as PlayIcon,
  default1882 as PlaySquareIcon,
  default1020 as Plug2Icon,
  default1022 as PlugIcon,
  default1770 as PlugZap2Icon,
  default1769 as PlugZapIcon,
  default1588 as PlusCircleIcon,
  default1023 as PlusIcon,
  default1888 as PlusSquareIcon,
  default1024 as PocketKnifeIcon,
  default1025 as PodcastIcon,
  default1026 as PointerIcon,
  default1027 as PointerOffIcon,
  default1028 as PopcornIcon,
  default1029 as PopsicleIcon,
  default1030 as PoundSterlingIcon,
  default1590 as PowerCircleIcon,
  default1032 as PowerIcon,
  default1031 as PowerOffIcon,
  default1890 as PowerSquareIcon,
  default1033 as PresentationIcon,
  default1035 as PrinterCheckIcon,
  default1036 as PrinterIcon,
  default1037 as PrinterXIcon,
  default1034 as ProjectorIcon,
  default1038 as ProportionsIcon,
  default1039 as PuzzleIcon,
  default1040 as PyramidIcon,
  default1041 as QrCodeIcon,
  default1042 as QuoteIcon,
  default1043 as RabbitIcon,
  default1044 as RadarIcon,
  default1045 as RadiationIcon,
  default1046 as RadicalIcon,
  default1049 as RadioIcon,
  default1047 as RadioOffIcon,
  default1048 as RadioReceiverIcon,
  default1050 as RadioTowerIcon,
  default1051 as RadiusIcon,
  default1052 as RainbowIcon,
  default1053 as RatIcon,
  default1054 as RatioIcon,
  default1055 as ReceiptCentIcon,
  default1056 as ReceiptEuroIcon,
  default1065 as ReceiptIcon,
  default1057 as ReceiptIndianRupeeIcon,
  default1058 as ReceiptJapaneseYenIcon,
  default1059 as ReceiptPoundSterlingIcon,
  default1061 as ReceiptRussianRubleIcon,
  default1060 as ReceiptSwissFrancIcon,
  default1062 as ReceiptTextIcon,
  default1063 as ReceiptTurkishLiraIcon,
  default1064 as RectangleCircleIcon,
  default1771 as RectangleEllipsisIcon,
  default1066 as RectangleGogglesIcon,
  default1067 as RectangleHorizontalIcon,
  default1068 as RectangleVerticalIcon,
  default1069 as RecycleIcon,
  default1071 as Redo2Icon,
  default1070 as RedoDotIcon,
  default1072 as RedoIcon,
  default1073 as RefreshCcwDotIcon,
  default1074 as RefreshCcwIcon,
  default1075 as RefreshCwIcon,
  default1076 as RefreshCwOffIcon,
  default1079 as RefrigeratorIcon,
  default1078 as RegexIcon,
  default1077 as RemoveFormattingIcon,
  default1080 as Repeat1Icon,
  default1081 as Repeat2Icon,
  default1083 as RepeatIcon,
  default1082 as RepeatOffIcon,
  default1084 as ReplaceAllIcon,
  default1085 as ReplaceIcon,
  default1086 as ReplyAllIcon,
  default1087 as ReplyIcon,
  default1088 as RewindIcon,
  default1089 as RibbonIcon,
  default1090 as RoadIcon,
  default1091 as RocketIcon,
  default1093 as RockingChairIcon,
  default1092 as RollerCoasterIcon,
  default1094 as RoseIcon,
  default1774 as Rotate3DIcon,
  default1773 as Rotate3dIcon,
  default1097 as RotateCcwIcon,
  default1095 as RotateCcwKeyIcon,
  default1096 as RotateCcwSquareIcon,
  default1099 as RotateCwIcon,
  default1098 as RotateCwSquareIcon,
  default1101 as RouteIcon,
  default1100 as RouteOffIcon,
  default1102 as RouterIcon,
  default1775 as Rows2Icon,
  default1779 as Rows3Icon,
  default1103 as Rows4Icon,
  default1776 as RowsIcon,
  default1104 as RssIcon,
  default1105 as RulerDimensionLineIcon,
  default1106 as RulerIcon,
  default1107 as RussianRubleIcon,
  default1108 as SailboatIcon,
  default1110 as SaladIcon,
  default1109 as SandwichIcon,
  default1112 as SatelliteDishIcon,
  default1111 as SatelliteIcon,
  default1113 as SaudiRiyalIcon,
  default1114 as SaveAllIcon,
  default1118 as SaveIcon,
  default1115 as SaveOffIcon,
  default1778 as Scale3DIcon,
  default1777 as Scale3dIcon,
  default1117 as ScaleIcon,
  default1116 as ScalingIcon,
  default1119 as ScanBarcodeIcon,
  default1120 as ScanEyeIcon,
  default1122 as ScanFaceIcon,
  default1121 as ScanHeartIcon,
  default1127 as ScanIcon,
  default1123 as ScanLineIcon,
  default1124 as ScanQrCodeIcon,
  default1125 as ScanSearchIcon,
  default1126 as ScanTextIcon,
  default1542 as ScatterChartIcon,
  default1933 as School2Icon,
  default1128 as SchoolIcon,
  default1130 as ScissorsIcon,
  default1129 as ScissorsLineDashedIcon,
  default1824 as ScissorsSquareDashedBottomIcon,
  default1892 as ScissorsSquareIcon,
  default1131 as ScooterIcon,
  default1133 as ScreenShareIcon,
  default1132 as ScreenShareOffIcon,
  default1135 as ScrollIcon,
  default1134 as ScrollTextIcon,
  default1136 as SearchAlertIcon,
  default1137 as SearchCheckIcon,
  default1138 as SearchCodeIcon,
  default1142 as SearchIcon,
  default1140 as SearchSlashIcon,
  default1139 as SearchXIcon,
  default1141 as SectionIcon,
  default1782 as SendHorizonalIcon,
  default1781 as SendHorizontalIcon,
  default1143 as SendIcon,
  default1144 as SendToBackIcon,
  default1145 as SeparatorHorizontalIcon,
  default1147 as SeparatorVerticalIcon,
  default1146 as ServerCogIcon,
  default1148 as ServerCrashIcon,
  default1150 as ServerIcon,
  default1149 as ServerOffIcon,
  default1151 as Settings2Icon,
  default1152 as SettingsIcon,
  default1154 as ShapesIcon,
  default1153 as Share2Icon,
  default1155 as ShareIcon,
  default1156 as SheetIcon,
  default1158 as ShellIcon,
  default1157 as ShelvingUnitIcon,
  default1159 as ShieldAlertIcon,
  default1160 as ShieldBanIcon,
  default1161 as ShieldCheckIcon,
  default1786 as ShieldCloseIcon,
  default1162 as ShieldCogCornerIcon,
  default1163 as ShieldCogIcon,
  default1164 as ShieldEllipsisIcon,
  default1165 as ShieldHalfIcon,
  default1170 as ShieldIcon,
  default1166 as ShieldMinusIcon,
  default1167 as ShieldOffIcon,
  default1168 as ShieldPlusIcon,
  default1784 as ShieldQuestionIcon,
  default1783 as ShieldQuestionMarkIcon,
  default1169 as ShieldUserIcon,
  default1785 as ShieldXIcon,
  default1172 as ShipIcon,
  default1171 as ShipWheelIcon,
  default1173 as ShirtIcon,
  default1174 as ShoppingBagIcon,
  default1175 as ShoppingBasketIcon,
  default1176 as ShoppingCartIcon,
  default1177 as ShovelIcon,
  default1178 as ShowerHeadIcon,
  default1179 as ShredderIcon,
  default1180 as ShrimpIcon,
  default1181 as ShrinkIcon,
  default1182 as ShrubIcon,
  default1184 as ShuffleIcon,
  default1748 as SidebarCloseIcon,
  default1758 as SidebarIcon,
  default1756 as SidebarOpenIcon,
  default1183 as SigmaIcon,
  default1894 as SigmaSquareIcon,
  default1185 as SignalHighIcon,
  default1189 as SignalIcon,
  default1188 as SignalLowIcon,
  default1186 as SignalMediumIcon,
  default1187 as SignalZeroIcon,
  default1191 as SignatureIcon,
  default1190 as SignpostBigIcon,
  default1192 as SignpostIcon,
  default1193 as SirenIcon,
  default1194 as SkipBackIcon,
  default1195 as SkipForwardIcon,
  default1196 as SkullIcon,
  default1197 as SlashIcon,
  default1896 as SlashSquareIcon,
  default1198 as SliceIcon,
  default1199 as SlidersHorizontalIcon,
  default1788 as SlidersIcon,
  default1787 as SlidersVerticalIcon,
  default1200 as SmartphoneChargingIcon,
  default1202 as SmartphoneIcon,
  default1201 as SmartphoneNfcIcon,
  default1204 as SmileIcon,
  default1203 as SmilePlusIcon,
  default1205 as SnailIcon,
  default1206 as SnowflakeIcon,
  default1207 as SoapDispenserDropletIcon,
  default1208 as SofaIcon,
  default1210 as SolarPanelIcon,
  default1496 as SortAscIcon,
  default1492 as SortDescIcon,
  default1209 as SoupIcon,
  default1211 as SpaceIcon,
  default1212 as SpadeIcon,
  default1213 as SparkleIcon,
  default1789 as SparklesIcon,
  default1214 as SpeakerIcon,
  default1215 as SpeechIcon,
  default1216 as SpellCheck2Icon,
  default1217 as SpellCheckIcon,
  default1219 as SplineIcon,
  default1218 as SplinePointerIcon,
  default1220 as SplitIcon,
  default1898 as SplitSquareHorizontalIcon,
  default1900 as SplitSquareVerticalIcon,
  default1221 as SpoolIcon,
  default1223 as SportShoeIcon,
  default1222 as SpotlightIcon,
  default1224 as SprayCanIcon,
  default1225 as SproutIcon,
  default1791 as SquareActivityIcon,
  default1795 as SquareArrowDownIcon,
  default1793 as SquareArrowDownLeftIcon,
  default1797 as SquareArrowDownRightIcon,
  default1799 as SquareArrowLeftIcon,
  default1801 as SquareArrowOutDownLeftIcon,
  default1803 as SquareArrowOutDownRightIcon,
  default1805 as SquareArrowOutUpLeftIcon,
  default1807 as SquareArrowOutUpRightIcon,
  default1226 as SquareArrowRightEnterIcon,
  default1227 as SquareArrowRightExitIcon,
  default1809 as SquareArrowRightIcon,
  default1813 as SquareArrowUpIcon,
  default1811 as SquareArrowUpLeftIcon,
  default1815 as SquareArrowUpRightIcon,
  default1817 as SquareAsteriskIcon,
  default1823 as SquareBottomDashedScissorsIcon,
  default1821 as SquareCenterlineDashedHorizontalIcon,
  default1819 as SquareCenterlineDashedVerticalIcon,
  default1825 as SquareChartGanttIcon,
  default1828 as SquareCheckBigIcon,
  default1830 as SquareCheckIcon,
  default1836 as SquareChevronDownIcon,
  default1834 as SquareChevronLeftIcon,
  default1832 as SquareChevronRightIcon,
  default1838 as SquareChevronUpIcon,
  default1840 as SquareCodeIcon,
  default1228 as SquareDashedBottomCodeIcon,
  default1229 as SquareDashedBottomIcon,
  default1849 as SquareDashedIcon,
  default1842 as SquareDashedKanbanIcon,
  default1844 as SquareDashedMousePointerIcon,
  default1846 as SquareDashedTextIcon,
  default1230 as SquareDashedTopSolidIcon,
  default1851 as SquareDivideIcon,
  default1853 as SquareDotIcon,
  default1857 as SquareEqualIcon,
  default1855 as SquareFunctionIcon,
  default1827 as SquareGanttChartIcon,
  default1239 as SquareIcon,
  default1861 as SquareKanbanIcon,
  default1859 as SquareLibraryIcon,
  default1863 as SquareMIcon,
  default1865 as SquareMenuIcon,
  default1867 as SquareMinusIcon,
  default1869 as SquareMousePointerIcon,
  default1873 as SquareParkingIcon,
  default1871 as SquareParkingOffIcon,
  default1231 as SquarePauseIcon,
  default1877 as SquarePenIcon,
  default1875 as SquarePercentIcon,
  default1883 as SquarePiIcon,
  default1885 as SquarePilcrowIcon,
  default1881 as SquarePlayIcon,
  default1887 as SquarePlusIcon,
  default1889 as SquarePowerIcon,
  default1232 as SquareRadicalIcon,
  default1233 as SquareRoundCornerIcon,
  default1891 as SquareScissorsIcon,
  default1893 as SquareSigmaIcon,
  default1895 as SquareSlashIcon,
  default1897 as SquareSplitHorizontalIcon,
  default1899 as SquareSplitVerticalIcon,
  default1234 as SquareSquareIcon,
  default1235 as SquareStackIcon,
  default1236 as SquareStarIcon,
  default1237 as SquareStopIcon,
  default1901 as SquareTerminalIcon,
  default1907 as SquareUserIcon,
  default1903 as SquareUserRoundIcon,
  default1905 as SquareXIcon,
  default1238 as SquaresExcludeIcon,
  default1240 as SquaresIntersectIcon,
  default1241 as SquaresSubtractIcon,
  default1242 as SquaresUniteIcon,
  default1244 as SquircleDashedIcon,
  default1243 as SquircleIcon,
  default1245 as SquirrelIcon,
  default1246 as StampIcon,
  default1247 as StarHalfIcon,
  default1250 as StarIcon,
  default1248 as StarOffIcon,
  default1790 as StarsIcon,
  default1249 as StepBackIcon,
  default1251 as StepForwardIcon,
  default1252 as StethoscopeIcon,
  default1253 as StickerIcon,
  default1255 as StickyNoteCheckIcon,
  default1259 as StickyNoteIcon,
  default1254 as StickyNoteMinusIcon,
  default1257 as StickyNoteOffIcon,
  default1256 as StickyNotePlusIcon,
  default1258 as StickyNoteXIcon,
  default1260 as StickyNotesIcon,
  default1261 as StoneIcon,
  default1597 as StopCircleIcon,
  default1262 as StoreIcon,
  default1263 as StretchHorizontalIcon,
  default1264 as StretchVerticalIcon,
  default1265 as StrikethroughIcon,
  default1266 as SubscriptIcon,
  default1514 as SubtitlesIcon,
  default1267 as SunDimIcon,
  default1271 as SunIcon,
  default1268 as SunMediumIcon,
  default1269 as SunMoonIcon,
  default1270 as SunSnowIcon,
  default1272 as SunriseIcon,
  default1275 as SunsetIcon,
  default1273 as SuperscriptIcon,
  default1274 as SwatchBookIcon,
  default1276 as SwissFrancIcon,
  default1277 as SwitchCameraIcon,
  default1278 as SwordIcon,
  default1279 as SwordsIcon,
  default1280 as SyringeIcon,
  default1281 as Table2Icon,
  default1282 as TableCellsMergeIcon,
  default1283 as TableCellsSplitIcon,
  default1284 as TableColumnsSplitIcon,
  default1618 as TableConfigIcon,
  default1287 as TableIcon,
  default1285 as TableOfContentsIcon,
  default1288 as TablePropertiesIcon,
  default1286 as TableRowsSplitIcon,
  default1291 as TabletIcon,
  default1289 as TabletSmartphoneIcon,
  default1290 as TabletsIcon,
  default1292 as TagIcon,
  default1295 as TagsIcon,
  default1294 as Tally1Icon,
  default1293 as Tally2Icon,
  default1296 as Tally3Icon,
  default1297 as Tally4Icon,
  default1298 as Tally5Icon,
  default1299 as TangentIcon,
  default1301 as TargetIcon,
  default1300 as TelescopeIcon,
  default1302 as TentIcon,
  default1303 as TentTreeIcon,
  default1304 as TerminalIcon,
  default1902 as TerminalSquareIcon,
  default1910 as TestTube2Icon,
  default1909 as TestTubeDiagonalIcon,
  default1305 as TestTubeIcon,
  default1306 as TestTubesIcon,
  default1915 as TextAlignCenterIcon,
  default1913 as TextAlignEndIcon,
  default1911 as TextAlignJustifyIcon,
  default1917 as TextAlignStartIcon,
  default1308 as TextCursorIcon,
  default1307 as TextCursorInputIcon,
  default1919 as TextIcon,
  default1922 as TextInitialIcon,
  default1309 as TextQuoteIcon,
  default1310 as TextSearchIcon,
  default1848 as TextSelectIcon,
  default1847 as TextSelectionIcon,
  default1920 as TextWrapIcon,
  default1311 as TheaterIcon,
  default1314 as ThermometerIcon,
  default1312 as ThermometerSnowflakeIcon,
  default1313 as ThermometerSunIcon,
  default1315 as ThumbsDownIcon,
  default1316 as ThumbsUpIcon,
  default1318 as TicketCheckIcon,
  default1322 as TicketIcon,
  default1317 as TicketMinusIcon,
  default1320 as TicketPercentIcon,
  default1319 as TicketPlusIcon,
  default1323 as TicketSlashIcon,
  default1321 as TicketXIcon,
  default1325 as TicketsIcon,
  default1324 as TicketsPlaneIcon,
  default1326 as TimelineIcon,
  default1329 as TimerIcon,
  default1327 as TimerOffIcon,
  default1328 as TimerResetIcon,
  default1330 as ToggleLeftIcon,
  default1332 as ToggleRightIcon,
  default1334 as ToiletIcon,
  default1331 as ToolCaseIcon,
  default1335 as ToolboxIcon,
  default1333 as TornadoIcon,
  default1336 as TorusIcon,
  default1339 as TouchpadIcon,
  default1337 as TouchpadOffIcon,
  default1338 as TowelRackIcon,
  default1340 as TowerControlIcon,
  default1342 as ToyBrickIcon,
  default1341 as TractorIcon,
  default1343 as TrafficConeIcon,
  default1345 as TrainFrontIcon,
  default1344 as TrainFrontTunnelIcon,
  default1925 as TrainIcon,
  default1346 as TrainTrackIcon,
  default1924 as TramFrontIcon,
  default1347 as TransgenderIcon,
  default1348 as Trash2Icon,
  default1350 as TrashIcon,
  default1349 as TreeDeciduousIcon,
  default1926 as TreePalmIcon,
  default1351 as TreePineIcon,
  default1352 as TreesIcon,
  default1353 as TrendingDownIcon,
  default1354 as TrendingUpDownIcon,
  default1355 as TrendingUpIcon,
  default1928 as TriangleAlertIcon,
  default1356 as TriangleDashedIcon,
  default1358 as TriangleIcon,
  default1357 as TriangleRightIcon,
  default1359 as TrophyIcon,
  default1360 as TruckElectricIcon,
  default1361 as TruckIcon,
  default1362 as TurkishLiraIcon,
  default1363 as TurntableIcon,
  default1364 as TurtleIcon,
  default1931 as Tv2Icon,
  default1366 as TvIcon,
  default1930 as TvMinimalIcon,
  default1365 as TvMinimalPlayIcon,
  default1368 as TypeIcon,
  default1367 as TypeOutlineIcon,
  default1369 as UmbrellaIcon,
  default1370 as UmbrellaOffIcon,
  default1372 as UnderlineIcon,
  default1371 as Undo2Icon,
  default1373 as UndoDotIcon,
  default1374 as UndoIcon,
  default1376 as UnfoldHorizontalIcon,
  default1375 as UnfoldVerticalIcon,
  default1377 as UngroupIcon,
  default1932 as UniversityIcon,
  default1378 as Unlink2Icon,
  default1379 as UnlinkIcon,
  default1728 as UnlockIcon,
  default1730 as UnlockKeyholeIcon,
  default1380 as UnplugIcon,
  default1611 as UploadCloudIcon,
  default1381 as UploadIcon,
  default1382 as UsbIcon,
  default1945 as User2Icon,
  default1937 as UserCheck2Icon,
  default1383 as UserCheckIcon,
  default1599 as UserCircle2Icon,
  default1601 as UserCircleIcon,
  default1935 as UserCog2Icon,
  default1384 as UserCogIcon,
  default1396 as UserIcon,
  default1385 as UserKeyIcon,
  default1387 as UserLockIcon,
  default1941 as UserMinus2Icon,
  default1386 as UserMinusIcon,
  default1388 as UserPenIcon,
  default1939 as UserPlus2Icon,
  default1389 as UserPlusIcon,
  default1936 as UserRoundCheckIcon,
  default1934 as UserRoundCogIcon,
  default1944 as UserRoundIcon,
  default1390 as UserRoundKeyIcon,
  default1940 as UserRoundMinusIcon,
  default1391 as UserRoundPenIcon,
  default1938 as UserRoundPlusIcon,
  default1392 as UserRoundSearchIcon,
  default1942 as UserRoundXIcon,
  default1393 as UserSearchIcon,
  default1904 as UserSquare2Icon,
  default1908 as UserSquareIcon,
  default1394 as UserStarIcon,
  default1943 as UserX2Icon,
  default1395 as UserXIcon,
  default1949 as Users2Icon,
  default1397 as UsersIcon,
  default1948 as UsersRoundIcon,
  default1946 as UtensilsCrossedIcon,
  default1950 as UtensilsIcon,
  default1398 as UtilityPoleIcon,
  default1399 as VanIcon,
  default1400 as VariableIcon,
  default1401 as VaultIcon,
  default1402 as VectorSquareIcon,
  default1403 as VeganIcon,
  default1404 as VenetianMaskIcon,
  default1405 as VenusAndMarsIcon,
  default1406 as VenusIcon,
  default1502 as VerifiedIcon,
  default1408 as VibrateIcon,
  default1407 as VibrateOffIcon,
  default1410 as VideoIcon,
  default1409 as VideoOffIcon,
  default1411 as VideotapeIcon,
  default1412 as ViewIcon,
  default1413 as VoicemailIcon,
  default1414 as VolleyballIcon,
  default1415 as Volume1Icon,
  default1417 as Volume2Icon,
  default1419 as VolumeIcon,
  default1416 as VolumeOffIcon,
  default1418 as VolumeXIcon,
  default1420 as VoteIcon,
  default1953 as Wallet2Icon,
  default1421 as WalletCardsIcon,
  default1423 as WalletIcon,
  default1952 as WalletMinimalIcon,
  default1422 as WallpaperIcon,
  default1955 as Wand2Icon,
  default1425 as WandIcon,
  default1954 as WandSparklesIcon,
  default1424 as WarehouseIcon,
  default1426 as WashingMachineIcon,
  default1427 as WatchIcon,
  default1428 as WavesArrowDownIcon,
  default1429 as WavesArrowUpIcon,
  default1956 as WavesHorizontalIcon,
  default1957 as WavesIcon,
  default1431 as WavesLadderIcon,
  default1430 as WavesVerticalIcon,
  default1432 as WaypointsIcon,
  default1433 as WebcamIcon,
  default1435 as WebhookIcon,
  default1434 as WebhookOffIcon,
  default1437 as WeightIcon,
  default1436 as WeightTildeIcon,
  default1439 as WheatIcon,
  default1438 as WheatOffIcon,
  default1440 as WholeWordIcon,
  default1441 as WifiCogIcon,
  default1442 as WifiHighIcon,
  default1447 as WifiIcon,
  default1443 as WifiLowIcon,
  default1444 as WifiOffIcon,
  default1445 as WifiPenIcon,
  default1446 as WifiSyncIcon,
  default1448 as WifiZeroIcon,
  default1449 as WindArrowDownIcon,
  default1450 as WindIcon,
  default1451 as WineIcon,
  default1452 as WineOffIcon,
  default1453 as WorkflowIcon,
  default1454 as WormIcon,
  default1921 as WrapTextIcon,
  default1455 as WrenchIcon,
  default1603 as XCircleIcon,
  default1457 as XIcon,
  default1456 as XLineTopIcon,
  default1746 as XOctagonIcon,
  default1906 as XSquareIcon,
  default1460 as ZapIcon,
  default1458 as ZapOffIcon,
  default1461 as ZodiacAquariusIcon,
  default1459 as ZodiacAriesIcon,
  default1462 as ZodiacCancerIcon,
  default1466 as ZodiacCapricornIcon,
  default1463 as ZodiacGeminiIcon,
  default1464 as ZodiacLeoIcon,
  default1467 as ZodiacLibraIcon,
  default1465 as ZodiacOphiuchusIcon,
  default1468 as ZodiacPiscesIcon,
  default1470 as ZodiacSagittariusIcon,
  default1469 as ZodiacScorpioIcon,
  default1471 as ZodiacTaurusIcon,
  default1472 as ZodiacVirgoIcon,
  default1474 as ZoomInIcon,
  default1473 as ZoomOutIcon
};
